---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  pdf_document:
. includes:
. . in_header: header.tex
. keep_tex: yes
  word_document: default
geometry: margin=1.0in
bibliography: reference.bib
---

# The Fecal Microbiome Before and After Treatment for Colorectal Adenoma or Carcinoma
```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("dplyr", "reshape2", "scales", "knitr", "rmarkdown"))

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

```

```{r rdata_tables}
# Extra data and meta data needed
good_metaf <- read.csv('../data/process/mod_metadata/good_metaf_final.csv', header = T, stringsAsFactors = F)
metaI <- read.csv('../data/process/mod_metadata/metaI_final.csv', header = T, stringsAsFactors = F)
total_sub_Seqs <- read.delim('../data/process/final.0.03.subsample.shared', header = T) %>%
  select(contains("Otu0")) %>% rowSums()

# initial microbiome and FIT description data
change_summary <- read.csv('../data/process/tables/change_theta_fit_summary.csv', header = T, row.names = 1)
alpha_summary <- read.csv('../data/process/tables/alpha_table_summary.csv', header = T, row.names = 1)
beta_summary <- read.csv('../data/process/tables/beta_diver_summary.csv', header = T, row.names = 1)
time_pvalues <- read.csv("../data/process/tables/time_pvalues.csv", header = T, row.names = 1)
time_summary <- read.csv("../data/process/tables/time_summary_data.csv", header = T, row.names = 1)
all_OTU_pair_wilcox <- read.csv("../data/process/tables/OTU_paired_wilcoxson_test.csv", header = T)

# Adenoma Model
adn_varsMDA <- read.csv('../data/process/tables/adn_reduced_model_top_vars_MDA_Summary.csv', header = T)
adn_imp_vars <- read.csv('../data/process/tables/adn_rf_otu_tax.csv', header = T, row.names = 1)
adn_AUC_run_summary <- read.csv('../data/process/tables/adn_Reduced_ROC_model_summary.csv', header = T)
adn_general_summary <- read.csv('../data/process/tables/adn_reduced_auc_summary.csv', header = T, row.names = 1)

# SRN Model
srn_varsMDA <- read.csv('../data/process/tables/srn_reduced_model_top_vars_MDA_Summary.csv', header = T)
srn_imp_vars <- read.csv('../data/process/tables/srn_rf_otu_tax.csv', header = T, row.names = 1)
srn_AUC_run_summary <- read.csv('../data/process/tables/srn_Reduced_ROC_model_summary.csv', header = T)
srn_general_summary <- read.csv('../data/process/tables/srn_reduced_auc_summary.csv', header = T, row.names = 1)

# CRC Model
crc_varsMDA <- read.csv('../data/process/tables/reduced_crc_model_top_vars_MDA_Summary.csv', header = T)
crc_imp_vars <- read.csv('../data/process/tables/crc_rf_otu_tax.csv', header = T, row.names = 1)
crc_AUC_run_summary <- read.csv('../data/process/tables/crc_Reduced_ROC_model_summary.csv', header = T)
crc_general_summary <- read.csv('../data/process/tables/crc_reduced_auc_summary.csv', header = T, row.names = 1)

# All model information
all_probs_paired_wilcox <- read.csv('../data/process/tables/all_crc_srn_adn_models_wilcox_paired_pvalue_summary.csv', header = T, stringsAsFactors = F, row.names = 4)
all_pred_stat_summary <- read.csv('../data/process/tables/all_crc_srn_adn_models_summary_info.csv', header = T, stringsAsFactors = F) %>% rename(comparison = X)
all_confu_matrix <- read.csv('../data/process/tables/all_crc_srn_adn_models_confusion_summary.csv', header = T, stringsAsFactors = F)

# Common Variable Info
common_vars_summary <- read.csv('../data/process/tables/pvalue_adn_srn_crc_common_imp_vars.csv', header = T, row.names = 1, stringsAsFactors = F)

#Therapy Info
chemo_rad_summary <- read.csv('../data/process/tables/crc_probs_chemo_rad_pvalue_summary.csv', header = T, row.names = 1)
chemo_rad_common_vars_summary <- read.csv('../data/process/tables/chemo_rads_treatment_pvalue_summary.csv', header = T, stringsAsFactors = F)

comb_adn_surg_summary <- read.csv('../data/process/tables/adn_combined_probs_surgery_pvalue_summary.csv', header = T, stringsAsFactors = F, row.names = 1)

comb_adn_surg_common_vars_summary <- read.csv('../data/process/tables/all_adn_surg_pvalue_summary.csv', header = T, stringsAsFactors = F)

```
\vspace{25mm}

\begin{center}
Running Title: Human Microbiome before and after Colorectal Cancer

\vspace{10mm}

Marc A Sze${^1}$, Nielson T Baxter${^2}$, Mack T Ruffin IV${^3}$, Mary AM Rogers${^2}$, and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI

$2$ Department of Internal Medicine, University of Michigan, Ann Arbor, M.

$3$ Department of Family Medicine and Community Medicine, Penn State Hershey Medical Center, Hershey, P.


\end{center.


\newpage
\linenumbers

## Abstract [ edit at end ]

**Background.** Colorectal cancer (CRC) is a worldwide health problem. Despite growing evidence that members of the gut microbiota can drive tumorigenesis, little is known about what happens to the microbiome after treatment for an adenoma or carcinoma. This study tested the hypothesis that treatment for adenoma or carcinoma alters the abundance of bacterial populations associated with with disease to those associated with a normal colon. We tested this hypothesis by sequencing the 16S rRNA genes in the feces of `r length(rownames(good_metaf))` individuals before and after treatment for adenoma (N=`r change_summary["thetayc", "adn_n"]`), advanced adenoma (N=`r change_summary["thetayc", "srn_n"]`), and carcinoma (N=`r change_summary["thetayc", "crc_n"]`).

**Results.** There were large changes the the bacterial communities associated with treatment across the three groups. The communities from patients with carcinomas changed significantly more than those with adenoma following treatment (P-value=`r change_summary["thetayc", "Pvalue_adnVcrc"]`); however, there was not a significant difference between those with advanced adenoma and those with adenoma or carcinoma (P-value>0.05). Although treatment brought about large intrapersonal changes, the change in the abundance of individual OTUs to treatment was not consistent within diagnosis groups (P-value=`r ifelse(min(all_OTU_pair_wilcox$BH_corrected, na.rm = TRUE) < 0.05, invisible("<"), invisible(">"))` 0.05). Because the distribution of OTUs across patients and diagnosis groups was patchy, we used the Random Forest machine learning algorithm to identify groups of OTUs that allowed us to successfully distinguish between pre- and post-treatment samples for each of the diagnosis groups. The OTUs that were shared across the models for the three diagnosis groups were dominated by lineages that are commonly considered to be commensals (e.g. *Lachnospiraceae*, *Bacteroides*, *Anaerostipes*, *Blautia*, and *Dorea*).

4. Models suggest that the positive predictive value from diagnostic models decrease following treatment of carcinomas

Finally, we used a larger cohort that contained individuals with

Using diagnostic Random Forest models that were trained to differentiate between individuals with normal colons or those with adenomas, advanced adenomas, or carcinomas we measured the change in the positive probability of having one of the three diagnoses.

Using Random Forest models to assess whether changes in post-treatment samples were towards a normal community, only those with carcinoma had a significant decrease in positive probability (P-value `r ifelse(all_probs_paired_wilcox["carcinoma", "BH_correction"] < 0.05, invisible("<"), invisible(">"))` 0.05); providing further evidence that treatment has the greatest effect in those with carcinoma.

5. CRC-associated bacterial populations are unique and they decline in relative abundance with treatment




6. Among the patients with carcinoma, the type of treatment did not significantly alter the amount of change to their bacterial communities.

Both chemotherapy and radiation did not provide any additional changes to the bacterial community in those treated for carcinoma (P-value `r ifelse(min(chemo_rad_summary$bh, na.rm = TRUE) < 0.05, invisible("<"), invisible(">"))` 0.05).



**Conclusions.** Our data partially supports the hypothesis that the microbiome changes after treatment towards a normal community. Individuals with carcinoma had more drastic differences to the overall community than those with adenoma. Common OTUs to all models were overwhelmingly from commensal bacteria, suggesting that these bacteria may be important in initial polyp formation, development of advanced adenoma, and transition to carcinoma.

\newpage

### Keywords

bacterial microbiome; colorectal cancer; polyps; FIT; post-surgery; risk factor. .

\newpage

## Background

Colorectal cancer (CRC) is the third most common cause of cancer deaths in the United States [@jemal_cancer_2010; @haggar_colorectal_2009]. The rate of disease mortality has seen a significant decrease, thanks mainly to improvements in screening [@jemal_cancer_2010]. Despite this improvement, there are still approximately 50,000 deaths from the disease per year [@haggar_colorectal_2009]. Current estimates indicate that 20-30% of those who undergo treatment will experience recurrence and 30-50% of those patients will die [@hellinger_reoperation_2006; @ryuk_predictive_2014]. Identification of methods to assess patients' risk of recurrence is of great importance to reduce mortality and healthcare costs.

[ 	fix: the following paragraph is mixing two things - mechanistic studies and biomarker studies. they need to be split up so that you establish that there is a mechanistic link and that we can use biomarkers to diagnose people with adenomas and carcinomas from those with normal colons. ]

Recent studies in humans and mouse models have demonstrated a connection between the  that altered membership and structure of the gut microbiome correlate with CRC pathogenesis [@zackular_manipulation_2016; @arthur_microbial_2014; @zackular_gut_2013]. Further, bacterial communities on the mucosa are altered between normal and tumor tissue [@dejea_microbiota_2014].
Collectively, these studies provide a tantalizing link between our gut bacteria and CRC and suggest the possibility of using biomarkers to diagnose disease.

Indeed, models created with microbiome data and clinical tests, such as Fecal Immunoglobulin Test (FIT), result in good predictions of CRC [@baxter_microbiota-based_2016; @zeller_potential_2014]. Although these studies show how changed microbial communities or invasion by inflammatory species can impact CRC progression [@flynn_metabolic_2016], they provide very little information as to whether treatment to remove lesions affects the presence of bacterial populations associated with those lesions.

Providing an answer to this question is important because it has far reaching implications on both how the bacterial community causes the formation of more polyps [@zackular_manipulation_2016; @arthur_microbial_2014] and the ability to use the microbiome as a predictive screening tool [@baxter_microbiota-based_2016; @zeller_potential_2014]. Understanding polyp formation and transition to advanced adenoma and then carcinoma is crucial to being able to understand how to prevent CRC occurrence. Response of the community to treatment is also equally important to predictive models designed for screening purposes since an unresponsive community would provide little additional information for important events, such as recurrence [@hassan_efficacy_2016].

Using pre- (initial) and post- (follow up) treatment samples we tested the hypothesis that treatment causes detectable changes towards a normal microbiome in those with adenoma, advanced adenoma, and carcinoma. First, we assessed differences between samples pre- and post-treatment in adenoma, advanced adenoma, or carcinoma using alpha or beta diversity metrics. Second, we explored whether models built to classify adenoma, advanced adenoma, or carcinoma versus normal were able to identify specific community members that differed between pre- and post-treatment samples. We also used these models to assess whether changes in the community were toward a more normal microbiome. Finally, we assessed both whether surgery for adenomas and advanced adenomas provided larger community changes and whether chemotherapy or radiation provided additional changes to the microbiome over surgical resection. This study aims to provide evidence as to whether the altered CRC microbiome persists or shifts back towards normal after such interventions.



\newpage

## Results


***The Bacterial Community:***
Within our `r length(rownames(good_metaf))`-person cohort we tested whether the microbiome in patients with adenoma (N=`r change_summary["thetayc", "adn_n"]`), advanced adenoma (N=`r change_summary["thetayc", "srn_n"]`), or carcinoma (N=`r change_summary["thetayc", "crc_n"]`) had any broad differences between pre- and post-treatment samples. We found that carcinoma patients had a more dissimilar bacterial community between these samples than those with adenoma (P-value `r ifelse(change_summary["thetayc", "Pvalue_adnVcrc"] < 0.001, invisible("<"), invisible(">"))` 0.001) [Figure 1A]. Although no significant differences were observed between advanced adenoma and carcinoma there was an increase in the dissimilarity of pre- and post-treatment samples from adenoma (`r paste(format(change_summary["thetayc", "adn_mean"], digits = 2), label = "\u00B1", format(change_summary["thetayc", "adn_SD"], digits = 2))` (mean `r paste(label = "\u00B1")` SD)) to advanced adenoma (`r paste(format(change_summary["thetayc", "srn_mean"], digits = 2), label = "\u00B1", format(change_summary["thetayc", "srn_SD"], digits = 2))`) to carcinoma (`r paste(format(change_summary["thetayc", "crc_mean"], digits = 2), label = "\u00B1", format(change_summary["thetayc", "crc_SD"], digits = 2))`) [Figure 1A]. The bacterial community structure pre- post-treatment was visualized using NMDS for adenoma [Figure 1B] (PERMANOVA `r ifelse(beta_summary["Adenoma", "PERMANOVA"] < 0.05, invisible("<"), invisible(">"))` 0.05), advanced adenoma [Figure 1C] (PERMANOVA `r ifelse(beta_summary["SRN", "PERMANOVA"] < 0.05, invisible("<"), invisible(">"))` 0.05), and carcinoma [Figure 1D] (PERMANOVA `r ifelse(beta_summary["Carcinoma", "PERMANOVA"] < 0.05, invisible("<"), invisible(">"))` 0.05). Interestingly, when pre- and post-treatment samples were compared, regardless of whether they were adenoma or carcinoma, there was no significant overall difference in beta diversity (PERMANOVA `r ifelse(beta_summary["All", "PERMANOVA"] < 0.05, invisible("<"), invisible(">"))` 0.05). There was no difference between pre- and post-treatment samples when investigating alpha diversity metrics for adenoma, advanced adenoma, or carcinoma [Table S1]. Additionally, there was also no difference in the relative abundance of any specific OTU between these samples for adenoma, advanced adenoma, or carcinoma only [Figure S1].


***Adenoma Model:***
Using normal and adenoma individuals from a separate cohort we created an adenoma model to help assess whether those with adenoma in this cohort changed towards normal. The range of model AUC's from 100 runs of 20 repeated 10-fold cross-validation was `r format(min(adn_AUC_run_summary$ROC), digits = 2)` - `r format(max(adn_AUC_run_summary$ROC), digits = 2)` with the AUC of the model used for classification having an AUC of `r format(adn_general_summary["full", "AUC"], digits = 2)` [Figure S2A]. There was a total of `r length(rownames(adn_imp_vars))` OTUs in this model with the vast majority classifying to bacteria typically thought of as commensal [Figure S3A]. There was a significant difference between the actual and predicted disease classification (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "adn" & comparison == "McnemarPValue")[, "all"] < 0.05, invisible("<"), invisible(">"))` 0.05). There was also no significant decrease in the positive probability of adenoma between pre- and post-treatment samples (P-value `r ifelse(all_probs_paired_wilcox["adenoma", "BH_correction"] < 0.05, invisible("<"), invisible(">"))` 0.05) [Figure 2A].


***Advanced Adenoma Model:***
Using normal and advanced adenoma individuals from a separate cohort we created an advanced adenoma model to help assess whether those with advanced adenoma in this cohort changed towards normal. The range of model AUC's from 100 runs of 20 repeated 10-fold cross-validation was `r format(min(srn_AUC_run_summary$ROC), digits = 2)` - `r format(max(srn_AUC_run_summary$ROC), digits = 2)` with the AUC of the model used for classification having an AUC of `r format(srn_general_summary["full", "AUC"], digits = 2)` [Figure S2B]. There was a total of `r length(rownames(srn_imp_vars))` OTUs in the advanced adenoma model [Figure S3B]. Similar to the adenoma model the vast majority of OTUs classified to bacteria typically thought of as commensal. Also similar to the adenoma model there was a significant difference between the actual and predicted disease classification (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "SRN" & comparison == "McnemarPValue")[, "all"] < 0.05, invisible("<"), invisible(">"))` 0.05) and no significant decrease in the positive probability of advanced adenoma between pre- and post-treatment samples (P-value `r ifelse(all_probs_paired_wilcox["SRN", "BH_correction"] < 0.05, invisible("<"), invisible(">"))` 0.05) [Figure 2B].


***Carcinoma Model:***
Using normal and carcinoma individuals from a separate cohort we created a carcinoma model to help assess whether those with carcinoma in this cohort changed towards normal. The range of model AUC's from 100 runs of 20 repeated 10-fold cross-validation was `r format(min(crc_AUC_run_summary$ROC), digits = 2)` - `r format(max(crc_AUC_run_summary$ROC), digits = 2)` with the AUC of the model used for classification being `r format(crc_general_summary["full", "AUC"], digits = 2)` [Figure S2C]. Interestingly, the AUCs improved from adenoma to advanced adenoma to carcinoma [Figure 2]. There was a total of `r length(rownames(crc_imp_vars))` OTUs in the carcinoma model [Figure S3C]. Similar to the adenoma and advanced adenoma models the vast majority of OTUs classified to bacteria typically thought of as commensal but OTUs that also classified to *Fusobacterium*, *Porphyromonas*, and *Parvimonas* were also important for carcinoma classification [Figure S3C]. Also, like the adenoma and advanced adenoma models there was a significant difference between the actual and predicted disease classification (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "crc" & comparison == "McnemarPValue")[, "all"] < 0.05, invisible("<"), invisible(">"))` 0.05). There was a significant decrease in the positive probability of carcinoma between pre- and post-treatment samples (P-value `r ifelse(all_probs_paired_wilcox["carcinoma", "BH_correction"] < 0.05, invisible("<"), invisible(">"))` 0.05) [Figure 2C]; suggesting that the carcinoma samples changed towards normal after treatment, unlike either adenoma or advanced adenoma. The one individual still positive for carcinoma after treatment had an increase in carcinoma positive proability between their pre- and post-treatment sample [Figure 2C].


***Common OTUs:***
Next, to identify which OTUs were important at all stages of disease, we identified common predictive OTUs within the adenoma, advanced adenoma, and carcinoma models. When we compared the three different models with each other there were a total of `r length(rownames(common_vars_summary))` common OTUs. Some of the most common taxonomic identifications belonged to *Bacteroides*, *Blautia*, *Anaerostipes*, *Lachnospiraceae*, and *Dorea*. The vast majority of the OTUs that were common between these models had classifications to bacteria typically thought of as commensal [Table S2]. The rank of these OTUs, based on median mean decrease in accuracy (MDA), varied higly depending on whether they were in the adenoma, advanced adenoma, or carcinoma model [Figure 3]. As an example, *Bacteroides* (OTU85) was ranked `r filter(crc_varsMDA, otu == "Otu000085") %>% select(rank)`^th^ in the carcinoma model but in both the adenoma and advanced adenoma model ranked `r filter(adn_varsMDA, otu == "Otu000085") %>% select(rank)`^th^ and `r filter(srn_varsMDA, otu == "Otu000085") %>% select(rank)`^st^ respectively [Figure 3]. .


***Treatment Affects on Community:***
After observing these changes from treatment we assessed the possible confounding impact of chemotherapy or radiation, in the carcinoma group, and surgical resection, in the adenoma group, on the observed results. In the carcinoma group neither chemotherapy nor radiation provided any additional change from initial sample over surgery alone (P-value `r ifelse(min(chemo_rad_summary$bh, na.rm = TRUE) < 0.05, invisible("<"), invisible(">"))` 0.05) [Table S3]. For the adenoma group there was a single difference in observed OTUs (sobs) between those that received surgerical resection and those that had regular polyp removal (P-value `r ifelse(min(comb_adn_surg_summary$bh, na.rm = TRUE) < 0.05, invisible("<"), invisible(">"))` 0.05) [Table S4]. For the surgical resection comparison, adenoma and advanced adenoma were combined due to the low number of resection occurances in these two groups. Using a fisher exact test there was no difference in the proportion of those receiving surgerical resection between the adenoma and advanced adenoma groups (P-value `r ifelse(comb_adn_surg_summary["prop_surg_adn_srn", "pvalue"] < 0.05, invisible("<"), invisible(">"))` 0.05). This data suggests that the microbiome changes observed in the carcinoma group were mostly a result from surgerical resection and not from chemotherapy or radiation.


\newpage


## Discussion
This study builds upon previous work from numerous labs that have considered both how the bacterial community between those with and without CRC differs and how it might be used as an early screening tool [@yu_metagenomic_2017; @baxter_microbiota-based_2016; @zeller_potential_2014; @zackular_human_2014; @warren_co-occurrence_2013]. Here we show that the microbiome changes towards normal after treatment for carcinoma and that chemotherapy and radiation did not provide any additional change. Although some of the important OTUs classified to genera from bacteria considered the "usual suspects" (e.g. *Fusobacterium*, *Porphyromonas*, and *Parvimonas*) many did not. The majority of important OTUs had taxonomic classifications for resident gut microbes and were common for the adenoma, advanced adenoma, and carcinoma models. This suggests that members within the commensal community may be the first that change during CRC pathogenesis. These subtle changes, in turn, could be the first step in allowing more inflammatory bacteria to gain a foothold within the colon [@flynn_metabolic_2016].

Unlike previous studies on the microbiome and CRC, ours focuses on comparing the microbiome during recovery from treatment in adenoma, advanced adenoma, and carcinoma groups. Although there were differences for genera associated with specific bacterium linked with CRC [Figure S3], most of the important OTUs taxonomically classified to commensal bacteria [Figure S3]. Although these changes may be subtle, due to the lack of significant differences in the bacterial community pre- and post-treatment in adenoma and advanced adenoma [Figure 1]; they support the hypothesis that the first members of the community to change and potentially stay changed even after treatment are those that are commensal bacteria.

Many of the common OTUs that we identified taxonomically classified to potential butyrate producers (e.g. *Clostridiales*, *Roseburia*. and *Anaerostipes*) [Table S2]. Other OTUs classified to bacteria that are inhibited by polyphenols (e.g. *Bacteroides*). Both butyrate and polyphenols are thought to be protective against cancer, in part by reducing inflammation [@louis_gut_2014]. These protective compounds are derived from the breakdown of fiber, fruits, and vegetables by resident gut microbes. One example of this potential diet-microbiome-inflammation-polyp axis is that *Bacteroides*, which was highly prevalent in our models, are known to be increased in those with high non-meat based protein consumption [@zhu_intake_2016]. High protein consumption in general has been linked with an increased CRC risk [@mu_colonic_2016]. Conversely, *Bacteroides* are inhibited by polyphenols which are derived from fruits and vegetables [@ozdal_reciprocal_2016]. Our data fits with the hypothesis that the microbial metabolites from breakdown products within our own diet could not only help to shape the existing community but also have an effect on CRC risk and disease progression. Within this context the commensal community may be an important modifiable risk factor for monitoring and preventing CRC recurrence after treatment.

A limitation in our study was that there was a significant difference in the time elapsed in the collection of the post-treatment sample between adenoma or advanced adenoma versus carcinoma (P-value `r ifelse(time_pvalues["adnVcrc", "bh"] < 0.05 & time_pvalues["srnVcrc", "bh"] < 0.05, invisible("<"), invisible(">"))` 0.05), with time passed being less for adenoma (`r paste(format(time_summary["adenoma", "mean_days"], digits = 0), label = "\u00B1", format(time_summary["adenoma", "sd_days"], digits = 0))` days) and advanced adenoma (`r paste(format(time_summary["srn", "mean_days"], digits = 0), label = "\u00B1", format(time_summary["srn", "sd_days"], digits = 0))`) than carcinoma (`r paste(format(time_summary["carcinoma", "mean_days"], digits = 0), label = "\u00B1", format(time_summary["carcinoma", "sd_days"], digits = 0))`). These results would indicate that some of the differences observed between the carcinoma and adenoma groups could be due to differences in collection time. Specifically, it could confound the observation that carcinomas changed more than adenomas [Figure 1]. However, there are two reasons that this may not be the case. First, the advanced adenoma group had a higher dissimilarity than adenoma but lower dissimilarity then carcinoma and the collection time to their follow up samples was less than the adenoma group. Second, this confounding would not affect the observations where models were used since they were built using a different cohort [Figures 2 & S2-S3].

Another limitation of this study was that it drew heavily from those with Caucasian ancestry making it possible that the observations may not be representative of those with either Asian or African ancestry. Although our training and test set are relatively large we still run the risk of over-fitting or having a model that may not be representative of other populations. We have done our best to safeguard against this by not only running 10-fold cross validation but also having over 100 different 80/20 splits to try and mimic the type of variation that might be expected to occur.

Building off of our results, an area for future research, is that within our study we do not know whether individuals who were still classified as positive by the carcinoma model eventually had a subsequent CRC recurrence. This information would help to strengthen the case for this model keeping numerous individuals above the cutoff threshold even though at follow up they were diagnosed as no longer having carcinoma. It would also provide additional evidence that the microbiome could be used as a risk stratification tool in monitoring recurrence risk and whether different interventions could potentially change this community and lower the probability of future recurrence.

Despite the stated shortcomings our findings add to the existing scientific knowledge on CRC and the microbiome: That there is a measurable difference in the bacterial community after adenoma, advanced adenoma, or carcinoma treatment. Further, the ability for machine learning algorithms to take OTU data and successfully lower positive probability of carcinoma after treatment provides evidence that there are specific signatures, attributable to both inflammatory and resident commensal organisms, associated with treatment. Our data provides evidence that commensal bacteria may be important in the development of polyps, potential transition of advanced adenoma to carcinoma, and recovery of the microbiome in CRC following treatment.



\newpage

## Methods

***Study Design and Patient Sampling:***
Sampling and design have been previously reported in Baxter, et al [@baxter_microbiota-based_2016]. Briefly, study exclusion involved those who had already undergone surgery, radiation, or chemotherapy, had colorectal cancer before a baseline fecal sample could be obtained, had IBD, a known hereditary non-polyposis colorectal cancer, or familial adenomatous polyposis. Samples used to build the models for prediction were collected either prior to a colonoscopy or between 1 - 2 weeks after. The bacterial community has been shown to normalize back to a pre-colonoscopy community within this time period [@obrien_impact_2013]. Our training cohort consisted of a total of `r length(rownames(filter(metaI, !(sample %in% c(good_metaf$initial, good_metaf$followUp)))))` individuals [Table 1]. Our study cohort consisted of `r length(rownames(good_metaf))` individuals with an initial sample as described and a follow up sample obtained between `r min(good_metaf$time)` - `r max(good_metaf$time)` days after treatment of lesion [Table 2]. This study was approved by the University of Michigan Institutional Review Board. All study participants provided informed consent and the study itself conformed to the guidelines set out by the Helsinki Declaration.


***16S rRNA Gene Sequencing:***
Sequencing was completed as described by Kozich, et al. [@kozich_development_2013]. DNA extraction used the 96-well Soil DNA isolation kit (MO BIO Laboratories) and an epMotion 5075 automated pipetting system (Eppendorf). The V4 variable region was amplified and the resulting product was split between three sequencing runs with normal, adenoma, and carcinoma evenly represented on each run. Each group was randomly assigned to avoid biases based on sample collection location. The pre- and post-treatment samples were sequenced on the same run.


***Sequence Processing:***
The mothur software package (v1.37.5) was used to process the 16S rRNA gene sequences and has been previously described [@kozich_development_2013]. The general workflow using mothur was: Paired-end reads were first merged into contigs, quality filtered, aligned to the SILVA database, screened for chimeras, classified with a naive Bayesian classifier using the Ribosomal Database Project (RDP), and clustered into Operational Taxonomic Units (OTUs) using a 97% similarity cutoff with an average neighbor clustering algorithm. The number of sequences for each sample was rarefied to `r ifelse(sd(total_sub_Seqs) == 0, paste(mean(total_sub_Seqs)), paste("Error Check Subsampling"))` to minimize uneven sampling.


***Model Building:***
The Random Forest [@breiman_random_2001] algorithm was used to create the model used to create the three models used. The adenoma model classified normal versus adenoma, advanced adenoma was normal versus advanced adenoma, and carcinoma was normal versus carcinoma. The toal number of individuals in this data set was `r length(rownames(metaI)) - length(rownames(good_metaf))` individuals. There were a total of `r length(filter(metaI, !(sample %in% c(good_metaf$initial, good_metaf$followUp)) & (Dx_Bin == "Adenoma" | dx == "normal"))[, "sample"])` individuals in the adenoma model, `r length(filter(metaI, !(sample %in% c(good_metaf$initial, good_metaf$followUp)) & (Dx_Bin == "adv Adenoma" | dx == "normal"))[, "sample"])` individuals in the advanced adenoma model, and `r length(filter(metaI, !(sample %in% c(good_metaf$initial, good_metaf$followUp)) & (Dx_Bin == "Cancer" | dx == "normal"))[, "sample"])` indivdiuals in the carcinoma model [Table 1]. Each model was then applied to our `r length(rownames(good_metaf))`-person cohort testing prediction of adenoma intial (adenoma n = `r length(filter(good_metaf, Dx_Bin == "adenoma")[, "initial"])`) versus adenoma follow up (adenoma n = `r length(filter(good_metaf, Dx_Bin == "adenoma" & Disease_Free == "n")[, "followUp"])`), advanced adenoma initial (advanced adenoma n = `r length(filter(good_metaf, Dx_Bin == "adv_adenoma")[, "initial"])`) versus advanced adenoma follow up (advanced adenoma n = `r length(filter(good_metaf, Dx_Bin == "adv_adenoma" & Disease_Free == "n")[, "followUp"])`), carcinoma initial (carinoma n = `r length(filter(good_metaf, Dx_Bin == "cancer")[, "initial"])`) versus carcinoma follow up (carcinoma n = `r length(filter(good_metaf, Dx_Bin == "cancer" & Disease_Free == "n")[, "followUp"])`).

The model included only OTU data obtained from 16S rRNA sequencing. Non-binary data was checked for near zero variance and OTUs that had near zero variance were removed. This pre-processing was performed with the R package caret (v`r paste(packageVersion("caret"))`). Optimization of the mtry hyper-parameter involved making 100 different 80/20 (train/test) splits of the data where normal and adenoma, normal and advanced adenoma, or normal and carcinoma were represented in the same proportion within both the whole data set and the 80/20 split. For each of the different splits, 20 repeated 10-fold cross validation was performed on the 80% component to optimize the mtry hyper-parameter by maximizing the AUC (Area Under the Curve of the Receiver Operator Characteristic). The resulting model was then tested on the hold out data obtained from the 20% component. All three models had an optimized mtry of `r ifelse(as.data.frame(count(adn_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"] == as.data.frame(count(srn_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"] & as.data.frame(count(adn_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"] == as.data.frame(count(crc_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"], as.data.frame(count(adn_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"], "not the same")`.

Assessment of the most important OTUs to the model involved counting the number of times an OTU was present in the top 10% of mean decrease in accuracy (MDA) for each of the 100 different splits run. This was then followed with filtering of this list to variables that were only present in more than 50% of these 100 runs. The final collated list of variables was then run through the mtry optimization again. Once the ideal mtry was found the entire sample set specific to normal versus adenoma, normal versus advanced adenoma, or normal versus carcinoma was used to create the final Random Forest model on which classifications on the `r length(rownames(good_metaf))`-person cohort was completed. For all three models the final optimized mtry was `r ifelse(as.data.frame(count(adn_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"] == as.data.frame(count(srn_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"] & as.data.frame(count(adn_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"] == as.data.frame(count(crc_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"], as.data.frame(count(adn_AUC_run_summary, best_mtry) %>% slice(which.max(n)))[, "best_mtry"], "not the same")`

The default cutoff of 0.5 was used as the threshold to classify individuals as positive or negative for lesion. The hyper-parameter, mtry, defines the number of variables to investigate at each split before a new division of the data was created with the Random Forest model.


***Statistical Analysis:***
The R software package (v`r paste(version$major, ".", version$minor, sep = "")`) was used for all statistical analysis. Comparisons between bacterial community structure utilized PERMANOVA [@anderson_permanova_2013] in the vegan package (v`r paste(packageVersion("vegan"))`). Comparisons between probabilities as well as overall OTU differences between pre- and post-treatment samples utilized a paired Wilcoxson ranked sum test. Where multiple comparison testing was appropriate, a Benjamini-Hochberg (BH) correction was applied [@benjamini_controlling_1995] and a corrected P-value of less than 0.05 was considered significant. Unless otherwise stated the P-values reported are those that were BH corrected. Model rank importance was determined by obtining the median MDA from the 100 20 repeated 10-fold cross validation and then ranking from largest to smallest MDA.


***Analysis Overview:***
We first tested whether there were any differences between pre- and post-treatment samples in alpha and beta diversity based on adenoma, SRN, or carcinoma. We then tested all OTUs for differences between pre- and post-treatment samples. We next used our specific models for adenoma, SRN, and carcinoma to test classification accuracy, response towards a normal microbiome, and common OTUs used across models. Finally, for the adenoma group differences between those that received surgery or not was tested while for the carcinoma group differences between those receiving chemotherapy and radiation was tested.


***Reproducible Methods:***
A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Sze_followUps_2017. Raw sequences have been deposited into the NCBI Sequence Read Archive (SRP062005 and SRP096978) and the necessary metadata can be found at https://www.ncbi.nlm.nih.gov/Traces/study/ and searching the respective SRA study accession.


\newpage


**Figure 1: General Differences between Adenoma, Advanced Adenoma, and Carcinoma Groups After Treatment.*. A) A significant difference was found between the adenoma and carcinoma group for thetayc (P-value = `r format(change_summary['thetayc', 'Pvalue'], scientific = FALSE, digits = 3)`). B) NMDS of the pre- and post-treatment samples for the adenoma group. C) NMDS of the pre- and post-treatment samples for the advanced adenoma group. D) NMDS of the pre- and post-treatment samples for the carcinoma group.

A: Convert mean to median w/ IQR. Add 0.00 to y-labels




**Figure 2: Treatment Response Based on Models Built for Adenoma, SRN, or Carcinoma.** A) Positive probability change from initial to follow up sample in those with adenoma. B) Positive probability change from initial to follow up sample in those with SRN. C) Positive probability change from initial to follow up sample in those with carcinoma..


**Figure 3: Common OTUs Model Rank Based on Median MDA.** For each common OTU the lowest taxonomic identification and importance rank for each model run is shown.



\newpage

**Table 1: Demographic Data of Training Cohort**

**Table 2: Demographic Data of Pre and Post Treatment Cohort**


\newpage


**Figure S1: Distribution of P-values from Paired Wilcoxson Analysis of All OTUs Before and After Treatment**

**Figure S2: ROC Curves of the Adenoma, Advanced Adenoma, and Carcinoma Models.*. A) Adenoma ROC curve: The light green shaded areas represent the range of values of a 100 different 80/20 splits of the test set data and the dark green line represents the model using 100% of the data set and what was used for subsequent classification. B) Advanced Adenoma ROC curve: The light yellow shaded areas represent the range of values of a 100 different 80/20 splits of the test set data and the dark yellow line represents the model using 100% of the data set and what was used for subsequent classification. C) Carcinoma ROC curve: The light red shaded areas represent the range of values of a 100 different 80/20 splits of the test set data and the dark red line represents the model using 100% of the data set and what was used for subsequent classification.

**Figure S3: Summary of Important Variables for the Adenoma, Advanced Adenoma, and Carcinoma Models.*. A) MDA of the most important variables in the adenoma model. The dark green point represents the mean and the lighter green points are the value of each of the 100 different runs. B) Summary of Important Variables in the advanced adenoma model. MDA of the most important variables in the SRN model. The dark yellow point represents the mean and the lighter yellow points are the value of each of the 100 different runs. C) MDA of the most important variables in the carcinoma model. The dark red point represents the mean and the lighter red points are the value of each of the 100 different runs.

\newpage

## Declarations

### Ethics approval and consent to participate

The University of Michigan Institutional Review Board approved this study, and all subjects provided informed consent. This study conformed to the guidelines of the Helsinki Declaration.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Sze_followUps_2017. Raw sequences have been deposited into the NCBI Sequence Read Archive (SRP062005 and SRP096978) and the necessary metadata can be found at https://www.ncbi.nlm.nih.gov/Traces/study/ and searching the respective SRA study accession.

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

This study was supported by funding from the National Institutes of Health to P. Schloss (R01GM099514, P30DK034933) and to the Early Detection Research Network (U01CA86400).

### Authors' contributions

All authors were involved in the conception and design of the study. MAS analyzed the data. NTB processed samples and analyzed the data. All authors interpreted the data. MAS and PDS wrote the manuscript. All authors reviewed and revised the manuscript. All authors read and approved the final manuscript.

### Acknowledgements

The authors thank the Great Lakes-New England Early Detection Research Network for providing the fecal samples that were used in this study. We would also like to thank Amanda Elmore for reviewing and correcting code error and providing feedback on manuscript drafts. We would also like to thank Nicholas Lesniak for providing feedback on manuscript drafts.

\newpage

## References
