---
title: ''
csl: mbio.csl
fontsize: 12pt
output:
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  html_document: default
geometry: margin=1.0in
bibliography: reference.bib
---

# Differences in the Stool Microbiome Before and After Colorectal Cancer Treatment


```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')
source('../code/graphFunctions.R')

loadLibs(c("dplyr", "ggplot2", "reshape2", "gridExtra", "scales", "wesanderson", "knitr", "rmarkdown"))

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

```


```{r rdata_tables}
# Extra data and meta data needed
good_metaf <- read.csv('../results/tables/mod_metadata/good_metaf_final.csv', header = T, stringsAsFactors = F)
metaI <- read.csv('../results/tables/mod_metadata/metaI_final.csv', header = T, stringsAsFactors = F)
total_sub_Seqs <- read.delim('../data/process/final.0.03.subsample.shared', header = T) %>% 
  select(contains("Otu0")) %>% rowSums()

# initial microbiome and FIT description data 
change_summary <- read.csv('../results/tables/change_theta_fit_summary.csv', header = T, row.names = 1)
alpha_summary <- read.csv('../results/tables/alpha_table_summary.csv', header = T, row.names = 1)
beta_summary <- read.csv('../results/tables/beta_diver_summary.csv', header = T, row.names = 1)
time_pvalues <- read.csv("../results/tables/time_pvalues.csv", header = T, row.names = 1)

# training of model data
model_train_summary <- read.csv('../results/tables/ROC_model_summary.csv', header = T)
model_test_AUC_summary <- read.csv('../results/tables/auc_summary.csv', header = T, row.names = 1)
imp_model_vars <- read.csv('../results/tables/rf_wCV_imp_vars_summary.csv', stringsAsFactors = F, header = T)
rf_otu_tax <- read.csv('../results/tables/rf_otu_tax.csv', stringsAsFactors = F, header = T, row.names = 1)
rf_lowest_ID <- createTaxaLabeller(rf_otu_tax)

# model prediction and adjustment performance
IF_model_wilcox_paired_summary <- read.csv('../results/tables/IF_model_wicox_paired_pvalue_summary.csv', header = T, row.names = 1)
confusion_summary <- read.csv('../results/tables/confusion_summary.csv', header = T, row.names = 1)
follow_up_prob_summary <- read.csv('../results/tables/follow_up_probability_summary.csv', header = T)
followup_cm <- read.csv('../results/tables/followup_confusion_matrix.csv', header = T, row.names = 1)
initial_cm <- read.csv('../results/tables/initial_confusion_matrix.csv', header = T, row.names = 1)
roc_pvalues <- read.csv('../results/tables/roc_pvalue_summary.csv', header = T, row.names = 1)

# Predictive of both initial and follow up and in lesion model
pvalue_common_imp_vars <- read.csv('../results/tables/pvalue_common_imp_vars.csv', header = T, row.names = 1)

# Results of CRC Bacteria
pvalue_adn_crc_summary <- read.csv('../results/tables/adn_crc_maybe_pvalue_summary.csv', header = T, row.names = 1)

```

\vspace{25mm}

\begin{center}
Running Title: Human Microbiome and Colorectal Cancer


\vspace{10mm}

Marc A Sze, Nielson T Baxter, Mack T Ruffin IV, Mary AM Rogers, and Patrick D Schloss${^\dagger}$

\vspace{10mm}

Contributions: All authors were involved in the conception and design of the study. MAS analyzed the data. NTB processed samples and analyzed the data. All authors interpreted the data. MAS and PDS wrote the manuscript. All authors reviewed and revised the manuscript. All authors read and approved the final manuscript.

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI
\end{center}


\newpage
\linenumbers

## Abstract

Colorectal cancer (CRC) continues to be a worldwide health problem with early detection being used as a key component in mitigating deaths due to the disease. Previous research suggests a link between stool bacterial microbiome and CRC.   In this study, we used a model based on the microbiome, demographics, and prior medical history to classify individuals as having a lesion (i.e. adenoma or carcinoma).  We then used this model to characterize the change in the gut microbiota before and after surgery.  The overall objective was to investigate the changes in the microbiome after surgery in patients with adenomas or carcinomas.  This model was tested on a `r sum(initial_cm$ref_yes)` person group that included samples before and after treatment to allow for the assessment of how the model adjusts risk after treatment. The model used for prediction had an AUC of `r format(model_test_AUC_summary["full", "AUC"], digits = 3)`. For the follow up samples our Random Forest model significantly decreased the positive probability of lesion compared to the initial samples for both adenoma (P-value = `r format(IF_model_wilcox_paired_summary['all_adenoma', 'BH_correction'], scientific = TRUE, digits = 3)`) and carcinoma (P-value = `r format(IF_model_wilcox_paired_summary['carcinoma_only', 'BH_correction'], scientific = TRUE, digits = 3)`).  Our model predicted that `r format((followup_cm["pred_no", "ref_no"]/(sum(followup_cm$ref_no) + sum(followup_cm$ref_yes)))*100, digits = 3)`% of the 67-person cohort had normal colons and `r format(((followup_cm["pred_yes", "ref_no"] + sum(followup_cm$ref_yes))/(sum(followup_cm$ref_no) + sum(followup_cm$ref_yes)))*100, digits = 3)`% had a lesion.  Some OTUs that changed the most before and after treatment included OTUs that were affiliated with members of `r pvalue_common_imp_vars[order(pvalue_common_imp_vars$Pvalue), "Genus"][1:2]` and `r pvalue_common_imp_vars[order(pvalue_common_imp_vars$Pvalue), "Genus"][3]`. Our model suggests that treatment does significantly reduce the probability of having a colonic lesion.  Further surveillance of these individuals will enable us to determine whether models such as the one we present here can also be used to predict recurrence of colorectal cancer.  

\newpage

### Importance

This is one of the first studies to investigate within humans what happens to the bacterial microbiome before and after adenoma and carcinoma treatment.  Specifically, it aims to assess how a random forest machine learning algorithm built model respond to treatment and adjust it's positive probability calls of whether the individual has an adenoma or carcinoma due to surgical removal of the lesion.      

\newpage

## Introduction

Colorectal cancer (CRC) continues to be a leading cause of cancer related deaths and is the second most common cancer death among men aged 40-79 years of age (**insert citation**).  Over the last few years death due to the disease has seen a significant decrease thanks mainly to improvements in screening (**insert citation**).  However, despite this giant improvement there are still approximately 50,000 deaths from the disease a year (**insert citation**).  It is estimated that around 5-10% of all CRCs can be explained by autosomal dominant inheritance (**insert citation**). The vast majority of CRCs are not inherited and the exact etiology to disease has not been well worked out (**insert citation**).  

Recent reports have found that the microbiome between those with and without CRC are different and that bacteia associated with CRC tissue is different than that found on the normal mucosa (**Insert citations**).  Further, there is evidence to suggest that more inflammatory mouth-associated bacteria are more prevalent in the guts of those with CRC (**insert citation**).  This has led to the hypothesis that these bacteria replace traditional residents that produce metabolites such as butyrate (**insert citation**) and lead to a more inflammatory state (**insert citation**).  This inflammatory state allows for more of these types of bacteria to populate the gut and leads to neoplasia and eventually CRC (**insert citation**).

Building upon these findings a number of groups, including our own, have investigated using the bacterial microbiome as a potential biomarker for adenoma and CRC detection (**insert citations**).  These collectively show that a reasonable area under the curve (AUC) can be obtained when using the stool bacterial microbiome to classify disease state (**insert citation**).  Further, a number of groups have extended this observation to show that using either specific bacterial species or the bacterial microbiome as a whole paired with the Fecal Immunochemical Test (FIT) can increase the AUC of the model (**insert citations**).

In this study we further refine the random forest model by including more background information on the individuals within it. We used a rigourous and accepted approach for machine learning training and testing to validate this model (**insert citation**).  We then applied it to individuals in which we have samples before and after treatment to assess how well the model adjusts their probability of having an adenoma or carcinoma.  Finally, we investigated which specific Operational Taxonomic Units (OTUs) are most affected by treatment.  Within this study lesion refers to both adenoma and carcinoma. 


\newpage

## Results

***Bacterial Community and Fit Changes before and after Treatment***
Based on thetayc distance metrics, comparing the initial to the follow up samples, there was no difference between the adenoma and carcinoma groups (P-value = `r format(change_summary['thetayc', 'Pvalue'], scientific = FALSE, digits = 3)`) [Figure 1a]. There was a difference in FIT between initial and follow up samples with the carcinoma group having a significant decrease in FIT versus the adenoma group (P-value = `r format(change_summary['fit', 'Pvalue'], scientific = TRUE, digits = 3)`) [Figure 1b].  Although the thetayc distance metric change was similar between adenoma and carcinoma the directionality of the change was significant in the cancer group between initial and follow up (P-value = `r format(beta_summary['Carcinoma', 'PERMANOVA'], scientific = FALSE, digits = 3)`) but not for the adenoma group (P-value = `r format(beta_summary['Adenoma', 'PERMANOVA'], scientific = FALSE, digits = 3)`).  This change can be visualized using an NMDS [Figure 2]. When all follow up samples were compared to each other there was no significant overall difference between them (P-value = `r format(beta_summary['All', 'PERMANOVA'], scientific = FALSE, digits = 3)`).  There was no significant difference between initial and follow up samples for observed OTUs, Shannon diversity, or evenness after correction for multiple comparisons [Table S1].  Time of follow up sample from initial sampling, did not have a significant difference between adenoma and carcinoma (uncorrected P-value = `r format(time_pvalues["thetayc_change", "uncorrected_p_value"], digits = 3)`).    


***Outcome of Model Training***
The range of the AUC for model training ranged from a minimum of `r format(model_test_AUC_summary["worse", "ROC_cv"], digits = 3)` to a maximum of `r format(model_test_AUC_summary["best", "ROC_cv"], digits = 3)` with the middle of all `r length(rownames(model_train_summary))` runs having an AUC of `r format(model_test_AUC_summary["middle", "ROC_cv"], 
digits = 3)`.  Interestingly, the worst AUC model from training performed the best on it's respective 80/20 split test data [Figure 3].  In fact the 80/20 test performance showed that the AUC for the middle model chosen was the most stable (best training model test set AUC = `r format(model_test_AUC_summary["best", "AUC"], digits = 3)`, middle training model test set AUC = `r format(model_test_AUC_summary["middle", "AUC"], digits = 3)`, worse training model test set AUC = `r format(model_test_AUC_summary["worse", "AUC"], digits = 3)`).  That is to say it had the smallest change in AUC in comparison to the minimum and maximum AUC trained models.  The middle model was close to the full training data AUC `r format(model_test_AUC_summary["full", "AUC"], digits = 3)`. There was no significant difference between the AUC of the best and middle training models (P-value = `r format(roc_pvalues["best", "middle"], digits = 3)`).  There was also no difference in the middle model versus worse (P-value = `r format(roc_pvalues["middle", "worse"], digits = 3)`) or full data model (P-value = `r format(roc_pvalues["middle", "full"], digits = 3)`).  The two comparisons with a significant difference were between the worse training model and the best training model (P-value = `r format(roc_pvalues["best", "worse"], scientific = TRUE, digits = 3)`) and the worse training model and full training model (P-value = `r format(roc_pvalues["worse", "full"], scientific = TRUE, digits = 3)`).  


***Most Important Variables to the Model***
Overall, there were a total of `r format(length(rownames(imp_model_vars)))` variables identified as being present in more than 50% of the training models [Table S2]. The top 5 most important bacterial OTUs were `r paste(rf_lowest_ID[1], " (", names(rf_lowest_ID)[1], ")", sep = "")`,  `r paste(rf_lowest_ID[2], " (", names(rf_lowest_ID)[2], ")", sep = "")`, `r paste(rf_lowest_ID[3], " (", names(rf_lowest_ID)[3], ")", sep = "")`, `r paste(rf_lowest_ID[4], " (", names(rf_lowest_ID)[4], ")", sep = "")`, and `r paste(rf_lowest_ID[5], " (", names(rf_lowest_ID)[5], ")", sep = "")`.  These 5 OTUs were present in at least `r min(filter(imp_model_vars, Variable %in% names(rf_lowest_ID)[1:5])[, "total_appearance"])` out of the total `r length(rownames(model_train_summary))` different 80/20 runs.


***Surgical Removal of an Adenoma or Carcinoma Results in a Decrease in Positive Probability Prediction***
A total of `r length(rownames(good_metaf)) - length(rownames(filter(good_metaf, !is.na(fit_followUp))))` sample was omitted from the original `r length(rownames(good_metaf))` test sample set since it was missing a complete set of follow up data.  This left a total of `r sum(initial_cm$ref_yes)` samples for test predictions.  After multiple comparison correction there was a significant overall decrease in positive probability of a carcinoma and adenoma (P-value = `r format(IF_model_wilcox_paired_summary["lesion", "BH_correction"], scientific = TRUE, digits = 3)`) [Figure 4].  This decrease was significant for both adenoma (P-value = `r format(IF_model_wilcox_paired_summary["all_adenoma", "BH_correction"], scientific = TRUE, digits = 3)` [Figure 4a] and carcinoma (P-value = `r format(IF_model_wilcox_paired_summary["carcinoma_only", "BH_correction"], scientific = TRUE, digits = 3)`) [Figure 4b] alone.  This decrease in probability of lesion also held specifically for those with screen relative neoplasias (SRN) (P-value = `r format(IF_model_wilcox_paired_summary["SRN_only", "BH_correction"], scientific = TRUE, digits = 3)`).  A total of `r initial_cm["pred_yes", "ref_yes"]` or `r format(confusion_summary["Accuracy", "initial"]*100, digits = 3)`% of all samples were correctly predicted to have a lesion.  Although there was a decrease in positive probability only `r followup_cm["pred_no", "ref_no"]` of the total `r sum(initial_cm$ref_yes)` individuals were classified as adenoma or carcinoma free on follow up (successful classification of `r format(confusion_summary["Accuracy", "followup"]*100, digits = 3)`%). There was no significant difference between the predictions and actual diagnosis for the initial samples in the `r length(rownames(good_metaf))`-sample cohort test set.  However, the predictions were significantly discordent with the diagnosis for the follow up samples (P-value = `r format(confusion_summary["McnemarPValue", "followup"], scientific = TRUE, digits = 3)`).  Although there were discordent results the respective sensitivity for the initial group was `r format(confusion_summary["Sensitivity", "initial"]*100, digits = 3)`% and for follow up was `r format(confusion_summary["Sensitivity", "followup"]*100, digits = 3)`%, respectively.

There was `r length(rownames(filter(good_metaf, Disease_Free == "n")))` individual who still clearly had CRC on follow up as well as `r length(rownames(filter(good_metaf, Disease_Free == "unknown")))` individuals whose status on follow up was unknown.  Although the `r length(rownames(filter(good_metaf, Disease_Free == "n")))` individual had a decrease in positive probability their follow up sample was still higher than the cutoff threshold of 0.5 (positive probability = `r format(filter(follow_up_prob_summary, sampleType == "followup" & disease_free == "n")[, "Yes"], digits = 3)`).  Interestingly, `r length(rownames(filter(follow_up_prob_summary, sampleType == "followup" & disease_free == "unknown" & Yes > 0.5)))` individuals who were unknown on follow up still were over the threshold cutoff of 0.5 even though, like the `r length(rownames(filter(good_metaf, Disease_Free == "n")))` individual with clear CRC on follow up, the probability of an adenoma or carcinoma decreased [Table S3]. 

The follow up positive probabilities were not affected by either chemotherapy treatment (uncorrected P-value = `r format(wilcox.test(filter(follow_up_prob_summary, sampleType == "followup", chemo == "yes")[, "Yes"], filter(follow_up_prob_summary, sampleType == "followup", chemo == "no")[, "Yes"])$p.value, digits = 3)`) or  radiation therapy (uncorrected P-value = `r format(wilcox.test(filter(follow_up_prob_summary, sampleType == "followup", rads == "yes")[, "Yes"], filter(follow_up_prob_summary, sampleType == "followup", rads == "no")[, "Yes"])$p.value, digits = 3)`).  There was also no difference in the amount of change in the positive probability based on whether individuals received chemotherapy (unccorected P-value = `r format(wilcox.test(filter(follow_up_prob_summary, sampleType == "initial", chemo == "yes")[, "Yes"] - filter(follow_up_prob_summary, sampleType == "followup", chemo == "yes")[, "Yes"], filter(follow_up_prob_summary, sampleType == "initial", chemo == "no")[, "Yes"] - filter(follow_up_prob_summary, sampleType == "followup", chemo == "no")[, "Yes"])$p.value, digits =3)`) or radiation therapy (uncorrected P-value = `r format(wilcox.test(filter(follow_up_prob_summary, sampleType == "initial", rads == "yes")[, "Yes"] - filter(follow_up_prob_summary, sampleType == "followup", rads == "yes")[, "Yes"], filter(follow_up_prob_summary, sampleType == "initial", rads == "no")[, "Yes"] - filter(follow_up_prob_summary, sampleType == "followup", rads == "no")[, "Yes"])$p.value, digits = 3)`).


***Specific OTUs in the Lesion Model are not Detected in Follow Up Versus Initial Samples*** 
Overall, there were a total of `r length(rownames(pvalue_common_imp_vars))` OTUs that were common between the main lesion model and the model for classifying initial and follow up samples specifically [Table S4].  A total of `r length(rownames(filter(pvalue_common_imp_vars, BH_corr < 0.05)))` OTU was still significant after multiple comparison correction and it's lowest taxonomic identification was to `r filter(pvalue_common_imp_vars, BH_corr < 0.05)[, "Genus"]`.  In general, `r filter(pvalue_common_imp_vars, BH_corr < 0.05)[, "otu"]` (`r filter(pvalue_common_imp_vars, BH_corr < 0.05)[, "Genus"]`) was decreased from initial to follow up [Figure 5].  The relative abundance was not drastically different then the mean of the values observed in the control training set [Figure 5].


***Differences in Adenoma and Carcinoma in Previously Associated Cancer Bacteria***
First, there was a clear magnitude difference in these specific OTUs based on whether they were from adenoma or carcinoma individuals [Figure 6].  The carcinoma samples showed a significant difference between initial and follow up samples for Peptostreptococcus stomatis (P-value = `r format(pvalue_adn_crc_summary["pept", "crc_BH"], digits = 3)`) and Porphyromonas asaccharolytica (P-value = `r format(pvalue_adn_crc_summary["porp", "crc_BH"], digits = 3)`) whereas there were no signficant differences in any of these OTUs in the adenoma samples [Table S5].



\newpage


## Discussion
In our training set we show that the overall community structure as measured by different alpha diversity metrics, shows very little change between controls and those with either adenoma or carcinoma [Table S1].  With respect to our test set there was very little difference in magnitude of change in the thetayc distance metric between those with adenoma or carcinoma [Figure 1a].  In contrast, FIT had a large change in the initial and follow up samples in the carcinoma group versus the adenoma [Figure 1b].  An NMDS showed that there was very little observable change between initial and follow up for the adenoma group but there was one for the carcinoma group [Figure 2].  This cursory information is suggestive that treatment of carcinoma, had the largest response.

We next created a model that incorporated both patient metadata, FIT, and the bacterial microbiome to be able to predict lesions (adenoma or carcinoma).  Our middle training model, based on AUC, from 100 80/20 (train/test) splits was similar to the full training data model.  It's 10-fold cross validated AUC was similar to it's test set AUC which was not the case for both the best and worse training model [Figure 3].  Using the full training data model we predicted the probability of a lesion in the inital and follow up samples [Figure 4].  There was a signficant decrease in positive probability regardless of whether the sample was a carcinoma or adenoma.  The overall sensitivity for lesion detection in the intial samples was `r format(confusion_summary["Sensitivity", "initial"]*100, digits = 3)` and for follow ups was `r format(confusion_summary["Sensitivity", "followup"]*100, digits = 3)`.  Although there was a decrease in overall probability of an adenoma or carcinoma only `r followup_cm["pred_no", "ref_no"]` were below the 0.5 threshold out of the total `r sum(followup_cm$ref_no)` individuals who were diagnosed as not having a carcinoma on follow up.  

We then investigated which OTUs could potentially be more important in our model [Figure 5 & Table S4].  Many of the OTUs identified classified to normal flora bacterium [Table S4].  Only a single OTU though was significant after multiple comparison correction and the lowest taxonomic identification of `r filter(pvalue_common_imp_vars, BH_corr < 0.05)[, "otu"]` was to `r filter(pvalue_common_imp_vars, BH_corr < 0.05)[, "Genus"]`. Although there was a difference in the relative abundance at initial and follow up these values were not drastically different from the relative abundance values observed in the control individuals of the training set [Figure 5].  Although we were interested in what we could use to classify those with either adenoma or carcinoma versus normal.  We found that the traditional bacteria associated with CRC were higher in magnitude in the carcinoma group and there were significant differences in some of these OTUs between the initial and follow up samples [Figure 5 and Tabl S5].  This research provides evidence that it is possible to use bacterial microbiome data to create a highily sensitive model, that is reactive to therapy, for detection of adenoma or carcinoma.  It accomplishes this by using a unique sample set in which before and after surgery stool samples are available for assessment.  By using these types of samples we are not only able to show sensitivity of lesion prediction but also able to show that this model is reactive.  That is to say that after surgery for removal of the adenoma or carcinoma it decreases the positive probability to reflect a lower likelihood of the individual having an adenoma or carcinoma.

This study builds upon previous work from numerous labs that have looked into the bacterial microbiome as a potential screening tool (**insert citation**).  Based on previous work by Jobin, et al. (**insert citation**)  it may not be surprising to see E.coli in the top 5 OTUs for this model.  Similarily, Porphyromonas has also been implicated in colorectal cancer (**insert citation**).  Interestingly, many of the other OTUs had taxonomic identification for resident gut microbes.  This could suggest that changes to the resident microbiome are important to the initiation of adenoma or carcinoma formation (**insert citation**) and provide support for the hypothesis that an initial change in the bacterial microbiome could pave the way for more inflammatory species: whether by creation of a new niche for oral microbes (**insert citation**) or allowing for a bloom of existing pro-inflammatory residents (**insert citation**).

Naturally, it is curious that normal staples of many screening studies such as Fusobacterium, Parvimonas, and Peptostreptococcus were not present in the majority of the training models.  One potential explanation for this is that FIT provides the same information to the model as these three organisms and so the model uses FIT preferentially over them.  This has been suggested to be the case in a previous study (**insert Baxter Study**).  It is also possible that these specific bacteria play a major role in the progression to carcinoma but may not be as important in the initiation of an adenoma, which would be supported by our data [Figure 5].  Regardless, our study does not argue against the importance of these bacterium in CRC initiation or pathogenesis but rather that the model does not utilize these specific bacteria for prediction purposes.  Another potential reason why we did not identify the "usual suspects"" is that these bacteria may not change much between initial and follow up samples in those with an identified lesion.  That is to say that the bacteria are consistently present even after removal of the lesion by surgery.  Finally, it is likely that within our test set there was not enough indviduals in which detection  was made or relative abundance high enough for these bacteria to be significant using a paired wilcoxson test.   

One limitation in this study is that we do not know whether individuals in our test set eventually had a subsequent CRC diagnosis.  This information would help to strengthen the case for our Random Forest based model keeping a number of individuals above the cutoff threshold even though at follow up they were diagnosed as no longer having a lesion.  Another limitation is that we do not know if adding modern tests such as the stool DNA test (**insert citation**) could help improve our overall AUC.  Another limitation is that this study drew heavily from those with caucasian ancestry.  The results may not be immediately representative of those with either Asian or African ancestry.  Finally, although our training and test set are relatively large we still run the risk of overfitting or having a model that may not be immediately extrapolateable to other populations.  We've done our best to safeguard against this by not only running 10-fold cross validation but also having over 100 different 80/20 splits to try and mimic the type of variation that might be expected to occur.

By adding patient data such as age, BMI, etc. to the model and showing that it can successfully help to predict both carcinoma and adenoma our study provides further data that these patient factors in conjunction with the bacterial microbiome could potentially influence CRC and perhaps have a role in formation of adenomas.  Further studies need to be carried out to verify our findings since not only are we dealing with stool, which could be very different than the communities present on the actual tissue, but also are dealing with correlations that may not be representative of the true pathogensis of disease.

Despite these limitations we think that these findings significantly add to the existing scientific knowledge on CRC and the bacterial microbiome.  The ability for machine learning algorithms to take bacterial microbiome data and successfully lower positive probability after either adenoma or carcinoma removal provides evidence that there are specific signatures associated with these lesions.  It also shows that these algorithms can not only successfully react to successful treatment regimens but also may be able to one day diagnose CRC with a high level of accuracy.



\newpage

## Methods

***Study Design and Patient Sampling***
The sampling and design of the study was similar to that reported in Baxter, et al (**insert citation**).  In brief, study exclusion involved those who had already undergone surgery, radiation, or chemotherapy, had colorectal cancer before a baseline stool sample could be obtained, had IBD, a known hereditary non-polyposis colorectal cancer, or Familial adenomatous polyposis.  Samples used to build the model used for prediction were collected either prior to a colonoscopy or between 1 - 2 weeks after.  The bacterial microbiome has been shown to nomralize within this time period (**insert citation**).  Kept apart from this training set were a total of `r length(rownames(good_metaf))` individuals that not only had a sample as described previoulsy but also a follow up sample between `r min(good_metaf$time)` - `r max(good_metaf$time)` days after surgery and treatment had been completed. This study was approved by the University of Michigan Institutional Review Board.  All study participants provided informed consent and the study itself conformed to the guidelines set out by the Helsinki Declaration.


***Fecal Immunochemical Test and 16S rRNA Gene Sequencing***
FIT was analyzed as previously published using both OC FIT-CHEK and OC-Auto Micro 80 automated system (Polymedco Inc.) (**insert citation**). 16S rRNA gene sequencing was completed as previously described by Kozich, et al. (**insert citation**).  In brief, DNA extraction used the 96 well Soil DNA isolation kit (MO BIO Laboratories) and an epMotion 5075 automated pipetting system (Eppendorf).  The V4 variable region was amplified and the resulting product was split between three sequencing runs with control, adenoma, and carcinoma evenly represented on each run.  Each group was randomly assigned to avoid biases based on sample collection location.


***Sequence Processing***
The mothur software package (v1.37.5) was used to process the 16S rRNA gene sequences.  This process has been previously described (**insert citations**).  The general processing workflow using mothur is as follows: Paired-end reads were first merged into contigs, quality filtered, aligned to the SILVA database, screening for chimeras, classified with a naive Bayesian classifier using the Ribosomal Database Project (RDP), and clustered into Operational Taxonomic Units (OTUs) using a 97% similarity cutoff with an average neighbor clustering algorithm.  The numer of sequences for each sample was rarified to `r ifelse(sd(total_sub_Seqs) == 0, paste(mean(total_sub_Seqs)), paste("Error Check Subsampling"))` in an attempt to minimize uneven sampling.    


***Model Creation***
The Random Forest (**insert citation**) algorithm was used to create the model used for prediction of lesion (adenoma or carcinoma) for the `r length(rownames(good_metaf))` individuals with follow up samples.  The model included data on FIT, the bacterial microbiome, sex, age, Body Mass Index (BMI), whether the indivdiual was caucasion or not, history of cancer, and family history of cancer. Non-binary data was checked for near zero variance and auto correlation.  Data columns that had near zero variance were removed.  Columns that were correlated with each other over a Spearman correlation coefficient of 0.75 had one of the two columns removed.  This pre-processing was performed with the R package caret (v`r paste(packageVersion("caret"))`).  Optimiation of the mtry hyperparameter as well as data on the best, middle, and worse performance of the model involved taking the `r length(rownames(metaI))` samples and making 100 80/20 (train/test) splits in the data where control and lesion were equally represented in the 80 and 20 split, respectively.  This 80% portion was then split again into an 80/20 split, and run through 20 repeated 10-fold cross validations to optimzie the model's AUC (Area Under the Curve of the Receiver Operator Characteristic).  This resulting model was then tested on the 20% of the data that was originally held out from this overall process.  Once the ideal mtry was found the entire `r length(rownames(metaI))` sample set was used to create the final Random Forest model on which testing on the `r length(rownames(good_metaf))`-person cohort was completed.  The default cutoff of 0.5 was used as the threshold to classify individuals as positive or negative for lesion.  


***Selection of Important OTUs***
In order to assess which variables were most central to all the models we counted the number of times a variable was present in the top 10% of mean decrease in accuracy (MDA) for each different 80/20 split model and then filtered this list to variables that were only present more than 50% of the time.  This final collated list of variables was what was considered the most important for the lesion prediction model.

A second model using the same types of conditions was used to identify OTUs that could predict initial and follow up samples.  These OTUs were considered the most important for predicting initial and follow up samples.  This new model important OTUs were then compared to the overall lesion model important OTUs and the resulting common OTUs to both models were selected to test for changes between initial and follow up using a paired wilcoxson rank sum test.   


***Investigation of Important CRC Bacteria***
In order to investigate how specific bacteria that have been previously associated with CRC with selected OTUs that taxonomically classified to Fusobacterium Nucleatum, Parvimonas Micra, Peptostreptococcus Assacharolytica, and Porphyromonas Stomatis.  Specifically we wanted to observe if there were any differences based on whether the individual had an adenoma or carcinoma that may have been missed with our lesion versus normal model.  


***Statistical Analysis***
The R software package (v`r paste(version$major, ".", version$minor, sep = "")`) was used for all statisitical analysis.  Comparisons between bacterial community structure utilized PERMANOVA (**insert citation**) in the vegan package (v`r paste(packageVersion("vegan"))`) while comparisons between ROC curves utilized the method by DeLong et al. (**insert citation**) executed by the pROC (v`r paste(packageVersion("pROC"))`) package.  Comparisons between probabilities as well as overall amount of OTU between initial and follow up samples utilized a paired wilcoxson ranked sum test.  Where multiple comparison testing was needed a Benjamini-Hochberg (BH) correction was applied (**insert citation**) and a corrected P-value of less than 0.05 was considered significant.  Unless otherwise stated the P-values reported are those of the BH corrected ones.      


***Reproducible methods.***
A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Baxter_followUps_2016.

\newpage

## Acknowledgements

The authors thank the Great Lakes-New England Early Detection Research Network for providing the fecal samples that were used in this study. This study was supported by funding from the National Institutes of Health to P. Schloss (R01GM099514, P30DK034933) and to the Early Detection Research Network (U01CA86400).


\newpage


**Figure 1: Change in Thetayc and Fit between initial and follow up in adenoma or carcinoma group.**  A) No significant difference was found between the adenoma and carcinoma group for thetayc (P-value = `r format(change_summary['thetayc', 'Pvalue'], scientific = FALSE, digits = 3)`).  B) A significant difference was found between the adenoma and carcinoma group for FIT (P-value = `r format(change_summary['fit', 'Pvalue'], scientific = TRUE, digits = 3)`).


**Figure 2: NMDS of the Overall Bacterial Community Changes.**  A) NMDS of the intial and follow up samples for the Adenoma group.  B) NMDS of the initial and follow up samples for the Carcinoma group.


**Figure 3: Graph of the Receiver Operating Characteristic Curve on Test Set Performance of the Best, Middle, Worse, and Full Training Models.** For each of the 100 training cohort sets used had `r round(length(rownames(metaI))*0.8)` individuals and the testing cohort sets had `r round(length(rownames(good_metaf))*0.2)` individuals.  The AUC on the test sets for the best, middle, and worse models from training were `r format(model_test_AUC_summary["best", "AUC"], digits = 3)`, `r format(model_test_AUC_summary["middle", "AUC"], digits = 3)`, and `r format(model_test_AUC_summary["worse", "AUC"], digits = 3)`, respectively.  cvAUC is the 20 times repeated 10-fold cross-validated AUC from training. 


**Figure 4: Breakdown by Carcinoma and Adenoma of Prediction Results for Initial and Follow Up***  A) Positive probability adjustment of those with carcinoma from intial to follow up sample  B) Positive probability adjustment of those with adenoma as well as those with SRN and the probability adjustment from initial to follow up sample.  The dotted line represents the threshold used to make the decision of whether a sample was lesion positive or not.


**Figure 5: Lesion Model OTU with a Significant Decrease in Relative Abundance that is also Predictive of Initial and Follow Up.**  After multiple comparison correction `r rownames(filter(pvalue_common_imp_vars, BH_corr < 0.05))` (`r filter(pvalue_common_imp_vars, BH_corr < 0.05)[, "Genus"]`) was the only one with a P-value < 0.05.  The dotted line represents the average relative abundance in the control training group.     


**Figure 6: Previously Associated CRC Bacteria in Initial and Follow up Samples.**  A) Carcinoma initial and follow up samples. There was a significant difference in initial and follow up sample for the OTUs classfied as Peptostreptococcus stomatis (P-value = `r format(pvalue_adn_crc_summary["pept", "crc_BH"], digits = 3)`) and Porphyromonas asaccharolytica (P-value = `r format(pvalue_adn_crc_summary["porp", "crc_BH"], digits = 3)`).  B) Adenoma initial and follow up samples.  There were no significant differences between initial and follow up.


\newpage

**Figure S1: Thetayc Graphed Against Time of Follow up Sample from Initial**

\newpage


## References




