---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  word_document: default
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
geometry: margin=1.0in
bibliography: reference.bib
---

# The Fecal Microbiome Before and After Treatment for Colorectal Adenoma or Carcinoma
```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("dplyr", "reshape2", "scales", "knitr", "rmarkdown"))

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

```

```{r rdata_tables}
# Extra data and meta data needed
good_metaf <- read.csv('../data/process/mod_metadata/good_metaf_final.csv', header = T, stringsAsFactors = F)
metaI <- read.csv('../data/process/mod_metadata/metaI_final.csv', header = T, stringsAsFactors = F)
total_sub_Seqs <- read.delim('../data/process/final.0.03.subsample.shared', header = T) %>% 
  select(contains("Otu0")) %>% rowSums()

# initial microbiome and FIT description data 
change_summary <- read.csv('../data/process/tables/change_theta_fit_summary.csv', header = T, row.names = 1)
alpha_summary <- read.csv('../data/process/tables/alpha_table_summary.csv', header = T, row.names = 1)
beta_summary <- read.csv('../data/process/tables/beta_diver_summary.csv', header = T, row.names = 1)
time_pvalues <- read.csv("../data/process/tables/time_pvalues.csv", header = T, row.names = 1)
time_summary <- read.csv("../data/process/tables/time_summary_data.csv", header = T, row.names = 1)
all_OTU_pair_wilcox <- read.csv("../data/process/tables/OTU_paired_wilcoxson_test.csv", header = T)

# Main CRC bacteria differences between Adenoma and Carcinoma
main_crc_bugs_summary <- read.csv('../data/process/tables/adn_crc_maybe_pvalue_summary.csv', header = T, row.names = 1)
crc_bug_info <- read.csv("../data/process/tables/adn_crc_maybe_diff.csv", header = T, stringsAsFactors = F)

# Lesion Model 
lesion_top10_varsMDA <- read.csv('../data/process/tables/reduced_lesion_model_top_vars_MDA_Summary.csv', header = T)
lesion_imp_vars <- read.csv('../data/process/tables/rf_otu_tax.csv', header = T, row.names = 1)
lesion_AUC_run_summary <- read.csv('../data/process/tables/Reduced_ROC_model_summary.csv', header = T)

# IF Model
IF_top10_varsMDA <- read.csv('../data/process/tables/reduced_IF_model_top_vars_MDA_Summary.csv', header = T)
IF_imp_vars <- read.csv('../data/process/tables/if_rf_otu_tax.csv', header = T, row.names = 1)
IF_AUC_run_summary <- read.csv('../data/process/tables/reduced_IF_ROC_model_summary.csv', header = T)

# All model information
all_probs_paired_wilcox <- read.csv('../data/process/tables/all_models_wilcox_paired_pvalue_summary.csv', header = T, stringsAsFactors = F)
all_pred_stat_summary <- read.csv('../data/process/tables/all_models_summary_info.csv', header = T, stringsAsFactors = F)
all_confu_matrix <- read.csv('../data/process/tables/all_models_confusion_summary.csv', header = T, stringsAsFactors = F)

# Common Variable Info
common_vars_summary <- read.csv('../data/process/tables/pvalue_IF_lesion_common_imp_vars.csv', header = T, row.names = 1, stringsAsFactors = F)

#Therapy Info
chemo_rad_summary <- read.csv('../data/process/tables/probs_chemo_rad_pvalue_summary.csv', header = T, row.names = 1)
chemo_rad_common_vars_summary <- read.csv('../data/process/tables/chemo_rads_treatment_pvalue_summary.csv', header = T, stringsAsFactors = F)

```
\vspace{25mm} 

\begin{center}
Running Title: Human Microbiome before and after Colorectal Cancer

\vspace{10mm}

Marc A Sze${^1}$, Nielson T Baxter${^2}$, Mack T Ruffin IV${^3}$, Mary AM Rogers${^2}$, and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI

$2$ Department of Internal Medicine, University of Michigan, Ann Arbor, MI   

$3$ Department of Family Medicine and Community Medicine, Penn State Hershey Medical Center, Hershey, PA    


\end{center}  


\newpage
\linenumbers

## Abstract

**Background:** Colorectal cancer (CRC) is a worldwide health problem and research suggests a correlation between the fecal bacterial microbiome and CRC.  This study tested the hypothesis that treatment for adenoma or carcinoma result in changes to the bacterial community.  Specifically, we tried to identify components within the community that were different before and after removal of lesion (adenoma or carcinoma).     

**Results:** There was a larger change in the bacterial community in response to lesion removal for carcinoma versus adenoma cases (P-value `r ifelse(change_summary['thetayc', 'Pvalue'] < 0.05, paste("<"), paste(">"))` 0.05).  Yet no difference was found in the relative abundance of any OTU before and after treatment for adenoma or carcinoma groups (P-value `r ifelse(min(filter(all_OTU_pair_wilcox, analysis == "ALL")[, 'BH_corrected']) < 0.05, paste("<"), paste(">"))` 0.05).  A lesion model had an AUC range of `r format(min(lesion_AUC_run_summary$ROC), digits = 3)` - `r format(max(lesion_AUC_run_summary$ROC), digits = 3)` and follow up samples decreased in the positive probability of lesion versus initial samples (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type == "red_lesion" & comparison =="lesion")[, "BH_correction"] < 0.05, paste("<"), paste(">"))` 0.05); suggesting a movement towards a normal bacterial community.  An initial sample model had an AUC range of `r format(min(IF_AUC_run_summary$ROC), digits = 3)` - `r format(max(IF_AUC_run_summary$ROC), digits = 3)` and had a decrease positive probability for the follow up samples to be an initial sample (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type == "red_IF" & comparison =="lesion")[, "BH_correction"] < 0.05, paste("<"), paste(">"))` 0.05).  The lesion model used a total of `r length(lesion_top10_varsMDA$variable)` variables while the initial sample model used a total of `r length(IF_top10_varsMDA$variable)` variables.  A total of `r length(common_vars_summary$otu)` OTUs were common to both models with the majority of these classifying to commensal bacteria (e.g. *`r filter(common_vars_summary, otu == "Otu000028")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000018")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000009")[, "lowest_ID"]`*, and *`r filter(common_vars_summary, otu == "Otu000005")[, "lowest_ID"]`*).   
  
**Conclusions:** Our data supports the hypothesis that the bacterial community changes after treatment.  Individuals with carcinoma have more drastic differences to the overall community then those with adenoma.  Commensal bacteria were crucial for accurate model classification, suggesting that these bacteria may be important to initial polyp formation and transition to carcinoma.  

\newpage

### Keywords

bacterial microbiome; colorectal cancer; polyps; FIT; post-surgery; risk factors      

\newpage

## Background

Colorectal cancer (CRC) is currently the third most common cause of cancer deaths [@jemal_cancer_2010; @haggar_colorectal_2009].  The rate of disease mortality has seen a significant decrease, thanks mainly to improvements in screening [@jemal_cancer_2010].  However, despite this improvement there are still approximately 50,000 deaths from the disease per year [@haggar_colorectal_2009].  

Recent studies in humans have shown that both the bacterial community and specific members within it correlate with CRC pathogenesis [@zackular_manipulation_2016; @arthur_microbial_2014].  Further, Dejea, et al. observed that bacterial communities are altered between normal and tumor tissue [@dejea_microbiota_2014].  Mouse models of CRC have further demonstrated the importance of the microbiome, both on a community [@zackular_gut_2013; @zackular_manipulation_2016] and species level [@arthur_microbial_2014], for tumorgenesis.  Collectively, these studies provide a tantalizing link between our gut bacteria and CRC and suggest that biomarkers using our microbes could be developed.  Indeed, builidng models using 16S rRNA gene sequencing along with clinical tests such as Fecal Immunoglobulin Test (FIT) result in good predictions of CRC [@baxter_microbiota-based_2016; @zeller_potential_2014].  Although these studies show how our gut bacteria can impact CRC progression via a changed community or invations by more inflammatory bacteria [@flynn_metabolic_2016].  They provide very little information as to whether these changed communities rebound after successful treatment of lesion (adenoma or carcinoma).

In this study we tested the hypothesis that there are detectable changes to the bacterial community between pre- (initial) and post- (follow up) treatment of lesion.  We analyzed changes in alpha and beta diversity as well as the relative abundance of specific Operational Taxonomic Units (OTUs).  We then investigated several important genera from oral microbes that have been suggested to be important in CRC pathogenesis [@flynn_metabolic_2016].  We next utilized Random Forest to build two models: The first was built to classify lesion versus non-lesion (normal) while the second was built to classify initial versus follow up samples.  Subsequent observations on how these models and OTUs within them performed before and after treatment helped inform us as to whether initial and follow up samples were changing and whether it was towards a normal community.  We also investigated the two models for similar important OTUs to identify the crucial OTUs involved with both classifying lesion or normal and initial versus follow up samples.  Finally, since treatment also included chemotherapy and radiation therapy along with lesion removal, we tested if these additionaly treatments significantly lowered any of the metrics examined versus those who received only lesion removal.  This study will provide evidence as to whether treatment can influence the community and if the CRC microbiome, identified in previous studies, persists.



\newpage

## Results

***The Bacterial Community:***
Within our `r length(rownames(good_metaf))`-person cohort we tested whether those with adenoma (n = `r length(rownames(filter(good_metaf, Diagnosis == "adenoma")))`) or carcinoma (n = `r length(rownames(filter(good_metaf, Diagnosis != "adenoma")))`) had any broad differences between their initial and follow up samples.  We found that those with carcinoma had a more dissimilar bacterial community between their initial and follow up sample than those with adenoma (P-value `r ifelse(change_summary['thetayc', 'Pvalue'] < 0.001, paste("<"), paste(">"))` 0.001) [Figure 1A].  The bacterial community structure before and after surgery was visualized using NMDS for both adenoma [Figure 1B] (PERMANOVA `r ifelse(beta_summary['Adenoma', 'PERMANOVA'] < 0.05, paste("<"), paste(">"))` 0.05) and carcinoma [Figure 1C] (PERMANOVA `r ifelse(beta_summary['Carcinoma', 'PERMANOVA'] < 0.05, paste("<"), paste(">"))` 0.05).  Interestingly, when initial and follow up samples were compared, regardless of whether the lesions were adenoma or carcinoma, there was no significant overall difference in beta diversity (PERMANOVA `r ifelse(beta_summary['All', 'PERMANOVA'] < 0.05, paste("<"), paste(">"))` 0.05).  There was no difference between initial and follow up samples when investigating alpha diversity metrics for lesion, adenoma only, or carcinoma only for any metric tested [Table S1].  Additionally, there was also no difference in the relative abundance of any OTU between initial and follow up samples for lesion, adenoma only, or carcinoma only [Figure S1].


***Carcinoma Associated Genera:*** 
For carcinoma [Figure S2A] (P-value `r ifelse(main_crc_bugs_summary["parv", "crc_BH"] < 0.05 & main_crc_bugs_summary["porp", "crc_BH"] < 0.05 & main_crc_bugs_summary["pept", "crc_BH"] < 0.05, paste("<"), paste(">"))` 0.05) but not adenoma [Figure S2B] (P-value `r ifelse(main_crc_bugs_summary["parv", "adn_BH"] < 0.05 & main_crc_bugs_summary["porp", "adn_BH"] < 0.05 & main_crc_bugs_summary["pept", "adn_BH"] < 0.05, paste("<"), paste(">"))` 0.05) there was a significant decrease in *Porphyromonas*, *Parvimonas*, and *Peptostreptococcus* between initial and follow up samples [Figure S2].  *Fusobacterium* for both adenoma and carcinoma had no significant differences between initial and follow up samples [FIgure S2] (P-value `r ifelse(main_crc_bugs_summary["fn", "crc_BH"] < 0.05 & main_crc_bugs_summary["fn", "adn_BH"] < 0.05, paste("<"), paste(">"))` 0.05).  Although there were significant differences, only a small percentage of those with adenoma or carcinoma were positive for any of these specific genera [Figure S2].


***The Lesion Model:***
We tested whether follow up samples were more normal than initial samples by building a model that classified lesion versus normal.  This model had an AUC range of `r format(min(lesion_AUC_run_summary$ROC), digits = 3)` - `r format(max(lesion_AUC_run_summary$ROC), digits = 3)` after 100 iterations of 20 repeated 10-fold corss validations.  The ROC curve for the final lesion model used was within the observed range of the 100-different test set AUC iterations [Figure 2A].  There was a total of `r length(lesion_top10_varsMDA$variable)` OTUs that were used in this model [Figure 2B].  *`r gsub("_unclassified", "", lesion_imp_vars[as.character((filter(lesion_top10_varsMDA, variable != "fit_result") %>% top_n(1, mean_MDA) %>% select(variable))[, 1]), "Genus"])`* (`r filter(lesion_top10_varsMDA, variable != "fit_result") %>% top_n(1, mean_MDA) %>% select(variable)`) resulted in the largest decrease in MDA [FIgure 2B].  Heavily represented genera within the model included OTUs from *Lachnospiraceae*, *Bacteroides*, *Roseburia*, *Blautia*, and *Ruminococcus*. 

If there was movement towards a more normal community after treatment of lesion, we would expect to find a decrease in the positive probability of the follow up sample to be a lesion.  We observed no such decrease in positive probability for either those with adenoma or carcinoma [FIgure 2C & 2D] (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_lesion" & comparison == "carcinoma_only")[, "BH_correction"] < 0.05 & filter(all_probs_paired_wilcox, model_type =="red_lesion" & comparison == "all_adenoma")[, "BH_correction"] < 0.05, paste("<"), paste(">"))` 0.05).  We also observed a significant difference between the predicted and actual calls (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "all" & model_type == "red_lesion" & info == "McnemarPValue")[, "all"] < 0.05, paste("<"), paste(">"))` 0.05).  This model correctly classified the one individual who still had carcinoma on follow up and this individual's positive probability of lesion increased between their initial and follow up sample [Figure 2C].


***The Treatment Model:***
To test if there were differences that could be used to classify initial and follow up treatment samples we built a model to classify initial samples.  This treatment model had an AUC range of `r format(min(IF_AUC_run_summary$ROC), digits = 3)` to `r format(max(IF_AUC_run_summary$ROC), digits = 3)`.  after 100 iterations of 20 repeated 10-fold cross validations.  The test set AUC range for this model performed better than the training set AUCs.  There was a marked decrease in the ROC curve for the final model used when compared to the 100 test set AUC iterations [Figure 3A].  There was a total of `r length(IF_top10_varsMDA$variable)` OTUs that were used for this model [Figure 3B].  The variable that resulted in the largest MDA was *`r gsub("_unclassified", "", IF_imp_vars[as.character((filter(IF_top10_varsMDA, variable != "fit_result") %>% top_n(1, mean_MDA) %>% select(variable))[, 1]), "Genus"])`* (`r filter(IF_top10_varsMDA, variable != "fit_result") %>% top_n(1, mean_MDA) %>% select(variable)`) [Figure 3B].  Similarly, to the lesion model heavily represented genera included OTUs from *Lachnospiraceae*, *Bacteroides*, *Roseburia*, *Blautia*, and *Ruminococcus*.

If there were changes between intial and follow up samples we would expect the positive probability of being an initial sample to be decreased in the follow up.  The is what we observed for the treatment model (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_IF" & comparison == "lesion")[, "BH_correction"] < 0.001, paste("<"), paste(">"))` 0.001).  When we separated lesion into adenoma and carcinoma there was a decrease in positive arobability for both the carcinoma [Figure 3C] (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_IF" & comparison == "carcinoma_only")[, "BH_correction"] < 0.001, paste("<"), paste(">"))` 0.001) and adenoma group [FIgure 3D] `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_IF" & comparison == "all_adenoma")[, "BH_correction"] < 0.001, paste("<"), paste(">"))` 0.001).  For this model, there was no difference between the predicted and actual classifications (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "all" & model_type == "red_IF" & info == "McnemarPValue")[, "all"] < 0.05, paste("<"), paste(">"))` 0.05).    


***Common OTUs to both Models:***
We next wanted to know what predictors within the lesion model were also in the initial sample model.  The main purpose was to identify which TOUs could be important in lesion formation and were impacted by treatment.  When we compared the two different models with each other there were a total of `r length(common_vars_summary$otu)` common OTUs.  Some of the most common taxonomic identifications belonged to *`r filter(common_vars_summary, otu == "Otu000028")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000061")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000037")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000006")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000015")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000167")[, "lowest_ID"]`*, and *`r filter(common_vars_summary, otu == "Otu000005")[, "lowest_ID"]`*.  These along withthe vast majority of the OTUs that were common between models had classifications to bacteria typically thought of as commensal [Table S2].    


***Chemotherapy and Radiation Differences:*** 
After observing these changes due to treatment we assessed whether chemotherapy or radiation impacted the observed results.  Only the treatment model had a significant change in positive probability for those treated with chemotherapy or radiation versus those who received neither [Figure 4A & Table S3] (P-value `r ifelse(chemo_rad_summary['chemo_IF', 'bh'] < 0.05 & chemo_rad_summary['rad_IF', 'bh'] < 0.05, paste("<"), paste(">"))` 0.05). Using the `r length(common_vars_summary$otu)` common OTUs we tested how chemotherapy or radiation affects important members of the community.  FOr both chemotherapy or radiation, the directionaly of change of these OTUs were similar [Figure 4B & 4C].  After multiple comparison correction, only *`r filter(chemo_rad_common_vars_summary, bh_chemo < 0.05)[, "lowest_ID"]`* (`r filter(chemo_rad_common_vars_summary, bh_chemo < 0.05)[, "otu"]`) remained significantly different between those who did and did not receive chemotherapy.  This data suggests that follow up samples from those treated with either chemotherapy or radiation may have had a larger change from initial sample due to affects on the bacterial community.


\newpage


## Discussion
This study builds upon previous work from numerous labs that have considered both how the bacterial community between those with and without CRC differ and how it might be used as an early screening tool [@yu_metagenomic_2017; @baxter_microbiota-based_2016; @zeller_potential_2014; @zackular_human_2014; @warren_co-occurrence_2013].  Here we describe how the bacterial community changes after treatment and which OTUs may be most important to lesion classification.  Interestingly, many of the most important OTUs for both treatment and lesion classification had taxonomic identification for resident gut microbes which could suggest that members within the commensal community may be the first that change during pathogenesis of disease.  Further, these changes could be the first step in allowing more inflammatory bacterium to gain a foothold within the colon [@flynn_metabolic_2016].

Unlike previous studies on the microbiome and CRC, ours focuses on identifying commonalities within both adenoma and carcinoma groups before and after treatment.  Although there were differences for genera associated with specific bacterium linked with CRC [Figure S2].  These changes were not consistent across lesion.   Instead, most of our results provide support that the first members of the community to change and potentially stay changed even after treatment are those that are commensal bacteria [Figure 2-3 & Table S3].  Additionally, treatment with chemotherapy or radiation changed these bacteria and provided a larger decrease in positive probability then removal of lesion. 

Curiously, we observed that the typical CRC associated bacteria were not predictive within our models. There are several reasons why this may have occurred. First, is that they were not present in enough individuals to be able to classify those with and without disease with a high degree of accuracy. Second, is that our Random Forest models were able to gather the same information from other OTUs. Third, it is also possible that changes in commensal bacteria are the common differences across adenoma and carcinoma. Finally, it is also possible that all these explanations could have played a role.  Our observations though, would suggest that an individual's resident bacteria have a large role to play in disease initiation and could change in a way that allows predictive models to lower the positive probability of wither lesion or initial sample before treatment [Figure 2 & 3]. It should be noted that our study does not argue against the importance of these CRC associated bacteria in the pathogenesis of disease but rather that they are not the main bacteria changing after treatment. It is possible that these CRC associated bacteria are important in the transition from adenoma to carcinoma and would be one explanation as to why in our data we not only see high initial relative abundances in carcinoma and not adenoma individuals but also large decreases in relative abundance in some of those with carcinoma but not in those with adenoma after lesion removal [Figure S2].

Many of the common OTUs that we identified taxnomically classified to potential butyrate producers [Table S2].  Another batch of OTUs classified to bacteria that can either degrade polyphenols or are inhibited by them. Both butyrate and polyphenols are thought to be protective against cancer in part by reducing inflammation [@louis_gut_2014].  These protective compounds are derived from the breakdown of fiber, fruits, and vegetables by resident gut microbes. One example of this potential diet-microbiome-inflammation-polyp axis is that *Bacteroides*, which was highly prevalent in our models, are known to be increased in those with high non-meat based protein consumption [@zhu_intake_2016].  High protein consumption in general has been linked with an increased CRC risk [@mu_colonic_2016].  Conversely, *Bacteroides* are inhibited by polyphenols which are derived from fruits and vegetables [@ozdal_reciprocal_2016].  Our data fits with the hypothesis that the microbial metabolites from breakdown products within our own diet could not only help to shape the existing community but also have an effect on CRC risk and disease progression.

A limitation, in our study, was that there was a significant difference in the time elapsed in the collection of the follow up sample between adenoma and carcinoma (uncorrected P-value `r ifelse(time_pvalues["time", "uncorrected_p_value"] < 0.05, paste("<"), paste(">"))` 0.05), with time passed being less for adenoma (`r format(time_summary['adenoma', 'mean_days'], digits = 3)` +/- `r format(time_summary['adenoma', 'sd_days'], digits = 3)` days) than carcinoma (`r format(time_summary['carcinoma', 'mean_days'], digits = 3)` +/- `r format(time_summary['carcinoma', 'sd_days'], digits = 3)` days).  These results would indicate that some of the differences observed between the carcinoma and adenoma groups could be due to differences in collection time. Specifically, it could confound the observation that carcinomas changed more than adenomas [Figure 1A & 1C]. This confounding though would not affect the observations where these individuals were grouped together [Figures 2-4].

Another limitation was that we do not know whether individuals who were still classified as positive by the lesion model eventually had a subsequent CRC diagnosis. This information would help to strengthen the case for this model keeping several individuals above the cutoff threshold even though at follow up they were diagnosed as no longer having a lesion. This study also drew heavily from those with Caucasian ancestry making it possible that the observations may not be representative of those with either Asian or African ancestry. Although our training and test set are relatively large we still run the risk of over-fitting or having a model that may not be representative of other populations. We've done our best to safeguard against this by not only running 10-fold cross validation but also having over 100 different 80/20 splits to try and mimic the type of variation that might be expected to occur.
  
Interestingly, within the treatment model the test data performed better than the training data. This may have occurred because the training AUC determined from 20 repeated 10-fold cross validation removed samples at random and did not consider that they were matched samples. Another potential reason is that the model itself may have been over-fit since the total number of samples was not that large. However, the lesion model did not suffer from these discrepancies. Further independent studies need to be carried out to verify our findings on lesion and treatment changes due to these limitations.
  
Despite these shortcomings our findings add to the existing scientific knowledge on CRC and the microbiome: That there is a measurable difference in the bacterial community after adenoma and carcinoma treatment. Further, the ability for machine learning algorithms to take OTU data and successfully lower positive probability of lesion or initial sample after treatment provides evidence that there are specific signatures, mostly attributable to commensal organisms, associated with both treatment and lesion. Our data provides evidence that commensal bacteria may be important in the development of polyps and potentially the transition from adenoma to carcinoma.



\newpage

## Methods

***Study Design and Patient Sampling:***
Sampling and design have been previously reported in Baxter, et al [@baxter_microbiota-based_2016]. Briefly, study exclusion involved those who had already undergone surgery, radiation, or chemotherapy, had colorectal cancer before a baseline fecal sample could be obtained, had IBD, a known hereditary non-polyposis colorectal cancer, or familial adenomatous polyposis. Samples used to build the models for prediction were collected either prior to a colonoscopy or between 1 - 2 weeks after. The bacterial community has been shown to normalize back to a pre-colonscopy community within this time period [@obrien_impact_2013]. Our study cohort consisted of `r length(rownames(good_metaf))` individuals with an initial sample as described and a follow up sample obtained between `r min(good_metaf$time)` - `r max(good_metaf$time)` days after treatment of lesion. This study was approved by the University of Michigan Institutional Review Board. All study participants provided informed consent and the study itself conformed to the guidelines set out by the Helsinki Declaration.


***16S rRNA Gene Sequencing:***
Sequencing was completed as described by Kozich, et al. [@kozich_development_2013]. DNA extraction used the 96-well Soil DNA isolation kit (MO BIO Laboratories) and an epMotion 5075 automated pipetting system (Eppendorf). The V4 variable region was amplified and the resulting product was split between three sequencing runs with normal, adenoma, and carcinoma evenly represented on each run. Each group was randomly assigned to avoid biases based on sample collection location.  The initial and follow up samples were sequenced on the same run.


***Sequence Processing:***
The mothur software package (v1.37.5) was used to process the 16S rRNA gene sequences and has been previously described [@kozich_development_2013]. The general workflow using mothur was: Paired-end reads were first merged into contigs, quality filtered, aligned to the SILVA database, screened for chimeras, classified with a naive Bayesian classifier using the Ribosomal Database Project (RDP), and clustered into Operational Taxonomic Units (OTUs) using a 97% similarity cutoff with an average neighbor clustering algorithm. The number of sequences for each sample was rarefied to `r ifelse(sd(total_sub_Seqs) == 0, paste(mean(total_sub_Seqs)), paste("Error Check Subsampling"))` to minimize uneven sampling.


***Lesion Model Creation:***
The Random Forest [@breiman_random_2001] algorithm was used to create the model used for prediction of lesion (adenoma or carcinoma) with the main training and testing of the model completed on an independent data set of  `r length(rownames(metaI)) - length(rownames(good_metaf))` individuals. This model was then applied to our `r length(rownames(good_metaf))`-person cohort. It should be noted that all individuals with an adenoma or carcinoma were grouped together to form the lesion group and the model was not created to find differences between normal, adenoma, and carcinoma but rather differences between both adenoma and carcinoma versus normal.

The model included only OTU data obtained from 16S rRNA sequencing. Non-binary data was checked for near zero variance and OTUs that had near zero variance were removed. This pre-processing was performed with the R package caret (v`r paste(packageVersion("caret"))`). Optimization of the mtry hyper-parameter involved making 100 different 80/20 (train/test) splits of the data where normal and lesion were represented in the same proportion within both the whole data set and the 80/20 split. For each a 20 repeated 10-fold cross validation was performed on 80% component to optimize the mtry hyper-parameter by maximizing the AUC (Area Under the Curve of the Receiver Operator Characteristic). The resulting model was then tested on the hold out data obtained from the 20% component. Assessment of the most important OTUs to the model involved counting the number of times an OTU was present in the top 10% of mean decrease in accuracy (MDA) for each of the 100 different splits run.  This was then followed with filtering of this list to variables that were only present in more than 50% of these 100 runs. The final collated list of variables was then run through the mtry optimization again. Once the ideal mtry was found the entire `r length(rownames(metaI)) - length(rownames(good_metaf))` sample set was used to create the final Random Forest model on which classifications on the `r length(rownames(good_metaf))`-person cohort was completed.
  
The default cutoff of 0.5 was used as the threshold to classify individuals as positive or negative for lesion. The hyper-parameter, mtry, defines the number of variables to investigate at each split before a new division of the data was created with the Random Forest model.


***Treatment Model Creation:***
We also investigated whether a model could be created that could identify pre- (initial) and post- (follow up) treatment samples. The main difference was that only the `r length(rownames(good_metaf))`-person cohort was used at all stages of model building and classification. Other than this difference the creation of this model and optimization of the mtry hyper-parameter was completed using the same procedure as was used for the lesion model. Instead of classifying samples as positive or negative of lesion this model classified samples as positive or negative for being an initial sample prior to treatment.


***Statistical Analysis:***
The R software package (v`r paste(version$major, ".", version$minor, sep = "")`) was used for all statistical analysis.  Comparisons between bacterial community structure utilized PERMANOVA [@anderson_permanova_2013] in the vegan package (v`r paste(packageVersion("vegan"))`).  Comparisons between probabilities as well as overall OTU differences between initial and follow up samples utilized a paired Wilcoxson ranked sum test.  Where multiple comparison testing was appropriate, a Benjamini-Hochberg (BH) correction was applied [@benjamini_controlling_1995] and a corrected P-value of less than 0.05 was considered significant.  Unless otherwise stated the P-values reported are those that were BH corrected.  


***Analysis Overview:***
We first tested for any differences based on whether the individual had an adenoma or carcinoma. This was done by testing initial and follow up samples for differences in alpha and beta diversity, testing all OTUs, and investigating the relative abundance of genera from previously associated CRC bacteria (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, and *Porphyromonas*). Next, the lesion model was tested for accuracy in prediction and whether it reduced the positive probability of lesion in follow up samples. We then used the treatment model to assess whether it could classify samples better than the lesion model and whether it could reduce the positive probability of an initial sample in the follow up samples.  Common OTUs were found for the two different models used to assess which were important for both models.  Finally, differences between those receiving chemotherapy and radiation versus those who received neither were tested.   


***Reproducible Methods:***
A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Sze_followUps_2017.  Raw sequences have been deposited into the NCBI Sequence Read Archive (SRP062005 and SRP096978) and the necessary metadata can be found at https://www.ncbi.nlm.nih.gov/Traces/study/ and searching the respective SRA study accession.


\newpage


**Figure 1: General Differences between the Adenoma and Carcinoma Group.**  A) A significant difference was found between the adenoma and carcinoma group for thetayc (P-value = `r format(change_summary['thetayc', 'Pvalue'], scientific = FALSE, digits = 3)`).  Advanced adenomas are denoted as Screen Relevant Neoplasia (SRN).  B) NMDS of the initial and follow up samples for the adenoma group.  C) NMDS of the initial and follow up samples for the carcinoma group.  


**Figure 2: The Lesion Model.** A) ROC curve: The shaded areas represent the range of values of a 100 different 80/20 splits of the test set data and the blue line represents the model using 100% of the data set and what was used for subsequent classification.  B) Summary of Important Variables.  MDA of the most important variables in the lesion model.  The black point represents the mean and the different colors are the values of each different run up to 100. C) Positive probability change from initial to follow up sample in those with carcinoma.  D) Positive probability change from initial to follow up sample if those with adenoma or advanced adenoma (Screen Relevant Neoplasia (SRN)).


**Figure 3: The Treatment Model.**  A) ROC curve: The shaded areas represent the range of values of a 100 different 80/20 splits of the test set data and the blue line represents the model using 100% of the data set and what was used for subsequent classification.  B) Summary of Important Variables.  MDA of the most important variables in the initial sample model.  The black point represents the mean and the different colors are the values of each different run up to 100.  C) Positive probability change from initial to follow up sample in those with carcinoma.  D) Positive probability change from initial to follow up sample of those with adenoma or advanced adenoma (Screen Relevant Neoplasia (SRN)).


**Figure 4: Chemotherapy and Radiation Therapy Effects on Community.**  A) Treatment model initial sample positive probability reduction for chemotherapy.  B) Treatment model initial sample positive probability reduction for radiation therapy.  C) Common OTUs based on whether chemotherapy was received or not.  The * denotes a P-value < 0.05 after multiple comparison correction.  D) Common OTUs based on whether radiation therapy was received or not.



\newpage

**Figure S1: Distribution of P-values from Paired Wilcoxson Analysis of All OTUs for Initial versus Follow Up**

**Figure S2: Previously Associated CRC Bacteria in Initial and Follow Up Samples.**  A) Carcinoma initial and follow up samples had an observed significant difference in initial and follow up sample for the OTUs classified as *Parvimonas* (P-value = `r format(main_crc_bugs_summary["parv", "crc_BH"], digits = 3)`), *Porphyromonas* (P-value = `r format(main_crc_bugs_summary["porp", "crc_BH"], digits = 3)`), *Peptostreptococcus* (P-value = `r format(main_crc_bugs_summary["pept", "crc_BH"], digits = 3)`).  B) Adenoma initial and follow up samples.  There were no significant differences between initial and follow up (P-value = `r format(main_crc_bugs_summary["porp", "adn_BH"], digits = 3)`.

\newpage

## Declarations

### Ethics approval and consent to participate

The University of Michigan Institutional Review Board approved this study, and all subjects provided informed consent. This study conformed to the guidelines of the Helsinki Declaration.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Sze_followUps_2017.  Raw sequences have been deposited into the NCBI Sequence Read Archive (SRP062005 and SRP096978) and the necessary metadata can be found at https://www.ncbi.nlm.nih.gov/Traces/study/ and searching the respective SRA study accession.

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

This study was supported by funding from the National Institutes of Health to P. Schloss (R01GM099514, P30DK034933) and to the Early Detection Research Network (U01CA86400).

### Authors' contributions

All authors were involved in the conception and design of the study. MAS analyzed the data. NTB processed samples and analyzed the data. All authors interpreted the data. MAS and PDS wrote the manuscript. All authors reviewed and revised the manuscript. All authors read and approved the final manuscript.

### Acknowledgements

The authors thank the Great Lakes-New England Early Detection Research Network for providing the fecal samples that were used in this study. We would also like to thank Amanda Elmore for reviewing and correcting code error and providing feedback on manuscript drafts.  We would also like to thank Nicholas Lesniak for providing feedback on manuscript drafts.

\newpage

## References




