---
title: ""
bibliography: reference.bib
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
csl: microbiome.csl
fontsize: 12pt
geometry: margin=1.0in
---

# Differences in the fecal Microbiome Before and After Colorectal Cancer Treatment
```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')
source('../code/graphFunctions.R')

loadLibs(c("dplyr", "ggplot2", "reshape2", "gridExtra", "scales", "wesanderson", "knitr", "rmarkdown"))

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

```

```{r rdata_tables}
# Extra data and meta data needed
good_metaf <- read.csv('../results/tables/mod_metadata/good_metaf_final.csv', header = T, stringsAsFactors = F)
metaI <- read.csv('../results/tables/mod_metadata/metaI_final.csv', header = T, stringsAsFactors = F)
total_sub_Seqs <- read.delim('../data/process/final.0.03.subsample.shared', header = T) %>% 
  select(contains("Otu0")) %>% rowSums()

# initial microbiome and FIT description data 
change_summary <- read.csv('../results/tables/change_theta_fit_summary.csv', header = T, row.names = 1)
alpha_summary <- read.csv('../results/tables/alpha_table_summary.csv', header = T, row.names = 1)
beta_summary <- read.csv('../results/tables/beta_diver_summary.csv', header = T, row.names = 1)
time_pvalues <- read.csv("../results/tables/time_pvalues.csv", header = T, row.names = 1)
all_OTU_pair_wilcox <- read.csv("../results/tables/OTU_paired_wilcoxson_test.csv", header = T)

# Main CRC bacteria differences between Adenoma and Carcinoma
main_crc_bugs_summary <- read.csv('../results/tables/adn_crc_maybe_pvalue_summary.csv', header = T, row.names = 1)
crc_bug_info <- read.csv("../results/tables/adn_crc_maybe_diff.csv", header = T, stringsAsFactors = F)

# Lesion Model (Full)
lesion_top10_varsMDA <- read.csv('../results/tables/lesion_model_top_vars_MDA_Summary.csv', header = T)
lesion_imp_vars <- read.csv('../results/tables/rf_otu_tax.csv', header = T, row.names = 1)
lesion_AUC_run_summary <- read.csv('../results/tables/ROC_model_summary.csv', header = T)

# Lesion Model (Reduced)
reduced_lesion_AUC_run_summary <- read.csv('../results/tables/Reduced_ROC_model_summary.csv', header = T)

# IF Model (Full)
IF_top10_varsMDA <- read.csv('../results/tables/IF_model_top_vars_MDA_Summary.csv', header = T)
IF_imp_vars <- read.csv('../results/tables/if_rf_otu_tax.csv', header = T, row.names = 1)
IF_AUC_run_summary <- read.csv('../results/tables/IF_ROC_model_summary.csv', header = T)

# IF Model (Reduced)
reduced_IF_AUC_run_summary <- read.csv('../results/tables/reduced_IF_ROC_model_summary.csv', header = T)


# All model information
all_probs_paired_wilcox <- read.csv('../results/tables/all_models_wilcox_paired_pvalue_summary.csv', header = T, stringsAsFactors = F)
all_pred_stat_summary <- read.csv('../results/tables/all_models_summary_info.csv', header = T, stringsAsFactors = F)
all_confu_matrix <- read.csv('../results/tables/all_models_confusion_summary.csv', header = T, stringsAsFactors = F)

# Common Variable Info
common_vars_summary <- read.csv('../results/tables/pvalue_IF_lesion_common_imp_vars.csv', header = T, row.names = 1, stringsAsFactors = F)

#Therapy Info
chemo_rad_summary <- read.csv('../results/tables/probs_chemo_rad_pvalue_summary.csv', header = T, row.names = 1)

```
\vspace{25mm} 

\begin{center}
Running Title: Human Microbiome and Colorectal Cancer

\vspace{10mm}

Marc A Sze${^1}$, Nielson T Baxter${^2}$, Mack T Ruffin IV${^3}$, Mary AM Rogers${^2}$, and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI

$2$ Department of Internal Medicine, University of Michigan, Ann Arbor, MI   

$3$ Department of Family Medicine and Community Medicine, Penn State Hershey Medical Center, Hershey, PA    


\end{center}  


\newpage
\linenumbers

## Abstract

**Background:** Colorectal cancer (CRC) continues to be a worldwide health problem with early detection being used as a key component in mitigating deaths due to the disease. Previous research suggests a link between fecal bacterial microbiome and CRC.   The overall objective of our study was to investigate the changes in the bacterial microbiome after surgery in patients with lesion (i.e. adenoma or carcinoma).  Specifically, we wanted to identify what within the community was different within those undergoing surgical removal of lesion.  We also wanted to investigate the use of the bacterial microbiome and Fecal Immunoglobulin Test (FIT) to build models which could classify individuals as having a lesion or as before surgery.     
**Results:** Adenoma individual's bacterial microbiome were more similar to their pre-surgery sample then those with carcinoma (P-value = `r format(change_summary['thetayc', 'Pvalue'], digits = 3)`).  Their change in FIT was also significantly different then those with carcinoma (P-value = `r format(change_summary['fit', 'Pvalue'], scientific = TRUE, digits = 3)`).  There was no significant difference in any indivdiual OTU between samples before and after surgery (P-value > `r format(min(filter(all_OTU_pair_wilcox, analysis == "ALL")[, 'BH_corrected']), digits = 3)`).  A model with a total of `r length(lesion_top10_varsMDA$variable)` variables was able to classify lesion (AUC = `r format(max(reduced_lesion_AUC_run_summary$ROC), digits = 3)` - `r format(min(reduced_lesion_AUC_run_summary$ROC), digits = 3)`) while the model to classify samples as before surgery had `r length(IF_top10_varsMDA$variable)` variables (AUC = `r format(max(reduced_IF_AUC_run_summary$ROC), digits = 3)` - `r format(min(reduced_IF_AUC_run_summary$ROC), digits = 3)`).  The after surgery samples had a significant decrease in the model positive probability of a lesion or before surgery sample (P-value = `r format(filter(all_probs_paired_wilcox, model_type == "red_lesion" & comparison =="lesion")[, "BH_correction"], scientific = TRUE, digits = 3)` and `r format(filter(all_probs_paired_wilcox, model_type == "red_IF" & comparison =="lesion")[, "BH_correction"], scientific = TRUE, digits = 3)`).  In total there were `r length(common_vars_summary$otu)` OTUs that were common to both models and the majority of these classified to commensal bacteria (`r common_vars_summary["Otu000005", "lowest_ID"]`, `r common_vars_summary["Otu000012", "lowest_ID"]`, `r common_vars_summary["Otu000082", "lowest_ID"]`, and `r paste(gsub("_unclassified", "", common_vars_summary["Otu000006", "lowest_ID"]))`).   
**Conclusions:** Our data suggests that treatment not only significantly reduces the probability of having a colonic lesion within our models but also causes detectable changes in the bacterial microbiome.  Further surveillance of these individuals will enable us to determine whether models such as the ones we present could be used to predict recurrence of colorectal cancer.  

\newpage

### Keywords

bacterial microbiome; colorectal cancer; polyps; FIT; detection; risk factors      

\newpage

## Background

Colorectal cancer (CRC) continues to be a leading cause of cancer related deaths and is the second most common cancer death among men aged 40-79 years of age [@jemal_cancer_2010; @haggar_colorectal_2009].  Over the last few years death due to the disease has seen a significant decrease, thanks mainly to improvements in screening [@jemal_cancer_2010].  However, despite this improvement there are still approximately 50,000 deaths from the disease a year [@haggar_colorectal_2009].  It is estimated that around 5-10% of all CRCs can be explained by autosomal dominant inheritance [@green_very_2007]. The vast majority of CRCs are not inherited and the exact etiology of the disease has not been well worked out [@haggar_colorectal_2009].  Although many risk factors have been identified [@haggar_colorectal_2009] and non-invasive screening techniques have started to be put into consistent use [@liao_application_2013; @johnson_multi-target_2016] there has been an additional increase in the incidence of CRC in the younger population.  

This increased incidence of CRC in the younger population is concerning since having either an adenoma or carcinoma increases ones risk for future adenomas or carcinomas [@laiyemo_short-_2013; @matsuda_five-year_2009; @ren_long-term_2016].  This increased risk can also carry with it an increased chance of mortality due to this recurrence [@loberg_long-term_2014; @freeman_natural_2013].  Therefore, there has been a great amount of interest in early risk stratification tools [@lee_identification_2016; @richards_evidence-based_2016] that can help identify those that may be at most susceptibility to reccurence.  Concurrently, there has also been a lot of interest in new areas that could have a role in disease pathogenesis, such as the gut bacterial microbiome.

There has been promising work on the bacterial microbiome and it's ability to be able to complement existing screening methods such as Fecal Immunoglobulin Test (FIT) or act alone as a screening tool [@baxter_microbiota-based_2016; @zeller_potential_2014]. There has also been research into how this microbiome could be altered directly on tumor tissue itself [@dejea_microbiota_2014].  A few studies have now even shown how this microbiome [@zackular_manipulation_2016] or specific members within it [@arthur_microbial_2014] could be directly involved with the pathogenesis of CRC.  These studies have helped to provide a tantilzing link between the bacterial microbiome and CRC. However, at this present time there remains limited information on the bacterial microbiome before and after successful surgery for removal of the adenoma or carcinoma and whether it changes at all.

In this study we investigated what happened to the bacterial microbiome before and after surgery for indivdiuals with either adenoma or carcinoma.  Our anlaysis includes both alpha and beta diversity analysis along with investigation of individual operational taxonomic units (OTUs).  We also utilized Random Forest models and observed how these models as well as specific OTUs within them performed before (initial) and after (follow up) surgery.  We also used these models to look for similar important OTUs to identify the crucial OTUs for not only classifying initial and follow up samples but also lesion or normal.



\newpage

## Results

***Bacterial Community and FIT***
Based on the thetayc distance metric, comparing the initial to the follow up samples, there was a significant difference between the adenoma and carcinoma groups (P-value = `r format(change_summary['thetayc', 'Pvalue'], scientific = FALSE, digits = 3)`) [Figure 1a]. There was also a significant difference in change in FIT between initial and follow up samples (P-value = `r format(change_summary['fit', 'Pvalue'], scientific = TRUE, digits = 3)`) [Figure 1b].  The whole community structure before and after surgery was visualized on NMDS graphs for both adenoma [Figure1c] (PERMANOVA = `r format(beta_summary['Adenoma', 'PERMANOVA'], scientific = FALSE, digits = 3)`) and carcinoma [Figure 1d] (PERMANOVA = `r format(beta_summary['Carcinoma', 'PERMANOVA'], scientific = FALSE, digits = 3)`).  When all initial and follow up samples were compared to each other there was no significant overall difference between them (PERMANOVA = `r format(beta_summary['All', 'PERMANOVA'], scientific = FALSE, digits = 3)`).  There was no significant difference between initial and follow up samples for observed OTUs, Shannon diversity, or evenness after correction for multiple comparisons [Table S1].  There was also no significant difference between initial and follow up samples for any single OTU [Figure S1].


***Previously Associated Cancer Bacteria***
We next examined whether there were differences in previously well described carcinoma associated OTUs.  These included the OTUs that aligned with *Porphyromonas asaccharolytica* (`r unique(filter(crc_bug_info, tax_id == "Porphyromonas")[, "otu"])`), *Fusobacterium nucleatum* (`r unique(filter(crc_bug_info, tax_id == "Fusobacterium")[, "otu"])`), *Parvimonas micra* (`r unique(filter(crc_bug_info, tax_id == "Parvimonas")[, "otu"])`), and *Peptostreptococcus stomatis* (`r unique(filter(crc_bug_info, tax_id == "Peptostreptococcus")[, "otu"])`).  First, the carcinoma samples showed a significant difference between initial and follow up samples for *Peptostreptococcus stomatis* (P-value = `r format(main_crc_bugs_summary["pept", "crc_BH"], digits = 3)`) and *Porphyromonas asaccharolytica* (P-value = `r format(main_crc_bugs_summary["porp", "crc_BH"], digits = 3)`) whereas there were no signficant differences in any of these OTUs in the adenoma samples [Table S4]. Second, when these OTUs were present, there was a clear magnitude difference based on whether they were from adenoma or carcinoma [Figure 2].  However, only a small percentage of those with adenoma or carcinoma were positive for any of these OTUs.


***Full and Reduced Model***
Since differences were observed between initial and follow up samples and only a small number of individuals were positive for previously associated CRC bacteria; we next investigated if we could create models that could adequately classify and adjust either lesion or before sample probability based on the bacterial community and FIT.  The lesion model had an AUC range of `r format(min(lesion_AUC_run_summary$ROC), digits = 3)` to `r format(max(lesion_AUC_run_summary$ROC), digits = 3)` versus the before sample model which had an AUC range of `r format(min(IF_AUC_run_summary$ROC), digits = 3)` to `r format(max(IF_AUC_run_summary$ROC), digits = 3)` after 100 iterations of 20 repeated 10-fold cross validations.  Interestingly, identification of the most important variables and reducing the models to only these factors increased the AUC in the lesion model (`r format(min(reduced_lesion_AUC_run_summary$ROC), digits = 3)` - `r format(max(reduced_lesion_AUC_run_summary$ROC), digits = 3)`) and before sample model (`r format(min(reduced_IF_AUC_run_summary$ROC), digits = 3)` - `r format(max(reduced_IF_AUC_run_summary$ROC), digits = 3)`).      

The test set AUC range for the full and reduced lesion model were similar to that reported for the training set AUC ranges and the ROC curve ranges overlap each other [Figure 3a].  The ROC curve for the final lesion model used was within the range of both the full and reduced lesion model [Figure 3a].  Interestingly, the test set AUC range for the before sample model performed much better then the training set AUCs.  Both the full and reduced before sample models overlapped with each other [Figure 3b] there was a marked decrease in the ROC curve for the final model used.  

***Most Important Variables***
The reduced models were built based on the most important variables to the respective full model.  For the lesion model there were a total of `r length(lesion_top10_varsMDA$variable)` variables [Figure S2] whereas for the before sample model there were a total of `r length(IF_top10_varsMDA$variable)` variables [Figure S3].  For both models FIT resulted in the largest decrease in MDA [Figure S2a & S3a].


***Positive Probability after Lesion Removal***
Regardless of model used there was a significant decrease in the positive probability of either the sample being lesion or a before sample on follow up [Figure 4 & S4] (full lesion P-value = `r format(filter(all_probs_paired_wilcox, model_type =="lesion" & comparison == "lesion")[, "BH_correction"], scientific = TRUE, digits = 3)`, reduced lesion P-value = `r format(filter(all_probs_paired_wilcox, model_type =="red_lesion" & comparison == "lesion")[, "BH_correction"], scientific = TRUE, digits = 3)`, initial and follow up P-value = `r format(filter(all_probs_paired_wilcox, model_type =="IF" & comparison == "lesion")[, "BH_correction"], scientific = TRUE, digits = 3)`, and reduced initial and follow up P-value = `r format(filter(all_probs_paired_wilcox, model_type =="red_IF" & comparison == "lesion")[, "BH_correction"], scientific = TRUE, digits = 3)`).    

For the full and reduced lesion model there was a significant difference in the classification for the lesion model between predicted and actual (P-value = `r format(filter(all_pred_stat_summary, samples_tested == "all" & model_type == "lesion" & info == "McnemarPValue")[, "all"], scientific = TRUE, digits = 3)` and `r format(filter(all_pred_stat_summary, samples_tested == "all" & model_type == "red_lesion" & info == "McnemarPValue")[, "all"], scientific = TRUE, digits = 3)`, respectively) but not for the before sample model (P-value = `r format(filter(all_pred_stat_summary, samples_tested == "all" & model_type == "IF" & info == "McnemarPValue")[, "all"], digits = 3, nsmall = 2)` and `r format(filter(all_pred_stat_summary, samples_tested == "all" & model_type == "red_IF" & info == "McnemarPValue")[, "all"], digits = 3, nsmall = 2)`).  However, the lesion model correctly kept the one individual who still had a carcinoma on follow up above the cut off threshold [Figure 4a & S4a] for a positive call while the before sample models did not [Figure 4b & S4b].    


***Common OTUs to both Models***
There were a total of `r length(common_vars_summary$otu)` OTUs that were common to both models.  Of these OTUs the most common taxonomic identifications were to  `r common_vars_summary['Otu000012', 'lowest_ID']`, `r common_vars_summary['Otu000005', 'lowest_ID']`, `r common_vars_summary['Otu000082', 'lowest_ID']`, and `r paste(gsub("_unclassified", "", common_vars_summary['Otu000006', 'lowest_ID']))`.  The majority of these OTUs had classifications to bacteria typically thought of as commensal [Table S2].         


***Treatment and Time Differences***
There was no difference in the amount of change in positive probability for either the full or reduced lesion model for either chemotherapy (P-value = `r format(chemo_rad_summary['chemo_les', 'bh'], digits = 3)` and `r format(chemo_rad_summary['chemo_red_les', 'bh'], digits = 3)`) or radiation therapy (P-value = `r format(chemo_rad_summary['rad_les', 'bh'], digits = 3)` and `r format(chemo_rad_summary['rad_red_les', 'bh'], digits = 3)`). Although the before sample model was similar there was a significant decrease in positive probability for those treated with chemotherapy (P-value = `r format(chemo_rad_summary['chemo_IF', 'bh'], scientific = TRUE, digits = 3)` and `r format(chemo_rad_summary['chemo_red_IF', 'bh'], scientific = TRUE, digits = 3)`).  Time elapsed between the collection of the follow up sample from the initial sample, did not have a significant difference between adenoma and carcinoma (uncorrected P-value = `r format(time_pvalues["thetayc_change", "uncorrected_p_value"], digits = 3)`).    

\newpage


## Discussion
There was no difference in alpha diversity metrics between the initial and follow up samples [Table S1].  Yet based on thetayc there was a significant difference between inital and follow up samples [Figure 1a] with carcinoma samples being more dissimilar to their initial samples then adenoma samples.  The change in FIT was also different between initial and follow up for those with adenoma or carcinoma [Figure 1b].  There was also a significant difference in the overall bacterial community structure for those in the carcinoma group [Figure 1d] before and after surgery but not for the adenoma group [Figure 1c].  Investigation of the OTU relative abundance before and after surgery found no single OTU to be significantly different [Figure S1].  

Interestingly, when only previously associated cancer bacteria were investigated only `r length(rownames(filter(main_crc_bugs_summary, crc_BH < 0.05)))` out of the `r length(rownames(main_crc_bugs_summary))` had a significant decrease in relative abundance between initial and follow up for the carcinoma group and `r length(rownames(filter(main_crc_bugs_summary, adn_BH < 0.05)))` out of the `r length(rownames(main_crc_bugs_summary))` were significant for the adenoma group.  This data suggests that the changes that may be important in the bacterial microbiome after surgery are not necessarily any one specific bacterium, a clear depression of CRC related bacteria, or any addition of new or depressed bacterium.  Instead it may be due to a shift of the bacterial community as a whole between existing members of the host's microbiome.

We next created a model that incorporated FIT and the bacterial microbiome to either be able to classify lesions (adenoma or carcinoma) or before samples in order to confirm that the community was what was responding or changing due to surgery.  We found that the OTUs that made up the most important variables to the model overwhelmingly belonged to comensal bacteria.  With only the lesion model having a single OTU from a previously associated cancer bacterium (*Porphyromonas asaccharolytica*).  Using only these important OTUs and FIT both models (lesion and before sample) had a significantly decreased positive probability of either lesion or being an initial sample on follow up [Figure 4 & S4].  Further confirmation of the importance of the changes of comensal bacteria to these classifications was that a total of `r length(common_vars_summary$otu)` OTUs were common to both models and the vast majority belonged to regular residents of our gut community.  

There was no difference for the majority of models tested for differences in positive probablity based on whether chemotherapy or radiation was received.  There was also no difference in the thetayc distance metric based on length of time between initial and follow up sample between adenoma and carcinoma.  These results would indicate that the findings described were specific to the surgical intervention and that differences observed between carcinoma and adenoma samples can not be simply attributed to collection time between samples.  

This study builds upon previous work from numerous labs that have looked into the bacterial microbiome as a potential screening tool [@baxter_microbiota-based_2016; @zeller_potential_2014] by exploring what happens to the bacterial community after surgical removal of a lesion.  Based on previous work by Arthur, et al. [@arthur_intestinal_2012]  it may not be surprising to have E.coli as one of the most important OTUs and one that was common to both models.  Interestingly, many of the most important OTUs had taxonomic identification for resident gut microbes.  This could suggest that the bacterial community is one of the first components that could change during the pathogenesis of disease.  These bacterial microbiome changes could be the first step in allowing more inflammatory bacterium to gain a foothold within the colon [@flynn_metabolic_2016].

Curiously, we observed that the typical CRC associated bacteria were not predictive within our models.  There are a number of reasons why this may have occured.  First, one potential explanation is that even with surgery and a shift of the bacterial community these specific bacteria still persisted within the colon.  Second, the bacteria even if they were reduced in relative abundance they were still present within the gut.  Third, is that they were not present in enough individuals to be able to classify those with and without disease with a high degree of accuracy.  Fourth, is that it is possible that our Random Forest models were able to get the same information from measures such as FIT or other OTUs.  Finally, it is also possible that all three of these potential explanations could have played a role.  Regardless, our observations would suggest that an individual's resident bacteria have a large role to play in disease initiation and could change in a way that allows predictive models to lower the positive probability of a lesion after surgery [Figure 4].  It should be noted that our study does not argue against the importance of these CRC associated bacteria in the pathogenesis of disease but rather that the models do not utilize these specific bacteria for classification purposes (lesion or before sample).  In fact, it is possible that these CRC associated bacteria are important in the transition from adenoma to carcinoma and would be one explanation as to why in our data we not only see high initial relative abundances, in certain individuals, but also large decreases in relative abundance in those with carcinoma but not in those with adenoma after surgery [Figure 2].      

One limitation of our study is that we do not know whether individuals who were still classified as positive by the lesion model eventually had a subsequent CRC diagnosis.  This information would help to strengthen the case for our Random Forest based model to have kept a number of individuals above the cutoff threshold even though at follow up they were diagnosed as no longer having a lesion.  Another limitation is that we do not know if adding modern tests such as the stool DNA test [@cotter_long-term_2016] could help improve our overall AUC.  Another limitation is that this study drew heavily from those with caucasian ancestry.  The results may not be immediately representative of those with either Asian or African ancestry.  Finally, although our training and test set are relatively large we still run the risk of overfitting or having a model that may not be immediately extrapolateable to other populations.  We've done our best to safeguard against this by not only running 10-fold cross validation but also having over 100 different 80/20 splits to try and mimic the type of variation that might be expected to occur.

Another interesting outcome was that Within figure 3 the before sample model showed better test AUC results then the training set AUC.  This may have occured because the training AUC that was determined from 20 repeated 10 fold cross validation removed samples at random and did not take into account that they were matched samples.  Another potential reason is that the model itself may be overfit since the total number of samples was not that large.  However, the lesion model did not suffer from these discrepencies and similar conclusions can be drawn solely from this model.  Regardless, further independent studies will need to be carried out to verify our findings since not only are we dealing with feces, which could be very different than the communities present on the actual tissue, but also are dealing with correlations that may not be representative of the true pathogensis of disease.

Despite these limitations we think that these findings significantly add to the existing scientific knowledge on CRC and the bacterial microbiome.  The ability for machine learning algorithms to take bacterial microbiome data and successfully lower positive probability after either adenoma or carcinoma removal provides evidence that there are specific signatures, mostly attributable to commensal organisms, associated with these lesions.  It also shows that these algorithms can not only successfully react to successful treatment regimens but also may be able to one day stratify CRC disease risk with a high level of accuracy.



\newpage

## Methods

***Study Design and Patient Sampling***
The sampling and design of the study was similar to that reported in Baxter, et al [@baxter_microbiota-based_2016].  In brief, study exclusion involved those who had already undergone surgery, radiation, or chemotherapy, had colorectal cancer before a baseline fecal sample could be obtained, had IBD, a known hereditary non-polyposis colorectal cancer, or Familial adenomatous polyposis.  Samples used to build the models for prediction were collected either prior to a colonoscopy or between 1 - 2 weeks after.  The bacterial microbiome has been shown to normalize within this time period [@obrien_impact_2013].  Our follow up data set had a total of `r length(rownames(good_metaf))` individuals that not only had a sample as described but also a follow up sample between `r min(good_metaf$time)` - `r max(good_metaf$time)` days after surgery and treatment had been completed. This study was approved by the University of Michigan Institutional Review Board.  All study participants provided informed consent and the study itself conformed to the guidelines set out by the Helsinki Declaration.


***FIT and 16S rRNA Gene Sequencing***
FIT was analyzed as previously published using both OC FIT-CHEK and OC-Auto Micro 80 automated system (Polymedco Inc.) [@daly_evaluation_2013]. 16S rRNA gene sequencing was completed as previously described by Kozich, et al. [@kozich_development_2013].  In brief, DNA extraction used the 96 well Soil DNA isolation kit (MO BIO Laboratories) and an epMotion 5075 automated pipetting system (Eppendorf).  The V4 variable region was amplified and the resulting product was split between three sequencing runs with normal, adenoma, and carcinoma evenly represented on each run.  Each group was randomly assigned to avoid biases based on sample collection location.


***Sequence Processing***
The mothur software package (v1.37.5) was used to process the 16S rRNA gene sequences.  This process has been previously described [@kozich_development_2013].  The general processing workflow using mothur was as follows: Paired-end reads were first merged into contigs, quality filtered, aligned to the SILVA database, screened for chimeras, classified with a naive Bayesian classifier using the Ribosomal Database Project (RDP), and clustered into Operational Taxonomic Units (OTUs) using a 97% similarity cutoff with an average neighbor clustering algorithm.  The numer of sequences for each sample was rarified to `r ifelse(sd(total_sub_Seqs) == 0, paste(mean(total_sub_Seqs)), paste("Error Check Subsampling"))` in an attempt to minimize uneven sampling.    


***Lesion Model Creation***
The Random Forest [@breiman_random_2001] algorithm was used to create the model used for prediction of lesion (adenoma or carcinoma) with the main testing and training of the model completed on a data set of `r length(rownames(metaI))` individuals.  This model was then applied to our follow up data set of `r length(rownames(good_metaf))` individuals.  The model included data on FIT and the bacterial microbiome. Non-binary data was checked for near zero variance and OTUs that had near zero variance were removed.  This pre-processing was performed with the R package caret (v`r paste(packageVersion("caret"))`).  Optimization of the mtry hyperparameter involved taking the samples and making 100 different 80/20 (train/test) splits of the data where normal and lesion were represented in the same proportion within both the whole data set and the 80/20 split.  Each of these splits were then run through 20 repeated 10-fold cross validations to optimize the mtry hyperparameter by maximizing the AUC (Area Under the Curve of the Receiver Operator Characteristic).  This resulting model was then tested on the 20% of the data that was originally held out from this overall process.  Once the ideal mtry was found the entire `r length(rownames(metaI))` sample set was used to create the final Random Forest model on which classifications on the `r length(rownames(good_metaf))`-person cohort was completed.  The default cutoff of 0.5 was used as the threshold to classify individuals as positive or negative for lesion.  The hyperparameter, mtry, defines the number of variables to investigate at each split before a new division of the data is created with the Random Forest model.  


***Before Sample Model Creation***
We also investigated whether a model could be created that could identify before and after surgery samples.  The main difference was that only the `r length(rownames(good_metaf))`-person cohort was used at all stages of model building and classification.  Other than this difference the creation of this model and optimization of the mtry hyperparameter was completed using the same procedure that was used to create the lesion model.  Instead of classifying samples as positive or negative of lesion this model classified samples as positive or negative for being a before surgery sample. 


***Selection of Important OTUs***
In order to assess which variables were most important to all the models we counted the number of times a variable was present in the top 10% of mean decrease in accuracy (MDA) for each of the 100 different 80/20 split models and then filtered this list to variables that were only present more than 50% of the time.  This final collated list of variables was what was considered the most important for the lesion or before sample models.


***Statistical Analysis***
The R software package (v`r paste(version$major, ".", version$minor, sep = "")`) was used for all statisitical analysis.  Comparisons between bacterial community structure utilized PERMANOVA [@anderson_permanova_2013] in the vegan package (v`r paste(packageVersion("vegan"))`).  Comparisons between probabilities as well as overall OTU differences between initial and follow up samples utilized a paired wilcoxson ranked sum test.  Where multiple comparison testing was needed a Benjamini-Hochberg (BH) correction was applied [@benjamini_controlling_1995] and a corrected P-value of less than 0.05 was considered significant.  Unless otherwise stated the P-values reported are those that were BH corrected.  


***Analysis Overview***
Initial and follow up samples were analyzed for differences in alpha and beta diversity.  Next, differences in FIT between initial and follow ups for either adenoma or carcinoma were investigated.  From here, all OTUs that were used in either model were then analyzed using a paired wilcoxson test.  We then investigated the relative abundance of specific previously associated CRC bacteria, specifically, OTUs that taxonomically classified to *Fusobacterium nucleatum*, *Parvimonas micra*, *Peptostreptococcus assacharolytica*, and *Porphyromonas stomatis*.  We wanted to test if there were any differences based on whether the individual had an adenoma or carcinoma.  From here the lesion model was then tested for accuracy in prediction and whether it reduced the positive probability of lesion after surgery.  The most important OTUs for this were used to build a reduced model and it was assessed for similarity to the original model.  We then used the before sample model to assess whether it could classify samples better then the lesion model.  The most important OTUs were then identified from this model and used to create a reduced feature before sample model.  This reduced feature model, as was done with the lesion model, was compared to the full model for loss of accuracy.  Finally, a list of common OTUs were found for the two different models used. 


***Reproducible Methods.***
A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Sze_followUps_2017.  Raw sequences have been deposited into the NCBI Sequence Read Archive (SRP062005) and the necessary metadata can be found at https://www.ncbi.nlm.nih.gov/Traces/study/ and searching the respective SRA study accession.


\newpage


**Figure 1: General Differences between the Adenoma or Carcinoma Group.**  A) A significant difference was found between the adenoma and carcinoma group for thetayc (P-value = `r format(change_summary['thetayc', 'Pvalue'], scientific = FALSE, digits = 3)`).  B) A significant difference was found between the adenoma and carcinoma group for change in FIT (P-value = `r format(change_summary['fit', 'Pvalue'], scientific = TRUE, digits = 3)`). C) NMDS of the intial and follow up samples for the Adenoma group.  D) NMDS of the initial and follow up samples for the Carcinoma group. For C) and D) the teal represents initial samples and the pink represents follow up samples. 


**Figure 2: Previously Associated CRC Bacteria in Initial and Follow up Samples.**  A) Carcinoma initial and follow up samples. There was a significant difference in initial and follow up sample for the OTUs classfied as *Peptostreptococcus stomatis* (P-value = `r format(main_crc_bugs_summary["pept", "crc_BH"], digits = 3)`) and *Porphyromonas asaccharolytica* (P-value = `r format(main_crc_bugs_summary["porp", "crc_BH"], digits = 3)`).  B) Adenoma initial and follow up samples.  There were no significant differences between initial and follow up.


**Figure 3: Graph of the Receiver Operating Characteristic Curve for lesion and Before Sample Models.** The shaded areas represents the range of values of a 100 different 80/20 splits of the test set data using either all variables (grey) or reduced variable (red) models.  The blue line represents the reduced variable model using 100% of the data set.  A) Lesion model.  B) Defore sample model


**Figure 4: Breakdown by Carcinoma and Adenoma of Prediction Results for Lesion and Before Sample Reduced Variable Models**  A) Lesion positive probability adjustment of those with carcinoma from initial to follow up sample  B) Initial follow up positive probability adjustment of those with carcinoma from initial to follow up sample  C) Lesion positive probability adjustment of those with adenoma as well as those with SRN and the probability adjustment from initial to follow up sample.    D) Initial follow up positive probability adjustment of those with adenoma as well as those with SRN and the probability adjustment from initial to follow up sample.  The dotted line represents the threshold used to make the decision of whether a sample was positive or not.


\newpage

**Figure S1: Distribution of P-values from Paired Wilcoxson Analysis of OTUs in Initial versus Follow Up**

**Figure S2: Summary of Important Variables in the Lesion Model** A) MDA of the most important variables in the lesion model.  The black point represents the median and the different colors are the different runs up to 100.  B) The total number of appearances of each variable in the 100 different lesion models.  The cutoff of 50% was used to assess importance. 

**Figure S3: Summary of Important Variables in Before Sample Model** A) MDA of the most important variables in the lesion model.  The black point represents the median and the different colors are the different runs up to 100.  B) The total number of appearances of each variable in the 100 different lesion models.  The cutoff of 50% was used to assess importance. 

**Figure S4: Breakdown by Carcinoma and Adenoma of Prediction Results for Lesion and Before Sample Full Variable Models** A) Lesion positive probability adjustment of those with carcinoma from initial to follow up sample  B) Initial follow up positive probability adjustment of those with carcinoma from initial to follow up sample  C) Lesion positive probability adjustment of those with adenoma as well as those with SRN and the probability adjustment from initial to follow up sample.    D) Initial follow up positive probability adjustment of those with adenoma as well as those with SRN and the probability adjustment from initial to follow up sample.  The dotted line represents the threshold used to make the decision of whether a sample was positive or not.

**Figure S5: Thetayc Graphed Against Time of Follow up Sample from Initial**

\newpage

## Declarations

### Ethics approval and consent to participate

### Consent for publication

### Availability of data and material

### Competing Interests

All authors declare that they do not have any relevent competing interests to report.

### Funding

This study was supported by funding from the National Institutes of Health to P. Schloss (R01GM099514, P30DK034933) and to the Early Detection Research Network (U01CA86400).

### Authors' contributions

All authors were involved in the conception and design of the study. MAS analyzed the data. NTB processed samples and analyzed the data. All authors interpreted the data. MAS and PDS wrote the manuscript. All authors reviewed and revised the manuscript. All authors read and approved the final manuscript.

### Acknowledgements

The authors thank the Great Lakes-New England Early Detection Research Network for providing the fecal samples that were used in this study. 

\newpage

## References




