---
title: ''
csl: microbiome.csl
fontsize: 12pt
output:
  word_document: default
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
geometry: margin=1.0in
bibliography: reference.bib
---

# The Fecal Microbiome Before and After Treatment for Colorectal Adenoma or Carcinoma
```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("dplyr", "reshape2", "scales", "knitr", "rmarkdown"))

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

```

```{r rdata_tables}
# Extra data and meta data needed
good_metaf <- read.csv('../data/process/mod_metadata/good_metaf_final.csv', header = T, stringsAsFactors = F)
metaI <- read.csv('../data/process/mod_metadata/metaI_final.csv', header = T, stringsAsFactors = F)
total_sub_Seqs <- read.delim('../data/process/final.0.03.subsample.shared', header = T) %>% 
  select(contains("Otu0")) %>% rowSums()

# initial microbiome and FIT description data 
change_summary <- read.csv('../data/process/tables/change_theta_fit_summary.csv', header = T, row.names = 1)
alpha_summary <- read.csv('../data/process/tables/alpha_table_summary.csv', header = T, row.names = 1)
beta_summary <- read.csv('../data/process/tables/beta_diver_summary.csv', header = T, row.names = 1)
time_pvalues <- read.csv("../data/process/tables/time_pvalues.csv", header = T, row.names = 1)
time_summary <- read.csv("../data/process/tables/time_summary_data.csv", header = T, row.names = 1)
all_OTU_pair_wilcox <- read.csv("../data/process/tables/OTU_paired_wilcoxson_test.csv", header = T)

# Adenoma Model 
adn_varsMDA <- read.csv('../data/process/tables/adn_reduced_crc_model_top_vars_MDA_Summary.csv', header = T)
adn_imp_vars <- read.csv('../data/process/tables/adn_rf_otu_tax.csv', header = T, row.names = 1)
adn_AUC_run_summary <- read.csv('../data/process/tables/adn_Reduced_ROC_model_summary.csv', header = T)
adn_general_summary <- read.csv('../data/process/tables/adn_reduced_auc_summary.csv', header = T, row.names = 1)

# SRN Model
srn_varsMDA <- read.csv('../data/process/tables/srn_reduced_model_top_vars_MDA_Summary.csv', header = T)
srn_imp_vars <- read.csv('../data/process/tables/srn_rf_otu_tax.csv', header = T, row.names = 1)
srn_AUC_run_summary <- read.csv('../data/process/tables/srn_Reduced_ROC_model_summary.csv', header = T)
srn_general_summary <- read.csv('../data/process/tables/srn_reduced_auc_summary.csv', header = T, row.names = 1)

# CRC Model
crc_varsMDA <- read.csv('../data/process/tables/reduced_crc_model_top_vars_MDA_Summary.csv', header = T)
crc_imp_vars <- read.csv('../data/process/tables/crc_rf_otu_tax.csv', header = T, row.names = 1)
crc_AUC_run_summary <- read.csv('../data/process/tables/crc_Reduced_ROC_model_summary.csv', header = T)
crc_general_summary <- read.csv('../data/process/tables/crc_reduced_auc_summary.csv', header = T, row.names = 1)

# All model information
all_probs_paired_wilcox <- read.csv('../data/process/tables/all_crc_srn_adn_models_wilcox_paired_pvalue_summary.csv', header = T, stringsAsFactors = F, row.names = 4)
all_pred_stat_summary <- read.csv('../data/process/tables/all_crc_srn_adn_models_summary_info.csv', header = T, stringsAsFactors = F) %>% rename(comparison = X)
all_confu_matrix <- read.csv('../data/process/tables/all_crc_srn_adn_models_confusion_summary.csv', header = T, stringsAsFactors = F)

# Common Variable Info
common_vars_summary <- read.csv('../data/process/tables/pvalue_adn_srn_crc_common_imp_vars.csv', header = T, row.names = 1, stringsAsFactors = F)

#Therapy Info
chemo_rad_summary <- read.csv('../data/process/tables/crc_probs_chemo_rad_pvalue_summary.csv', header = T, row.names = 1)
chemo_rad_common_vars_summary <- read.csv('../data/process/tables/chemo_rads_treatment_pvalue_summary.csv', header = T, stringsAsFactors = F)

comb_adn_surg_summary <- read.csv('../data/process/tables/adn_combined_probs_surgery_pvalue_summary.csv', header = T, stringsAsFactors = F, row.names = 1)

comb_adn_surg_common_vars_summary <- read.csv('../data/process/tables/all_adn_surg_pvalue_summary.csv', header = T, stringsAsFactors = F)

```
\vspace{25mm} 

\begin{center}
Running Title: Human Microbiome before and after Colorectal Cancer

\vspace{10mm}

Marc A Sze${^1}$, Nielson T Baxter${^2}$, Mack T Ruffin IV${^3}$, Mary AM Rogers${^2}$, and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI

$2$ Department of Internal Medicine, University of Michigan, Ann Arbor, MI   

$3$ Department of Family Medicine and Community Medicine, Penn State Hershey Medical Center, Hershey, PA    


\end{center}  


\newpage
\linenumbers

## Abstract

**Background:** Colorectal cancer (CRC) is a worldwide health problem and research suggests a correlation between the fecal bacterial microbiome and CRC.  This study tested the hypothesis that treatment for adenoma or carcinoma results in changes to the bacterial community.  Specifically, we tried to identify components within the community that were different before and after treatment of adenoma, advanced adenoma (Screen Relevant Neoplasia (SRN)), and carcinoma.     

**Results:** There was a larger change in the bacterial community in response to treatment for carcinoma versus adenoma (P-value `r ifelse(change_summary["thetayc", "Pvalue_adnVcrc"] < 0.05, invisible("<"), invisible(">"))` 0.05) but not carcinoma versus SRN cases (P-value `r ifelse(change_summary["thetayc", "Pvalue_srnVcrc"] < 0.05, invisible("<"), invisible(">"))` 0.05).  But there was a trend for increasingly less community similarity between initial and follow up samples from adenoma to SRN to carcinoma.  Despite this, no difference was found in the relative abundance of any OTU before and after treatment for adenoma, SRN, or carcinoma groups (P-value `r ifelse(min(all_OTU_pair_wilcox$BH_corrected, na.rm = TRUE) < 0.05, invisible("<"), invisible(">"))` 0.05).  Using Random Forest models to assess whether changes in follow up samples were towards a normal community showed that only those with carcinoma had a significant decrease in positive probability (P-value `r ifelse(all_probs_paired_wilcox["carcinoma", "BH_correction"] < 0.05, invisible("<"), invisible(">"))` 0.05); providing further evidence that treatment has the greatest effect in those with carcinoma. The adenoma model used a total of `r length(rownames(adn_imp_vars))` OTUs, the SRN model used a total of `r length(rownames(srn_imp_vars))` OTUs, and the carcinoma model used a total of `r length(rownames(crc_imp_vars))` OTUs.  A total of `r length(common_vars_summary$otu)` OTUs were common to both models with many classifying to commensal bacteria (e.g. *Lachnospiraceae*, *Bacteroides*, *Anaerostipes*, *Blautia*, and *Dorea*).  Both chemotherapy and radiation did not provide any additional changes to the bacterial community in those treated for carcinoma (Pvalue `r ifelse(min(chemo_rad_summary$bh, na.rm = TRUE) < 0.05, invisible("<"), invisible(">"))` 0.05).     
  
**Conclusions:** Our data partially supports the hypothesis that the bacterial community changes after treatment.  Individuals with carcinoma had more drastic differences to the overall community then those with adenoma.  Common OTUs to all models were overwhelmingly from commensal bacteria, suggesting that these bacteria may be important to initial polyp formation, development of advanced adenoma, and transition to carcinoma.  

\newpage

### Keywords

bacterial microbiome; colorectal cancer; polyps; FIT; post-surgery; risk factors      

\newpage

## Background

Colorectal cancer (CRC) is currently the third most common cause of cancer deaths [@jemal_cancer_2010; @haggar_colorectal_2009].  The rate of disease mortality has seen a significant decrease, thanks mainly to improvements in screening [@jemal_cancer_2010].  However, despite this improvement there are still approximately 50,000 deaths from the disease per year [@haggar_colorectal_2009].  

Recent studies in humans have shown that both the microbiome and specific members within it correlate with CRC pathogenesis [@zackular_manipulation_2016; @arthur_microbial_2014].  Further, bacterial communities have been observed to be altered between normal and tumor tissue [@dejea_microbiota_2014].  Mouse models of CRC have further demonstrated the importance of the microbiome, both on a community [@zackular_gut_2013; @zackular_manipulation_2016] and species level [@arthur_microbial_2014], for tumorgenesis.  Collectively, these studies provide a tantalizing link between our gut bacteria and CRC and suggest that biomarkers using our microbes could be developed.  Indeed, builidng models using 16S rRNA gene sequencing along with clinical tests such as Fecal Immunoglobulin Test (FIT) result in good predictions of CRC [@baxter_microbiota-based_2016; @zeller_potential_2014].  Although these studies show how our gut bacteria can impact CRC progression via a changed community or invasion by more inflammatory bacteria [@flynn_metabolic_2016].  They provide very little information as to whether these communities change and rebound towards normal after successful treatment of adenoma, advanced adenoma (Screen Relevent Neoplasia (SRN)), or carcinoma.

Providing an answer to this question is important because it has far reaching implications on both how the bacterial community causes the formation of more polyps [@zackular_manipulation_2016; @arthur_microbial_2014] and the ability to be able to use the microbiome as a predictive screening tool [@baxter_microbiota-based_2016; @zeller_potential_2014].  Understanding polyp formation and transition to SRN and then carinoma is crucial to being able to understand how to prevent CRC occurence.  Response of the community to treatment is also eqaully important to predictive models designed for screening purposes since an unresponsive community would provide little additional information for important events, such as recurrence [@hassan_efficacy_2016].    

Using pre- (initial) and post- (follow up) treatment samples we tested the hypothesis that treatment causes detectable changes to the microbiome in those with adenoma, SRN, and carincoma.  First, we assessed differences between initial and follow up samples in adenoma, SRN, or carinoma using alpha or beta diversity metrics.  Second, we explored whether models built to classify adenoma, SRN, or carcinoma versus normal were able to identify specific community members that differed between initial and follow up.  We also used these models to assess whether changes in the community were toward a more normal micorbiome.  Finally, we assessed both whether surgery for adenomas and SRN provided larger community changes or whether chemotherapy or radiation provided additive changes to the microbiome over surgerical resection. This study helps to provide evidence as to whether treatment can influence the community and if the CRC microbiome, identified in previous studies, persists after such interventions.    



\newpage

## Results


***The Bacterial Community:*** 
Within our `r length(rownames(good_metaf))`-person cohort we tested whether those with adenoma (n = `r change_summary["thetayc", "adn_n"]`), SRN (n = `r change_summary["thetayc", "srn_n"]`), or carcinoma (n = `r change_summary["thetayc", "crc_n"]`) had any broad differences between their initial and follow up samples.  We found that those with carcinoma had a more dissimilar bacterial community between their initial and follow up sample than those with adenoma (P-value `r ifelse(change_summary["thetayc", "Pvalue_adnVcrc"] < 0.001, invisible("<"), invisible(">"))` 0.001) [Figure 1A].  Although no significant differences were observed between SRN and carcinoma there was an increase in the dissimilarity of the initial and follow up samples from adenoma (`r paste(format(change_summary["thetayc", "adn_mean"], digits = 2), label = "\u00B1", format(change_summary["thetayc", "adn_SD"], digits = 2))` (mean `r paste(label = "\u00B1")` SD)) to SRN (`r paste(format(change_summary["thetayc", "srn_mean"], digits = 2), label = "\u00B1", format(change_summary["thetayc", "srn_SD"], digits = 2))`) to carcinoma (`r paste(format(change_summary["thetayc", "crc_mean"], digits = 2), label = "\u00B1", format(change_summary["thetayc", "crc_SD"], digits = 2))`) [Figure 1A].  The bacterial community structure before and after surgery was visualized using NMDS for adenoma [Figure 1B] (PERMANOVA `r ifelse(beta_summary["Adenoma", "PERMANOVA"] < 0.05, invisible("<"), invisible(">"))` 0.05), SRN [Figure 1C] (PERMANOVA `r ifelse(beta_summary["SRN", "PERMANOVA"] < 0.05, invisible("<"), invisible(">"))` 0.05), and carcinoma [Figure 1D] (PERMANOVA `r ifelse(beta_summary["Carcinoma", "PERMANOVA"] < 0.05, invisible("<"), invisible(">"))` 0.05).  Interestingly, when initial and follow up samples were compared, regardless of whether the lesions were adenoma or carcinoma, there was no significant overall difference in beta diversity (PERMANOVA `r ifelse(beta_summary["All", "PERMANOVA"] < 0.05, invisible("<"), invisible(">"))` 0.05).  There was no difference between initial and follow up samples when investigating alpha diversity metrics for adenoma, SRN, or carcinoma for any metric tested [Table S1].  Additionally, there was also no difference in the relative abundance of any OTU between initial and follow up samples for adenoma, SRN, or carcinoma only [Figure S1].


***Adenoma Model***
The range of model AUC's from 100 runs of 20 repeated 10 fold cross-validation was `r format(min(adn_AUC_run_summary$ROC), digits = 2)` - `r format(max(adn_AUC_run_summary$ROC), digits = 2)` with the AUC of the model used for classification havin an AUC of `r format(adn_general_summary["full", "AUC"], digits = 2)` [Figure S2A].  There was a total of `r length(rownames(adn_imp_vars))` OTUs in this model with the vast majority classifying to bacteria typically thought of as commensal [Figure S3A]. There was a significant difference between the actual and model predicted group stratification calls (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "adn" & comparison == "McnemarPValue")[, "all"] < 0.05, invisible("<"), invisible(">"))` 0.05).  There was no significant decrease in the positive probability of adenoma between initial and follow up samples (P-value `r ifelse(all_probs_paired_wilcox["adenoma", "BH_correction"] < 0.05, invisible("<"), invisible(">"))` 0.05) [Figure 2A].    


***SRN Model***
The range of model AUC's from 100 runs of 20 repeated 10 fold cross-validation was `r format(min(srn_AUC_run_summary$ROC), digits = 2)` - `r format(max(srn_AUC_run_summary$ROC), digits = 2)` with the AUC of the model used for classification having an AUC of `r format(srn_general_summary["full", "AUC"], digits = 2)` [Figure S2B].  There was a total of `r length(rownames(srn_imp_vars))` OTUs in the SRN model [Figure S3B].  Similar to the adenoma model the vast majority of OTUs classified to bacteria typically thought of as commensal. Also similar to the adenoma model there was a significant difference between the actual and model predicted group stratification calls (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "SRN" & comparison == "McnemarPValue")[, "all"] < 0.05, invisible("<"), invisible(">"))` 0.05).  There was no significant decrease in the positive probability of SRN between initial and follow up samples (P-value `r ifelse(all_probs_paired_wilcox["SRN", "BH_correction"] < 0.05, invisible("<"), invisible(">"))` 0.05) [Figure 2B].  


***Carcinoma Model***
The range of model AUC's from 100 runs of 20 repeated 10 fold cross-validation was `r format(min(crc_AUC_run_summary$ROC), digits = 2)` - `r format(max(crc_AUC_run_summary$ROC), digits = 2)` with the AUC of the model used for classification being `r format(crc_general_summary["full", "AUC"], digits = 2)` [Figure S2C].  Interestingly, the AUCs improved from adenoma to SRN to carcinoma [Figure 2].  There was a total of `r length(rownames(crc_imp_vars))` OTUs in the carinoma model [Figure S3C].  Similar to the adenoma and SRN models the vast majority of OTUs classified to bacteria typically thought of as commensal but OTUs that also classified to *Fusobacterium*, *Porphyromonas*, and *Parvimonas* appear to be important for carcinoma classification [Figure S3C].  Also similar to the adenoma and SRN models there was a significant difference between the actual and model predicted group stratification calls (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "crc" & comparison == "McnemarPValue")[, "all"] < 0.05, invisible("<"), invisible(">"))` 0.05).  There was a significant decrease in the positive probability of carcinoma between initial and follow up samples (P-value `r ifelse(all_probs_paired_wilcox["carcinoma", "BH_correction"] < 0.05, invisible("<"), invisible(">"))` 0.05) [Figure 2C]; suggesting that the carcinoma samples changed towards normal after treatment, unlike either adenoma or SRN.  The one indivdiual still positive for carcinoma after treatment had an increase in carcinoma positive proability on follow up [Figure 2C]. 


***Adenoma, SRN, and Carcinoma Common OTUs***
We next wanted to know what predictors within the adenoma, SRN, and carcinoma models were similar to each other.  The main purpose was to identify which OTUs could be important at all three stages of disease.  When we compared the three different models with each other there were a total of `r length(rownames(common_vars_summary))` common OTUs.  Some of the most common taxonomic identifications belonged to *Bacteroides*, *Blautia*, *Anaerostipes*, *Lachnospiraceae*, and *Dorea*.  These along with the vast majority of the OTUs that were common between these models had classifications to bacteria typically thought of as commensal [Table S2].    


***Treatment Affects on Community***
After observing these these changes from treatment we assessed whether chemotherapy or radiation, in the carcinoma group, and surgery, in the adenoma group, impacted the observed results.  In the carcinoma group neither chemotherapy or radiation provided any additive change from initial sample over surgery alone (P-value `r ifelse(min(chemo_rad_summary$bh, na.rm = TRUE) < 0.05, invisible("<"), invisible(">"))` 0.05) [Table S3]. For the adenoma group there was a single difference in observed OTUs (sobs) between those that received surgery and those that did not (P-value `r ifelse(min(comb_adn_surg_summary$bh, na.rm = TRUE) < 0.05, invisible("<"), invisible(">"))` 0.05) [Table S4]. For the surgery comparison, adenoma and SRN were combined due to the low number of surgery occurances in these two groups.  There was no difference in the proportion of those receiving surgery between the adenoma and SRN groups (P-value `r ifelse(comb_adn_surg_summary["prop_surg_adn_srn", "pvalue"] < 0.05, invisible("<"), invisible(">"))` 0.05).  This data suggests that microbiome changes observed in the carcinoma group were mostly a result from surgery and not from chemotherapy or radiation. 


\newpage


## Discussion
This study builds upon previous work from numerous labs that have considered both how the bacterial community between those with and without CRC differ and how it might be used as an early screening tool [@yu_metagenomic_2017; @baxter_microbiota-based_2016; @zeller_potential_2014; @zackular_human_2014; @warren_co-occurrence_2013].  Here we show that the bacterial community changes towards normal after treatment for carcinoma and that chemotherapy and radiation did not provide an additive change.  Although some of the important OTUs classified to genera from bacteria considered the usual suspects (e.g. *Fusobacterium*, *Porphyromonas*, and *Parvimonas*) many did not.  The majority of important OTUs had taxonomic classifications for resident gut microbes and were common for the adenoma, SRN, and carcinoma models.  This suggests that members within the commensal community may be the first that change during CRC pathogenesis.  These subtle changes, in turn, could be the first step in allowing more inflammatory bacteria to gain a foothold within the colon [@flynn_metabolic_2016].

Unlike previous studies on the microbiome and CRC, ours focuses both on identifying commonalities and differences within adenoma, SRN, and carcinoma groups before and after treatment.  Although there were differences for genera associated with specific bacterium linked with CRC [Figure S3].  The majority of important OTUs taxonmically classified to commensal bacteria [Figure S3].  Although these changes may be subtle, due to the lack of signifcant difference in the bacterial community before and after treatment in adenoma and SRN [Figure 1], they support the hypothesis that the first members of the community to change and potentially stay changed even after treatment are those that are commensal bacteria.   

Many of the common OTUs that we identified taxnomically classified to potential butyrate producers (e.g. *Clostridiales*, *Roseburia*,  and *Anaerostipes*) [Table S2].  Other OTUs classified to bacteria that are inhibited by polyphenols (e.g. *Bacteroides*). Both butyrate and polyphenols are thought to be protective against cancer, in part by reducing inflammation [@louis_gut_2014].  These protective compounds are derived from the breakdown of fiber, fruits, and vegetables by resident gut microbes. One example of this potential diet-microbiome-inflammation-polyp axis is that *Bacteroides*, which was highly prevalent in our models, are known to be increased in those with high non-meat based protein consumption [@zhu_intake_2016].  High protein consumption in general has been linked with an increased CRC risk [@mu_colonic_2016].  Conversely, *Bacteroides* are inhibited by polyphenols which are derived from fruits and vegetables [@ozdal_reciprocal_2016].  Our data fits with the hypothesis that the microbial metabolites from breakdown products within our own diet could not only help to shape the existing community but also have an effect on CRC risk and disease progression.

A limitation, in our study, was that there was a significant difference in the time elapsed in the collection of the follow up sample between adenoma or SRN versus carcinoma (P-value `r ifelse(time_pvalues["adnVcrc", "bh"] < 0.05 & time_pvalues["srnVcrc", "bh"] < 0.05, invisible("<"), invisible(">"))` 0.05), with time passed being less for adenoma (`r paste(format(time_summary["adenoma", "mean_days"], digits = 0), label = "\u00B1", format(time_summary["adenoma", "sd_days"], digits = 0))` days) and SRN (`r paste(format(time_summary["srn", "mean_days"], digits = 0), label = "\u00B1", format(time_summary["srn", "sd_days"], digits = 0))`) than carcinoma (`r paste(format(time_summary["carcinoma", "mean_days"], digits = 0), label = "\u00B1", format(time_summary["carcinoma", "sd_days"], digits = 0))`).  These results would indicate that some of the differences observed between the carcinoma and adenoma groups could be due to differences in collection time. Specifically, it could confound the observation that carcinomas changed more than adenomas [Figure 1]. However, there are two reasons that this may not be the case.  First, the SRN group did not have a significant difference and the collection time of their follow up sample was less than the adenoma group.  Second, this confounding would not affect the observations where models were used since they were built using a different cohort [Figures 2 & S2-S3].

Another limitation was that we do not know whether individuals who were still classified as positive by the carcinoma model eventually had a subsequent CRC recurrence. This information would help to strengthen the case for this model keeping numerous individuals above the cutoff threshold even though at follow up they were diagnosed as no longer having carcinoma. This study also drew heavily from those with Caucasian ancestry making it possible that the observations may not be representative of those with either Asian or African ancestry. Although our training and test set are relatively large we still run the risk of over-fitting or having a model that may not be representative of other populations. We've done our best to safeguard against this by not only running 10-fold cross validation but also having over 100 different 80/20 splits to try and mimic the type of variation that might be expected to occur.
  
Despite these shortcomings our findings add to the existing scientific knowledge on CRC and the microbiome: That there is a measurable difference in the bacterial community after adenoma, SRN or carcinoma treatment. Further, the ability for machine learning algorithms to take OTU data and successfully lower positive probability of carcinoma after treatment provides evidence that there are specific signatures, attributable to both inflammtory and resident commensal organisms, associated with treatment. Our data provides evidence that commensal bacteria may be important in the development of polyps and potentially the transition from adenoma to carcinoma.



\newpage

## Methods

***Study Design and Patient Sampling:***
Sampling and design have been previously reported in Baxter, et al [@baxter_microbiota-based_2016]. Briefly, study exclusion involved those who had already undergone surgery, radiation, or chemotherapy, had colorectal cancer before a baseline fecal sample could be obtained, had IBD, a known hereditary non-polyposis colorectal cancer, or familial adenomatous polyposis. Samples used to build the models for prediction were collected either prior to a colonoscopy or between 1 - 2 weeks after. The bacterial community has been shown to normalize back to a pre-colonscopy community within this time period [@obrien_impact_2013]. Our training cohort consisted of a total of `r length(rownames(filter(metaI, !(sample %in% c(good_metaf$initial, good_metaf$followUp)))))` individuals [Table 1].  Our study cohort consisted of `r length(rownames(good_metaf))` individuals with an initial sample as described and a follow up sample obtained between `r min(good_metaf$time)` - `r max(good_metaf$time)` days after treatment of lesion [Table 2]. This study was approved by the University of Michigan Institutional Review Board. All study participants provided informed consent and the study itself conformed to the guidelines set out by the Helsinki Declaration.


***16S rRNA Gene Sequencing:***
Sequencing was completed as described by Kozich, et al. [@kozich_development_2013]. DNA extraction used the 96-well Soil DNA isolation kit (MO BIO Laboratories) and an epMotion 5075 automated pipetting system (Eppendorf). The V4 variable region was amplified and the resulting product was split between three sequencing runs with normal, adenoma, and carcinoma evenly represented on each run. Each group was randomly assigned to avoid biases based on sample collection location.  The initial and follow up samples were sequenced on the same run.


***Sequence Processing:***
The mothur software package (v1.37.5) was used to process the 16S rRNA gene sequences and has been previously described [@kozich_development_2013]. The general workflow using mothur was: Paired-end reads were first merged into contigs, quality filtered, aligned to the SILVA database, screened for chimeras, classified with a naive Bayesian classifier using the Ribosomal Database Project (RDP), and clustered into Operational Taxonomic Units (OTUs) using a 97% similarity cutoff with an average neighbor clustering algorithm. The number of sequences for each sample was rarefied to `r ifelse(sd(total_sub_Seqs) == 0, paste(mean(total_sub_Seqs)), paste("Error Check Subsampling"))` to minimize uneven sampling.


***Lesion Model Creation:***
The Random Forest [@breiman_random_2001] algorithm was used to create the model used for prediction of lesion (adenoma or carcinoma) with the main training and testing of the model completed on an independent data set of  `r length(rownames(metaI)) - length(rownames(good_metaf))` individuals. This model was then applied to our `r length(rownames(good_metaf))`-person cohort. It should be noted that all individuals with an adenoma or carcinoma were grouped together to form the lesion group and the model was not created to find differences between normal, adenoma, and carcinoma but rather differences between both adenoma and carcinoma versus normal.

The model included only OTU data obtained from 16S rRNA sequencing. Non-binary data was checked for near zero variance and OTUs that had near zero variance were removed. This pre-processing was performed with the R package caret (v`r paste(packageVersion("caret"))`). Optimization of the mtry hyper-parameter involved making 100 different 80/20 (train/test) splits of the data where normal and lesion were represented in the same proportion within both the whole data set and the 80/20 split. For each a 20 repeated 10-fold cross validation was performed on 80% component to optimize the mtry hyper-parameter by maximizing the AUC (Area Under the Curve of the Receiver Operator Characteristic). The resulting model was then tested on the hold out data obtained from the 20% component. Assessment of the most important OTUs to the model involved counting the number of times an OTU was present in the top 10% of mean decrease in accuracy (MDA) for each of the 100 different splits run.  This was then followed with filtering of this list to variables that were only present in more than 50% of these 100 runs. The final collated list of variables was then run through the mtry optimization again. Once the ideal mtry was found the entire `r length(rownames(metaI)) - length(rownames(good_metaf))` sample set was used to create the final Random Forest model on which classifications on the `r length(rownames(good_metaf))`-person cohort was completed.
  
The default cutoff of 0.5 was used as the threshold to classify individuals as positive or negative for lesion. The hyper-parameter, mtry, defines the number of variables to investigate at each split before a new division of the data was created with the Random Forest model.


***Treatment Model Creation:***
We also investigated whether a model could be created that could identify pre- (initial) and post- (follow up) treatment samples. The main difference was that only the `r length(rownames(good_metaf))`-person cohort was used at all stages of model building and classification. Other than this difference the creation of this model and optimization of the mtry hyper-parameter was completed using the same procedure as was used for the lesion model. Instead of classifying samples as positive or negative of lesion this model classified samples as positive or negative for being an initial sample prior to treatment.


***Statistical Analysis:***
The R software package (v`r paste(version$major, ".", version$minor, sep = "")`) was used for all statistical analysis.  Comparisons between bacterial community structure utilized PERMANOVA [@anderson_permanova_2013] in the vegan package (v`r paste(packageVersion("vegan"))`).  Comparisons between probabilities as well as overall OTU differences between initial and follow up samples utilized a paired Wilcoxson ranked sum test.  Where multiple comparison testing was appropriate, a Benjamini-Hochberg (BH) correction was applied [@benjamini_controlling_1995] and a corrected P-value of less than 0.05 was considered significant.  Unless otherwise stated the P-values reported are those that were BH corrected.  


***Analysis Overview:***
We first tested for any differences based on whether the individual had an adenoma or carcinoma. This was done by testing initial and follow up samples for differences in alpha and beta diversity, testing all OTUs, and investigating the relative abundance of genera from previously associated CRC bacteria (*Fusobacterium*, *Parvimonas*, *Peptostreptococcus*, and *Porphyromonas*). Next, the lesion model was tested for accuracy in prediction and whether it reduced the positive probability of lesion in follow up samples. We then used the treatment model to assess whether it could classify samples better than the lesion model and whether it could reduce the positive probability of an initial sample in the follow up samples.  Common OTUs were found for the two different models used to assess which were important for both models.  Finally, differences between those receiving chemotherapy and radiation versus those who received neither were tested.   


***Reproducible Methods:***
A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Sze_followUps_2017.  Raw sequences have been deposited into the NCBI Sequence Read Archive (SRP062005 and SRP096978) and the necessary metadata can be found at https://www.ncbi.nlm.nih.gov/Traces/study/ and searching the respective SRA study accession.


\newpage


**Figure 1: General Differences between Adenoma, SRN, and Carcinoma Groups After Treatment.**  A) A significant difference was found between the adenoma and carcinoma group for thetayc (P-value = `r format(change_summary['thetayc', 'Pvalue'], scientific = FALSE, digits = 3)`).  Advanced adenomas are denoted as Screen Relevant Neoplasia (SRN).  B) NMDS of the initial and follow up samples for the adenoma group.  C) NMDS of the initial and follow up samples for the carcinoma group.  


**Figure 2: Treatment Response Based on Models Built for Adenoma, SRN, or Carcinoma.** A) Positive probability change from initial to follow up sample in those with adenoma.  B) Positive probability change from initial to follow up sample in those with SRN. C) Positive probability change from initial to follow up sample in those with carcinoma..


**Figure 3: The Treatment Model.**  B)   C) Positive probability change from initial to follow up sample in those with carcinoma.  D) Positive probability change from initial to follow up sample of those with adenoma or advanced adenoma (Screen Relevant Neoplasia (SRN)).


\newpage

**Table 1: Demographic Data of Training Cohort**

**Table 2: Demographic Data of Pre and Post Treatment Cohort**


\newpage


**Figure S1: Distribution of P-values from Paired Wilcoxson Analysis of All OTUs Before and After Treatment**

**Figure S2: ROC Curves of the Adenoma, SRN, and Carcinoma Models.**  A) Adenoma ROC curve: The light greenshaded areas represent the range of values of a 100 different 80/20 splits of the test set data and the dark green line represents the model using 100% of the data set and what was used for subsequent classification.  B) SRN ROC curve: The light yellow shaded areas represent the range of values of a 100 different 80/20 splits of the test set data and the dark yellow line represents the model using 100% of the data set and what was used for subsequent classification.  C) Carcinoma ROC curve: The light red shaded areas represent the range of values of a 100 different 80/20 splits of the test set data and the dark red line represents the model using 100% of the data set and what was used for subsequent classification.

**Figure S3: Summary of Important Variables for the Adenoma, SRN, and Carcinoma Models.**  A) MDA of the most important variables in the adenoma model.  The dark green point represents the mean and the lighter green points are the value of each of the 100 different runs.  B) Summary of Important Variables in the SRN model.  MDA of the most important variables in the SRN model.  The dark yellow point represents the mean and the lighter yellow points are the value of each of the 100 different runs.  C) MDA of the most important variables in the carcinoma model.  The dark red point represents the mean and the lighter redpoints are the value of each of the 100 different runs.

\newpage

## Declarations

### Ethics approval and consent to participate

The University of Michigan Institutional Review Board approved this study, and all subjects provided informed consent. This study conformed to the guidelines of the Helsinki Declaration.

### Consent for publication

Not applicable.

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Sze_followUps_2017.  Raw sequences have been deposited into the NCBI Sequence Read Archive (SRP062005 and SRP096978) and the necessary metadata can be found at https://www.ncbi.nlm.nih.gov/Traces/study/ and searching the respective SRA study accession.

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

This study was supported by funding from the National Institutes of Health to P. Schloss (R01GM099514, P30DK034933) and to the Early Detection Research Network (U01CA86400).

### Authors' contributions

All authors were involved in the conception and design of the study. MAS analyzed the data. NTB processed samples and analyzed the data. All authors interpreted the data. MAS and PDS wrote the manuscript. All authors reviewed and revised the manuscript. All authors read and approved the final manuscript.

### Acknowledgements

The authors thank the Great Lakes-New England Early Detection Research Network for providing the fecal samples that were used in this study. We would also like to thank Amanda Elmore for reviewing and correcting code error and providing feedback on manuscript drafts.  We would also like to thank Nicholas Lesniak for providing feedback on manuscript drafts.

\newpage

## References




