---
title: ""
bibliography: reference.bib
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
csl: microbiome.csl
fontsize: 12pt
geometry: margin=1.0in
---

# The Fecal Microbiome Before and After Treatment for Colorectal Adenoma or Carcinoma
```{r knitr_settings, eval=TRUE, echo=FALSE, warning=FALSE, message=FALSE}

source('../code/functions.R')

loadLibs(c("dplyr", "reshape2", "scales", "knitr", "rmarkdown"))

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)
opts_chunk$set("message" = FALSE)

```

```{r rdata_tables}
# Extra data and meta data needed
good_metaf <- read.csv('../results/tables/mod_metadata/good_metaf_final.csv', header = T, stringsAsFactors = F)
metaI <- read.csv('../results/tables/mod_metadata/metaI_final.csv', header = T, stringsAsFactors = F)
total_sub_Seqs <- read.delim('../data/process/final.0.03.subsample.shared', header = T) %>% 
  select(contains("Otu0")) %>% rowSums()

# initial microbiome and FIT description data 
change_summary <- read.csv('../results/tables/change_theta_fit_summary.csv', header = T, row.names = 1)
alpha_summary <- read.csv('../results/tables/alpha_table_summary.csv', header = T, row.names = 1)
beta_summary <- read.csv('../results/tables/beta_diver_summary.csv', header = T, row.names = 1)
time_pvalues <- read.csv("../results/tables/time_pvalues.csv", header = T, row.names = 1)
time_summary <- read.csv("../results/tables/time_summary_data.csv", header = T, row.names = 1)
all_OTU_pair_wilcox <- read.csv("../results/tables/OTU_paired_wilcoxson_test.csv", header = T)

# Main CRC bacteria differences between Adenoma and Carcinoma
main_crc_bugs_summary <- read.csv('../results/tables/adn_crc_maybe_pvalue_summary.csv', header = T, row.names = 1)
crc_bug_info <- read.csv("../results/tables/adn_crc_maybe_diff.csv", header = T, stringsAsFactors = F)

# Lesion Model 
lesion_top10_varsMDA <- read.csv('../results/tables/reduced_lesion_model_top_vars_MDA_Summary.csv', header = T)
lesion_imp_vars <- read.csv('../results/tables/rf_otu_tax.csv', header = T, row.names = 1)
lesion_AUC_run_summary <- read.csv('../results/tables/Reduced_ROC_model_summary.csv', header = T)

# IF Model
IF_top10_varsMDA <- read.csv('../results/tables/reduced_IF_model_top_vars_MDA_Summary.csv', header = T)
IF_imp_vars <- read.csv('../results/tables/if_rf_otu_tax.csv', header = T, row.names = 1)
IF_AUC_run_summary <- read.csv('../results/tables/reduced_IF_ROC_model_summary.csv', header = T)

# All model information
all_probs_paired_wilcox <- read.csv('../results/tables/all_models_wilcox_paired_pvalue_summary.csv', header = T, stringsAsFactors = F)
all_pred_stat_summary <- read.csv('../results/tables/all_models_summary_info.csv', header = T, stringsAsFactors = F)
all_confu_matrix <- read.csv('../results/tables/all_models_confusion_summary.csv', header = T, stringsAsFactors = F)

# Common Variable Info
common_vars_summary <- read.csv('../results/tables/pvalue_IF_lesion_common_imp_vars.csv', header = T, row.names = 1, stringsAsFactors = F)

#Therapy Info
chemo_rad_summary <- read.csv('../results/tables/probs_chemo_rad_pvalue_summary.csv', header = T, row.names = 1)

```
\vspace{25mm} 

\begin{center}
Running Title: Human Microbiome before and after Colorectal Cancer

\vspace{10mm}

Marc A Sze${^1}$, Nielson T Baxter${^2}$, Mack T Ruffin IV${^3}$, Mary AM Rogers${^2}$, and Patrick D Schloss${^1}$${^\dagger}$

\vspace{20mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI

$2$ Department of Internal Medicine, University of Michigan, Ann Arbor, MI   

$3$ Department of Family Medicine and Community Medicine, Penn State Hershey Medical Center, Hershey, PA    


\end{center}  


\newpage
\linenumbers

## Abstract

**Background:** Colorectal cancer (CRC) continues to be a worldwide health problem with previous research suggesting that a link may exist between the fecal bacterial microbiome and CRC.  The overall objective of our study was to test the hypothesis that changes in the bacterial microbiome occur after lesion (i.e. adenoma or carcinoma) removal.  Specifically, we wanted to identify what within the community was different before and after removal of said lesion.     

**Results:** The bacterial microbiome changed more in response to lesion removal in carcinoma cases compared to adenoma cases (P-value `r ifelse(change_summary['thetayc', 'Pvalue'] < 0.05, paste("<"), paste(">"))` 0.05).  There was no difference for either the adenoma or carcinoma group in the relative abundance of any OTU between pre and post lesion removal (P-value `r ifelse(min(filter(all_OTU_pair_wilcox, analysis == "ALL")[, 'BH_corrected']) < 0.05, paste("<"), paste(">"))` 0.05).  A model built to classify lesion had an AUC range of `r format(min(lesion_AUC_run_summary$ROC), digits = 3)` - `r format(max(lesion_AUC_run_summary$ROC), digits = 3)` while a model built to classify initial samples had an AUC range of `r format(min(IF_AUC_run_summary$ROC), digits = 3)` - `r format(max(IF_AUC_run_summary$ROC), digits = 3)`.  Follow up samples had a decrease in the positive probability of lesion (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type == "red_lesion" & comparison =="lesion")[, "BH_correction"] < 0.05, paste("<"), paste(">"))` 0.05) suggesting a movement towards a more normal bacterial community.  The initial sample model had a decrease in positive probability for the follow up samples to be an initial sample (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type == "red_IF" & comparison =="lesion")[, "BH_correction"] < 0.05, paste("<"), paste(">"))` 0.05).  The lesion model used a total of `r length(lesion_top10_varsMDA$variable)` variables while the initial sample model used a total of `r length(IF_top10_varsMDA$variable)` variables.  A total of `r length(common_vars_summary$otu)` OTUs were common to both models with the majority of these classifying to commensal bacteria (e.g. *`r filter(common_vars_summary, otu == "Otu000028")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000018")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000009")[, "lowest_ID"]`*, and *`r filter(common_vars_summary, otu == "Otu000005")[, "lowest_ID"]`*).   
  
**Conclusions:** Our data supports the hypothesis that there are changes in the bacterial microbiome following colorectal cancer lesion removal.  Individuals with carcinoma have more drastic differences to the overall community then those with adenoma.  Changes to commensal bacteria were some of the most important variables for model classification, suggesting that these bacteria may be central to initial polyp formation and transition to carcinoma.  

\newpage

### Keywords

bacterial microbiome; colorectal cancer; polyps; FIT; post surgery; risk factors      

\newpage

## Background

Colorectal cancer (CRC) continues to be a leading cause of cancer related deaths and is currently the third most common cause of cancer deaths [@jemal_cancer_2010; @haggar_colorectal_2009].  Over the last few years the rate of disease mortality has seen a significant decrease, thanks mainly to improvements in screening [@jemal_cancer_2010].  However, despite this improvement there are still approximately 50,000 deaths from the disease per year [@haggar_colorectal_2009].  

Over the last few years a few studies have shown how this bacterial microbiome [@zackular_manipulation_2016] or specific members within it [@arthur_microbial_2014] could be directly involved with the pathogenesis of CRC.  There has also been research into how the bacterial microbiome could be altered directly on tumor tissue itself [@dejea_microbiota_2014].  These studies have helped to provide a tantalizing link between the bacterial microbiome and CRC.  Building off of these findings, there has been promising work on the bacterial microbiome and its ability to complement existing screening methods such as Fecal Immunoglobulin Test (FIT) or act alone as a screening tool [@baxter_microbiota-based_2016; @zeller_potential_2014].  Although these studies suggest that the bacterial microbiome might change after treatment there remains limited information on the bacterial microbiome response to lesion (adenoma or carcinoma) removal.

In this study we tested the hypothesis that the bacterial microbiome changes between pre (initial) and post (follow up) removal of lesion (adenoma or carcinoma).  We analyzed changes in alpha and beta diversity as well as specific Operational Taxonomic Units (OTUs) relative abundance.  We utilized Random Forest to build two models.  The first model was built to classify lesion and normal samples while the second was built to classify initial and follow up samples.  Subsequent observations on how these models, as well as specific OTUs within them, performed pre and post lesion removal helped inform us as to whether initial and follow up samples were changing and whether it was towards a more normal bacterial microbiome or not.  We also investigated the two models for similar important OTUs to identify the crucial OTUs for not only classifying lesion or normal samples but also initial and follow up.



\newpage

## Results

***Bacterial Community and FIT:***
Within our `r length(rownames(good_metaf))` person cohort we first wanted to test whether there were any broad differences between initial and follow up samples based on lesion being either adenoma (n = `r length(rownames(filter(good_metaf, Diagnosis == "adenoma")))`) or carcinoma (n = `r length(rownames(filter(good_metaf, Diagnosis != "adenoma")))`).  We found that the bacterial community in those with carcinoma were more dissimilar to their initial sample than those with adenoma (P-value `r ifelse(change_summary['thetayc', 'Pvalue'] < 0.001, paste("<"), paste(">"))` 0.001) [Figure 1A]. We also found that there were larger changes in fecal blood (measured by FIT) for those with carcinoma versus adenoma (P-value `r ifelse(change_summary['fit', 'Pvalue'] < 0.0001, paste("<"), paste(">"))` 0.0001) [Figure 1B].  The bacterial community structure before and after surgery was visualized using NMDS for both adenoma [Figure 1C] (PERMANOVA `r ifelse(beta_summary['Adenoma', 'PERMANOVA'] < 0.05, paste("<"), paste(">"))` 0.05) and carcinoma [Figure 1D] (PERMANOVA `r ifelse(beta_summary['Carcinoma', 'PERMANOVA'] < 0.05, paste("<"), paste(">"))` 0.05).  Interestingly, when initial and follow up samples were compared, regardless of whether the lesions were adenoma or carcinoma, there was no significant overall difference in beta diversity (PERMANOVA `r ifelse(beta_summary['All', 'PERMANOVA'] < 0.05, paste("<"), paste(">"))` 0.05).  When investigating alpha diversity metrics there was no difference found between initial and follow up samples for lesion, adenoma only, or carcinoma only for any metric tested [Table S1].  We also observed that there was no difference in the relative abundance of any OTU between initial and follow up samples for lesion, adenoma only, or carcinoma only [Figure S1].


***Carcinoma Associated Bacteria:***
Previous literature has suggested that a number of oral microbes may be important in CRC pathogenesis [@flynn_metabolic_2016].  So we next examined whether there were changes in previously well described carcinoma associated OTUs, such as *Porphyromonas asaccharolytica* (`r unique(filter(crc_bug_info, tax_id == "Porphyromonas")[, "otu"])`), *Fusobacterium nucleatum* (`r unique(filter(crc_bug_info, tax_id == "Fusobacterium")[, "otu"])`), *Parvimonas micra* (`r unique(filter(crc_bug_info, tax_id == "Parvimonas")[, "otu"])`), and *Peptostreptococcus stomatis* (`r unique(filter(crc_bug_info, tax_id == "Peptostreptococcus")[, "otu"])`).  We first observed that only a small percentage of those with adenoma or carcinoma were positive or had an appreciable relative abundance for any of these respective OTUs [Figure 2].  Despite this, those with carcinoma had a decrease in relative abundance from initial to follow up for *Parvimonas micra* (P-value `r ifelse(main_crc_bugs_summary["parv", "crc_BH"] < 0.05, paste("<"), paste(">"))` 0.05) and *Porphyromonas asaccharolytica* (P-value `r ifelse(main_crc_bugs_summary["porp", "crc_BH"] < 0.05, paste("<"), paste(">"))` 0.05) [Figure 2A]. In contrast, there was no difference in relative abundance in any of these OTUs when considering only those with adenoma [Figure 2B].


***The Lesion Model:***
We next wanted to identify if there were any common bacterial microbiome changes in individuals with adenoma or carcinoma versus normal controls.  We investigated this by creating a model to classify samples as lesion versus normal based on the bacterial community and FIT measurements.  This model had an AUC range of `r format(min(lesion_AUC_run_summary$ROC), digits = 3)` - `r format(max(lesion_AUC_run_summary$ROC), digits = 3)` after 100 iterations of 20 repeated 10-fold cross validations.  The ROC curve for the final lesion model used was within the observed range of the 100 different test set AUC iterations [Figure 3A].  There were a total of `r length(lesion_top10_varsMDA$variable)` variables that were used in this model [Figure 3B].  The FIT measurement for fecal blood resulted in the largest decrease in MDA while the OTU with the largest MDA was *`r gsub("_unclassified", "", lesion_imp_vars[as.character((filter(lesion_top10_varsMDA, variable != "fit_result") %>% top_n(1, mean_MDA) %>% select(variable))[, 1]), "Genus"])`* (`r filter(lesion_top10_varsMDA, variable != "fit_result") %>% top_n(1, mean_MDA) %>% select(variable)`) [Figure 3B].     

If there were common OTUs that could separate adenoma and carcinoma from normal controls, we would expect to find a decrease in the positive probability of the follow up sample to be a lesion. This is what we observed for the lesion model (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_lesion" & comparison == "lesion")[, "BH_correction"] < 0.001, paste("<"), paste(">"))` 0.001).  When we separated individuals based on whether they had an adenoma or carcinoma there was only a decrease in positive probability for the carcinoma group [Figure 3C] (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_lesion" & comparison == "carcinoma_only")[, "BH_correction"] < 0.001, paste("<"), paste(">"))` 0.001) and not for the adenoma group [Figure 3D] (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_lesion" & comparison == "all_adenoma")[, "BH_correction"] < 0.05, paste("<"), paste(">"))` 0.05).   We also observed that there were no significant differences between the predicted and actual calls (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "all" & model_type == "red_lesion" & info == "McnemarPValue")[, "all"] < 0.05, paste("<"), paste(">"))` 0.05).  The lesion model was also able to correctly classify the one individual who still had a carcinoma on follow up [Figure 3C]. 


***The Initial Sample Model:***
After building a model to classify based on lesion we built a separate model specifically to be able to classify whether samples were initial samples based on the bacterial community and FIT measurements.  The initial sample model had an AUC range of `r format(min(IF_AUC_run_summary$ROC), digits = 3)` to `r format(max(IF_AUC_run_summary$ROC), digits = 3)` after 100 iterations of 20 repeated 10-fold cross validations.  The test set AUC range for this model performed better then the training set AUCs.  There was a marked decrease in the ROC curve for the final model used when compared to the 100 test set AUC iterations [Figure 4A].  There were a total of `r length(IF_top10_varsMDA$variable)` variables that were used for this model [Figure 4B].  The variable that resulted in the largest MDA was *`r gsub("_unclassified", "", IF_imp_vars[as.character((filter(IF_top10_varsMDA, variable != "fit_result") %>% top_n(1, mean_MDA) %>% select(variable))[, 1]), "Genus"])`* (`r filter(IF_top10_varsMDA, variable != "fit_result") %>% top_n(1, mean_MDA) %>% select(variable)`) while FIT measurement for fecal blood resulted in the sixth largest decrease in MDA [Figure 4B].  

If there were common OTUs that could separate initial from follow up sample regardless of whether the lesion was adenoma or carcinoma we would expect to find a decrease in the positive probability of the follow up sample to be an initial sample. This is what we observed for the initial sample model (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_IF" & comparison == "lesion")[, "BH_correction"] < 0.001, paste("<"), paste(">"))` 0.001).  When we separated individuals based on whether they had an adenoma or carcinoma there was a decrease in positive probability for both the carcinoma group [Figure 4C] (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_IF" & comparison == "carcinoma_only")[, "BH_correction"] < 0.001, paste("<"), paste(">"))` 0.001) and for the adenoma group [Figure 4D] (P-value `r ifelse(filter(all_probs_paired_wilcox, model_type =="red_IF" & comparison == "all_adenoma")[, "BH_correction"] < 0.001, paste("<"), paste(">"))` 0.001).  For this model there was no difference between the predicted and actual classifications (P-value `r ifelse(filter(all_pred_stat_summary, samples_tested == "all" & model_type == "red_IF" & info == "McnemarPValue")[, "all"] < 0.05, paste("<"), paste(">"))` 0.05).        


***Common OTUs to both Models:***
We next wanted to compare the similarity between the OTU variables used in either model.  The main purpose was to identify which OTUs were important not only for the classification of lesion but also for the classification of initial or follow up sample.  Potentially, these specific OTUs are the most important with respect to the bacterial microbiome response to removal of lesion.  When we compared the two different models with each other there were a total of `r length(common_vars_summary$otu)` common OTUs.  Some of the most common taxonomic identifications belonged to *`r filter(common_vars_summary, otu == "Otu000028")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000018")[, "lowest_ID"]`*, *`r filter(common_vars_summary, otu == "Otu000009")[, "lowest_ID"]`*, and *`r filter(common_vars_summary, otu == "Otu000005")[, "lowest_ID"]`*.  The vast majority of these OTUs had classifications to bacteria typically thought of as commensal [Table S2]. 


***Treatment Differences:***
After observing these changes in the bacterial community and positive probability we wanted to assess whether additional treatments, such as chemotherapy and radiation, could have an impact on the results that we observed.  There was only a significant difference for change in positive probability for those treated with chemotherapy for the initial sample model (P-value `r ifelse(chemo_rad_summary['chemo_red_IF', 'bh'] < 0.05, paste("<"), paste(">"))` 0.05).  This suggests that follow up samples for those treated for chemotherapy my have had a larger change from the initial sample then those without such treatment.  All other variables that were tested showed no difference based on whether chemotherapy or radiation was used [Table S3].  


\newpage


## Discussion

This study builds upon previous work from numerous labs that have looked into the bacterial microbiome as a potential screening tool [@yu_metagenomic_2017; @baxter_microbiota-based_2016; @zeller_potential_2014; @zackular_human_2014; @warren_co-occurrence_2013] by exploring what happens to the bacterial community after surgical removal of a lesion.  Based on previous work by Arthur, et al. [@arthur_intestinal_2012]  it may not be surprising to have E.coli as one of the most important OTUs and one that was common to both models.  Interestingly, many of the most important OTUs had taxonomic identification for resident gut microbes.  This could suggest that the bacterial community is one of the first components that could change during the pathogenesis of disease.  These bacterial microbiome changes could be the first step in allowing more inflammatory bacterium to gain a foothold within the colon [@flynn_metabolic_2016].

From our results there were large observed differences in the bacterial microbiome in samples before and after lesion removal based on whether the individual had an adenoma or carcinoma.  In individuals with carcinoma compared to adenoma there were much larger differences between initial and follow up samples based on the community beta diversity and in fecal blood as measured by FIT [Figure 1].  However, there were no differences between initial and follow up samples for any alpha diversity metric measured regardless of whether the individual had an adenoma or carcinoma [Table S1].  There was also no differences in relative abundance of any specific OTU for lesion, adenoma only, or carcinoma only [Figure S1].   

Although there were no differences when investigating all OTUs, when looking specifically at four OTUs that taxonomically classified to previously suggested carcinoma associated microbes we found that only `r length(rownames(filter(main_crc_bugs_summary, round(crc_BH, 2) < 0.05)))`/`r length(rownames(main_crc_bugs_summary))` had a decrease in relative abundance between initial and follow up for those with carcinoma and `r length(rownames(filter(main_crc_bugs_summary, adn_BH < 0.05)))`/`r length(rownames(main_crc_bugs_summary))` had differences for those with adenoma.  This data would suggest that these specific OTUs may be important in the transition of an adenoma to a carcinoma but less so in the initiation of an adenoma from benign tissue.  

We next created a model that incorporated FIT and the bacterial microbiome to be able to classify lesions (adenoma or carcinoma).  Based on this model we found that the follow up samples were closer to normal then the initial samples due to a decrease in positive probability for lesion and that the commonly associated CRC bacteria were not highly represented within this model with the exception of *Porphyromonas asaccharolytica* [Figure 3].  Although there was a detectable change and this change was towards what would be expected for normal controls, it should be noted that the follow up samples may not be a completely normal bacterial microbiome.  

After creating the lesion mode we then created a second model to classify initial versus follow up samples.  We found that this model was able to accurately classify initial versus follow up samples suggesting that regardless of adenoma or carcinoma there are distinct common changes within the bacterial microbiome that occurs after lesion removal [Figure 4].  Both models had OTUs that overwhelmingly belonged to commensal bacteria.  Providing additional information on the importance of commensal bacteria was that there were a total of `r length(common_vars_summary$otu)` OTUs in common to both models and the vast majority belonged to regular residents of our gut community [Table S2].    

Within our study there was a significant difference for the time elapsed in the collection of the follow up sample between adenoma and carcinoma (uncorrected P-value `r ifelse(time_pvalues["time", "uncorrected_p_value"] < 0.05, paste("<"), paste(">"))` 0.05), with time passed being less for adenoma (`r format(time_summary['adenoma', 'mean_days'], digits = 3)` +/- `r format(time_summary['adenoma', 'sd_days'], digits = 3)` days) than carcinoma (`r format(time_summary['carcinoma', 'mean_days'], digits = 3)` +/- `r format(time_summary['carcinoma', 'sd_days'], digits = 3)` days).  These results would indicate that the findings described were specific to the surgical intervention and that some of the differences observed between carcinoma and adenoma samples could be due to differences in collection time between samples for the two different groups.  Specifically, it could confound the observation that carcinomas changed more than adenomas [Figure 1A & 1D]. This confounding though would not affect the observations where these individuals were grouped together [Figure 3 & 4]. 

Curiously, we observed that the typical CRC associated bacteria were not predictive within our models.  There are a number of reasons why this may have occurred.  First, is that they were not present in enough individuals to be able to classify those with and without disease with a high degree of accuracy.  Second, is that our Random Forest models were able to gather the same information from measures such as FIT or other OTUs.  It is also possible that both of these explanations could have played a role.  Regardless, our observations would suggest that an individual's resident bacteria have a large role to play in disease initiation and could change in a way that allows predictive models to lower the positive probability of a lesion after surgery [Figure 3C & 3D].  It should be noted that our study does not argue against the importance of these CRC associated bacteria in the pathogenesis of disease but rather that they are not the main bacteria changing after surgical removal of lesion.  In fact, it is possible that these CRC associated bacteria are important in the transition from adenoma to carcinoma and would be one explanation as to why in our data we not only see high initial relative abundances in carcinoma and not adenoma individuals but also large decreases in relative abundance in some of those with carcinoma but not in those with adenoma after surgery [Figure 2].  

Many of the common OTUs between the two models had OTUs that taxonomically classified to potential butyrate producers [Table S2].  Another batch of OTUs classified to bacteria that can either degrade polyphenols or are inhibited by them.  Both butyrate and polyphenols are thought to be protective against cancer in part by reducing inflammation [@louis_gut_2014].   These protective compounds are derived from the breakdown of fiber, fruits, and vegetables by resident gut microbes.  One example of this potential diet-microbiome-inflammation-polyp axis is that *Bacteroides*, which was highly prevalent in our models, are known to be increased in those with high non-meat based protein consumption [@zhu_intake_2016].  High protein consumption in general has been linked with an increased CRC risk [@mu_colonic_2016].  Conversely, *Bacteroides* are inhibited by polyphenols which are derived from fruits and vegetables [@ozdal_reciprocal_2016].  Our data fits with the hypothesis that the microbial metabolites from breakdown products within our own diet could not only help to shape the existing community but also have an effect on CRC risk and disease progression.   
  
One limitation of our study is that we do not know whether individuals who were still classified as positive by the lesion model eventually had a subsequent CRC diagnosis.  This information would help to strengthen the case for our lesion model keeping a number of individuals above the cutoff threshold even though at follow up they were diagnosed as no longer having a lesion.  Another limitation is that we do not know if adding modern tests such as the stool DNA test [@cotter_long-term_2016] could help improve our overall AUC.  This study also drew heavily from those with Caucasian ancestry making it possible that the observations may not be representative of those with either Asian or African ancestry.  Although our training and test set are relatively large we still run the risk of over-fitting or having a model that may not be representative of other populations.  We've done our best to safeguard against this by not only running 10-fold cross validation but also having over 100 different 80/20 splits to try and mimic the type of variation that might be expected to occur.

Interestingly, within the initial sample model the test data performed better than the training data.  This may have occurred because the training AUC determined from 20 repeated 10 fold cross validation removed samples at random and did not take into account that they were matched samples.  Another potential reason is that the model itself may be over-fit since the total number of samples was not that large.  However, the lesion model did not suffer from these discrepancies.  Further independent studies need to be carried out to verify our findings since we are dealing with correlations that may not be truly representative of the pathogenesis of disease.

Despite these limitations our findings add to the existing scientific knowledge on CRC and the bacterial microbiome: That there is a measurable difference in the bacterial community after adenoma or carcinoma removal.  Further, the ability for machine learning algorithms to take bacterial microbiome data and successfully lower positive probability after either adenoma or carcinoma removal provides evidence that there are specific signatures, mostly attributable to commensal organisms, associated with these lesions.  Our data provides evidence that commensal bacteria may be important in the development of polyps and also potentially the transition from adenoma to carcinoma.  



\newpage

## Methods

***Study Design and Patient Sampling:***
The sampling and design were similar to that reported in Baxter, et al [@baxter_microbiota-based_2016].  In brief, study exclusion involved those who had already undergone surgery, radiation, or chemotherapy, had colorectal cancer before a baseline fecal sample could be obtained, had IBD, a known hereditary non-polyposis colorectal cancer, or familial adenomatous polyposis.  Samples used to build the models for prediction were collected either prior to a colonoscopy or between 1 - 2 weeks after.  The bacterial microbiome has been shown to normalize within this time period [@obrien_impact_2013].  Our follow up data set had a total of `r length(rownames(good_metaf))` individuals that not only had a sample as described but also a follow up sample between `r min(good_metaf$time)` - `r max(good_metaf$time)` days after lesion removal and treatment had been completed. This study was approved by the University of Michigan Institutional Review Board.  All study participants provided informed consent and the study itself conformed to the guidelines set out by the Helsinki Declaration.


***FIT and 16S rRNA Gene Sequencing:***
FIT was analyzed as previously published using both OC FIT-CHEK and OC-Auto Micro 80 automated system (Polymedco Inc.) [@daly_evaluation_2013]. 16S rRNA gene sequencing was completed as previously described by Kozich, et al. [@kozich_development_2013].  DNA extraction used the 96 well Soil DNA isolation kit (MO BIO Laboratories) and an epMotion 5075 automated pipetting system (Eppendorf).  The V4 variable region was amplified and the resulting product was split between three sequencing runs with normal, adenoma, and carcinoma evenly represented on each run.  Each group was randomly assigned to avoid biases based on sample collection location.


***Sequence Processing:***
The mothur software package (v1.37.5) was used to process the 16S rRNA gene sequences.  This process has been previously described [@kozich_development_2013].  The general processing workflow using mothur was as follows: Paired-end reads were first merged into contigs, quality filtered, aligned to the SILVA database, screened for chimeras, classified with a naive Bayesian classifier using the Ribosomal Database Project (RDP), and clustered into Operational Taxonomic Units (OTUs) using a 97% similarity cutoff with an average neighbor clustering algorithm.  The number of sequences for each sample was rarefied to `r ifelse(sd(total_sub_Seqs) == 0, paste(mean(total_sub_Seqs)), paste("Error Check Subsampling"))` in an attempt to minimize uneven sampling.    


***Lesion Model Creation:***
The Random Forest [@breiman_random_2001] algorithm was used to create the model used for prediction of lesion (adenoma or carcinoma) with the main training and testing of the model completed on a data set of `r length(rownames(metaI)) - length(rownames(good_metaf))` individuals.  This model was then applied to our follow up data set of `r length(rownames(good_metaf))` individuals.  

In brief, the model included data on FIT and the bacterial microbiome. Non-binary data was checked for near zero variance and OTUs that had near zero variance were removed.  This pre-processing was performed with the R package caret (v`r paste(packageVersion("caret"))`).  Optimization of the mtry hyper-parameter involved taking the samples and making 100 different 80/20 (train/test) splits of the data where normal and lesion were represented in the same proportion within both the whole data set and the 80/20 split.  Each of these splits were then run through 20 repeated 10-fold cross validations to optimize the mtry hyper-parameter by maximizing the AUC (Area Under the Curve of the Receiver Operator Characteristic).  This resulting model was then tested on the 20% of the data that was originally held out from this overall process. Next, in order to assess which variables were most important to the model we counted the number of times a variable was present in the top 10% of mean decrease in accuracy (MDA) for each of the 100 different 80/20 split models and then filtered this list to variables that were only present more than 50% of the time.  This final collated list of variables was what was considered the most important for the model.  This reduced data set was then run through the mtry optimization again.  Once the ideal mtry was found the entire `r length(rownames(metaI)) - length(rownames(good_metaf))` sample set was used to create the final Random Forest model on which classifications on the `r length(rownames(good_metaf))`-person cohort was completed.  

The default cutoff of 0.5 was used as the threshold to classify individuals as positive or negative for lesion.  The hyper-parameter, mtry, defines the number of variables to investigate at each split before a new division of the data was created with the Random Forest model.  


***Initial Sample Model Creation:***
We also investigated whether a model could be created that could identify pre (initial) and post (follow up) lesion removal samples from each other.  The main difference was that only the `r length(rownames(good_metaf))`-person cohort was used at all stages of model building and classification.  Other than this difference the creation of this model and optimization of the mtry hyper-parameter was completed using the same procedure as was used for the lesion model.  Instead of classifying samples as positive or negative of lesion this model classified samples as positive or negative for being an initial to lesion removal sample. 


***Statistical Analysis:***
The R software package (v`r paste(version$major, ".", version$minor, sep = "")`) was used for all statistical analysis.  Comparisons between bacterial community structure utilized PERMANOVA [@anderson_permanova_2013] in the vegan package (v`r paste(packageVersion("vegan"))`).  Comparisons between probabilities as well as overall OTU differences between initial and follow up samples utilized a paired Wilcoxson ranked sum test.  Where multiple comparison testing was appropriate a Benjamini-Hochberg (BH) correction was applied [@benjamini_controlling_1995] and a corrected P-value of less than 0.05 was considered significant.  Unless otherwise stated the P-values reported are those that were BH corrected.  


***Analysis Overview:***
We first wanted to test if there were any differences based on whether the individual had an adenoma or carcinoma.  This was done by testing initial and follow up samples for differences in alpha and beta diversity, testing differences in FIT between initial and follow ups, testing all OTUs, and investigating the relative abundance of specific previously associated CRC bacteria (*Fusobacterium nucleatum*, *Parvimonas micra*, *Peptostreptococcus assacharolytica*, and *Porphyromonas stomatis*) based on adenoma and carcinoma.  From here the lesion model was then tested for accuracy in prediction and whether it reduced the positive probability of lesion after surgery.  We then used the initial sample model to assess whether it could classify samples better then the lesion model and whether it could reduce the positive probability of an initial sample in the follow up samples.  Finally, a list of common OTUs were found for the two two different models used.  


***Reproducible Methods:***
A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Sze_followUps_2017.  Raw sequences have been deposited into the NCBI Sequence Read Archive (SRP062005 and SRP096978) and the necessary metadata can be found at https://www.ncbi.nlm.nih.gov/Traces/study/ and searching the respective SRA study accession.


\newpage


**Figure 1: General Differences between the Adenoma and Carcinoma Group.**  A) A significant difference was found between the adenoma and carcinoma group for thetayc (P-value = `r format(change_summary['thetayc', 'Pvalue'], scientific = FALSE, digits = 3)`).  B) A significant difference was found between the adenoma and carcinoma group for change in FIT measurement (P-value = `r format(change_summary['fit', 'Pvalue'], scientific = TRUE, digits = 3)`). C) NMDS of the initial and follow up samples for the Adenoma group.  D) NMDS of the initial and follow up samples for the Carcinoma group. For C) and D) the teal represents initial samples and the pink represents follow up samples. 

**Figure 2: Previously Associated CRC Bacteria in Initial and Follow Up Samples.**  A) Carcinoma initial and follow up samples had an observed significant difference in initial and follow up sample for the OTUs classified as *Parvimonas micra* (P-value = `r format(main_crc_bugs_summary["parv", "crc_BH"], digits = 3)`) and *Porphyromonas asaccharolytica* (P-value = `r format(main_crc_bugs_summary["porp", "crc_BH"], digits = 3)`).  B) Adenoma initial and follow up samples.  There were no significant differences between initial and follow up (P-value = `r format(main_crc_bugs_summary["porp", "adn_BH"], digits = 3)`.


**Figure 3: The Lesion Model.** A) ROC curve: The shaded areas represents the range of values of a 100 different 80/20 splits of the test set data and the blue line represents the model using 100% of the data set and what was used for subsequent classification.  B) Summary of Important Variables.  MDA of the most important variables in the lesion model.  The black point represents the mean and the different colors are the values of each different run up to 100. C) Positive probability change from initial to follow up sample in those with carcinoma.  D) Positive probability change from initial to follow up sample if those with adenoma or advanced adenoma (Screen Relevant Neoplasia (SRN)).


**Figure 4: The Initial Sample Model.**  A) ROC curve: The shaded areas represents the range of values of a 100 different 80/20 splits of the test set data and the blue line represents the model using 100% of the data set and what was used for subsequent classification.  B) Summary of Important Variables.  MDA of the most important variables in the initial sample model.  The black point represents the mean and the different colors are the values of each different run up to 100.  C) Positive probability change from initial to follow up sample in those with carcinoma.  D) Positive probability change from initial to follow up sample of those with adenoma or advanced adenoma (Screen Relevant Neoplasia (SRN)).



\newpage

**Figure S1: Distribution of P-values from Paired Wilcoxson Analysis of All OTUs for Initial versus Follow Up**

**Figure S2: Thetayc Versus Time of Follow up Sample from Initial**

\newpage

## Declarations

### Ethics approval and consent to participate

### Consent for publication

### Availability of data and material

A detailed and reproducible description of how the data were processed and analyzed can be found at https://github.com/SchlossLab/Sze_followUps_2017.  Raw sequences have been deposited into the NCBI Sequence Read Archive (SRP062005 and SRP096978) and the necessary metadata can be found at https://www.ncbi.nlm.nih.gov/Traces/study/ and searching the respective SRA study accession.

### Competing Interests

All authors declare that they do not have any relevant competing interests to report.

### Funding

This study was supported by funding from the National Institutes of Health to P. Schloss (R01GM099514, P30DK034933) and to the Early Detection Research Network (U01CA86400).

### Authors' contributions

All authors were involved in the conception and design of the study. MAS analyzed the data. NTB processed samples and analyzed the data. All authors interpreted the data. MAS and PDS wrote the manuscript. All authors reviewed and revised the manuscript. All authors read and approved the final manuscript.

### Acknowledgements

The authors thank the Great Lakes-New England Early Detection Research Network for providing the fecal samples that were used in this study. We would also like to thank Amanda Elmore for reviewing and correcting code error and providing feedback on manuscript drafts.  We would also like to thank Nicholas Lesniak for providing feedback on manuscript drafts.

\newpage

## References




